{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "23e83cff",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-03T16:16:08.298582Z",
     "start_time": "2022-11-03T16:16:07.301043Z"
    }
   },
   "outputs": [],
   "source": [
    "import docx\n",
    "from collections import Counter\n",
    "import pandas as pd\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "5f63f7f6",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-03T16:06:22.423650Z",
     "start_time": "2022-11-03T16:06:22.139957Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['', '', 'Processing of non-coding RNAs by Mlp1 in Tetrahymena thermophila', '', 'Kyra Kerkhofs', '', 'A dissertation submitted to the faculty of graduate studies in partial fulfillment of the requirements for the degree of ', '', '', 'Doctor of Philosophy ', '', 'Graduate Program in Biology\\nYork University\\n Toronto, Ontario', 'December 2022', '', '© Kyra Kerkhofs 2022', 'Abstract', 'In eukaryotes, many RNA species are transcribed as precursor molecules and require post-transcriptional processing to become a functional mature RNA. This includes the very abundant cytoplasmic transfer RNA (tRNAs) and ribosomal RNA (rRNAs) that function in translation of messenger RNAs (mRNAs). In addition, other non-coding RNAs also require extensive processing and assembly into ribonucleoproteins (RNPs), including small nuclear RNAs (snRNAs) that function in pre-mRNA splicing and small nucleolar RNAs (snoRNAs) involved in rRNA and non-coding RNA processing and modification. The ciliate Tetrahymena thermophila is a highly studied and well-characterized model organism with many groundbreaking discoveries. However, many important non-coding RNA processing pathways remain elusive in this organism. We used molecular biology, biochemistry, cell biology and bioinformatics to investigate the role of a novel La protein named Macronucleus Localized Protein of unknown function (Mlp1) in pre-tRNA, snoRNA and snRNA biogenesis. ', 'RNA polymerase III transcribes nascent pre-tRNA resulting in the formation of the 5’-leader, 3’-trailer and typical 3’-terminal uridylate sequence. Pre-tRNA processing has been well-described in humans, yeasts and other eukaryotic species where the 5’- and 3’-extensions (i.e. the 5’-leader and 3’-trailer) are removed sequentially. La proteins have a high binding affinity for uridylate sequences through concomitant binding between highly conserved RNA binding domains including the La motif (LaM) and RNA-recognition motif-1 (RRM1). Therefore, La proteins are the first proteins to associate with nascent pre-tRNAs. The next step involves endonucleolytic processing of the 5’-leader, followed by endonucleolytic processing of the 3’-trailer. In absence of the La protein, the processing pathway is reversed where the 3’-trailer is rapidly trimmed by a 3’-exonuclease, followed by endonucleolytic 5’-leader processing. In Chapter II, we characterize a novel La protein lacking a typical RNA-binding domain required for high affinity binding of uridylate RNA. We performed typical La protein assays and confirm that Mlp1 performs typical La protein functions including, preferential binding of pre-tRNAs compared to mature tRNAs, uridylate-specific binding through highly conserved residues in the LaM and RNA chaperone activity to promote processing and maturation of misfold nascent pre-tRNAs. However, in contrast to pre-tRNA processing in other eukaryotes, depletion of Mlp1 results in 3’-trailer stabilization instead of rapid trimming by 3’-exonucleases, indicating that Mlp1 is linked to a fundamentally different mechanism of tRNA processing in T. thermophila.\\xa0', 'La proteins have been shown to associate with nascent RNA polymerase II non-coding transcripts through association with precursor forms of spliceosomal snRNAs ending in 3’-uridylate sequences. In Chapter III, we found that Mlp1 associates with mature snRNAs lacking the typical high affinity uridylate binding site and a core protein component of the snRNP. More specifically, Mlp1 interact with the U4/U6 di-snRNP complex, which is formed during the splicing reaction, and Mlp1 depletion affect assembly of this complex. Mlp1 depletion results in diminished splicing efficiency of pre-mRNAs, indicating a functional role for the genuine La protein Mlp1 in pre-mRNA splicing in T. thermophila. ', 'In chapter IV, we described that Mlp1 associates with all previously described snoRNAs and core proteins components of the snoRNPs which function as guide RNAs in post-transcriptional modification of rRNA, snRNA and tRNAs. We used this as a metric to predict novel snoRNAs in T. thermophila and validate expression. Mlp1 binding to mature snoRNAs and core snoRNP proteins led us to investigate if Mlp1 is involved in assembly of snoRNPs. We did not find a striking difference in snoRNP assembly upon Mlp1 depletion. This could indicate that Mlp1 could function redundantly with another protein resulting in only minor effects on snoRNP assembly and function. ', '', '', ' ', '', '\\n', 'Acknowledgements', '', '', '', ' ', '\\n', 'Table of Contents', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '\\n', 'List of Figures', 'Chapter I: Literature Review', '', '', '', '', '', '', '', '', '', '', '', '', '', 'Chapter II: Altered tRNA processing is linked to a distinct and unusual La protein in Tetrahymena thermophila', '', '', '', '', '', '', '', 'Chapter III: Genuine La protein Mlp1 functions in U4/U6 di-snRNP assembly and affects splicing in Tetrahymena thermophila', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '\\n', 'Chapter IV: The Tetrahymena thermophila La protein Mlp1 is involved in snoRNP assembly', '', '', '', '', '', '', '', '', '', '', '', 'Chapter V: Conclusion and future perspectives ', '', '', '', '', '', '', '', 'List of Supplementary Figures', 'Chapter II: Altered tRNA processing is linked to a distinct and unusual La protein in Tetrahymena thermophila', '', '', '', '', '', '', '', '', '', 'Chapter III: Genuine La protein Mlp1 functions in U4/U6 di-snRNP assembly and affects splicing in Tetrahymena thermophila', '', '', '', '\\n', 'List of Tables', 'Chapter II: Altered tRNA processing is linked to a distinct and unusual La protein in Tetrahymena thermophila', '', '', 'Chapter III: Genuine La protein Mlp1 functions in U4/U6 di-snRNP assembly and affects splicing in Tetrahymena thermophila', '', '', '', 'Chapter IV: The Tetrahymena thermophila La protein Mlp1 is involved in snoRNP assembly', '', '', '', '', '\\n', 'List of Supplementary Tables', 'Chapter II: Altered tRNA processing is linked to a distinct and unusual La protein in Tetrahymena thermophila', '', '', '', 'Chapter IV: The Tetrahymena thermophila La protein Mlp1 is involved in snoRNP assembly', '', '', '', '', '', 'Chapter I: Literature Review', 'Chapter II: Altered tRNA processing is linked to a distinct and unusual La protein in Tetrahymena thermophila', 'Chapter III: Genuine La protein Mlp1 functions in U4/U6 di-snRNP assembly and affects splicing in Tetrahymena thermophila', 'Chapter IV: The Tetrahymena thermophila La protein Mlp1 is involved in snoRNP assembly', 'Chapter V: Conclusion and future perspectives ', '\\n', 'List of Abbreviations', '', '', '', '', '', '', '', '', '', 'Chapter I:\\nReview of Literature', '\\n', 'Multiple noncoding RNAs are required in protein translation ', 'RNA synthesis is achieved by three different multi-subunit RNA polymerases in eukaryotes (Carter & Drouin 2009). RNA polymerase II transcribes all coding messenger RNAs (mRNAs) and many noncoding RNAs including small nuclear RNAs (snRNAs) and small nucleolar RNAs (snoRNAs) (Guiro & Murphy 2017). RNA polymerase I produces a single 47S ribosomal RNA (rRNA) (35S rRNA in yeast) which is processed into the 18S, 5.8S and 28S rRNAs (18S, 5.8S and 25S in yeast), which are components of the small and large ribosomal subunits (Klinge & Woolford 2019), and RNA polymerase III transcribes short transcripts, including the 5S rRNA which is a part of the large ribosomal subunit and transfer RNAs (tRNAs) (Dieci et al. 2007). RNA polymerase I and III are similar in that they produce a relatively small number of highly expressed transcripts (Paule & White 2000), while RNA polymerase II transcribes all 20,000 coding genes with approximately 50% of transcription contributing to the production mRNAs encoding ribosomal proteins (Warner 1999). ', 'Ribosome biogenesis involves (1) transcription, processing, and post-transcriptional modification of rRNA by snoRNAs, (2) transcription, splicing by snRNAs, nuclear export, translation and nuclear import of ribosomal proteins (RPs), (3) assembly of the small ribosomal subunit (40S – consisting of 18S rRNA and 33 ribosomal proteins (RPS)) and the large ribosomal subunit (60S – consisting of 28S, 5.8S and 5S rRNA and 46 ribosomal proteins (RPL)), and (4) export of the small and large ribosomal subunits to the cytoplasm (Figure 1) (Henras et al. 2015; Lafontaine 2015). Polypeptide production relies on coordinated function of different types of RNA and must be tightly regulated at different levels to ensure normal cell growth and proliferation (Moir & Willis 2013). Translation is an intricate process where each puzzle piece needs to fit in exactly the right place in order to maintain homeostasis. The focus of this thesis is on the processing and function of RNA polymerase III transcripts, including nascent pre-tRNAs and the U6 small nuclear RNA (snRNA) and RNA polymerase II transcripts including U1, U2, U4 and U5 snRNAs and snoRNAs. ', '', '', 'Figure legend on next page', 'Figure 1. Ribosome biogenesis involves all three RNA polymerases. ', 'RNA polymerase II is found in the nucleus and transcribes messenger RNAs (mRNAs) and multiple non-coding RNAs such as small nuclear RNAs (snRNAs) (not shown) and small nucleolar RNAs (snoRNAs). mRNAs encoding ribosomal proteins and assembly factors are translated in the cytoplasm and imported into the nucleus for ribosome assembly. RNA polymerase I located in the nucleolus and transcribes a single polycistronic transcript, the 47S pre-rRNA, that is processed into 18S, 5.8S and 28S rRNA. Further maturation occurs through post-transcriptional modifications by snoRNAs. RNA polymerase III is found the nucleus and transcribes 5S rRNA which is a component of the large ribosomal subunit 60S. The small and large ribosomal subunits 40S and 60S, respectively, are assembled in the nucleus and transported into the cytoplasm for translation of mRNAs into proteins. Figure adapted from (Lafontaine 2015). ', 'Transcription and biogenesis of non-coding RNAs', 'RNA polymerase I transcription of rRNA and maturation ', 'Transcription of rRNA in eukaryotes takes place in the nucleolus from ribosomal DNA (rDNA) that is encoded in multiple copies throughout the genome. For example, the yeast S. cerevisiae encodes approximately 150 copies per haploid genome and humans between 300 and 400 copies in per diploid genome (Henras et al. 2015). The rRNA is transcribed as a large precursor rRNA by RNA polymerase I, containing the 18S, 5.8S and 28S rRNA (see Figure 1) and external (5’-ETS and 3’-ETS) and internal spacers (ITS-1 and ITS-2) (Klinge & Woolford 2019), whereas the 5S rRNA is transcribed by RNA polymerase III (see Figure 1) (see later) (Dieci et al. 2007). ', 'Processing of the nascent pre-rRNA is initiated co-transcriptionally and involves multiple complex endo- and exonucleolytic processing steps (reviewed in Henras et al., 2015). In addition to the multiple endo- and exonucleolytic processing step, rRNAs molecules obtain extensive nucleotide modifications by up to approximately 200 snoRNAs (Lestrade & Weber 2006). Most snoRNAs are transcribed by RNA polymerase II (Guiro & Murphy 2017) and function through base pairing with the target rRNA sequence. Two classes of snoRNAs, box C/D and box H/ACA snoRNAs, direct 2’-hydroxyl-methylation (2’-O-methylation) of the RNA backbone at the 2’-O position of the sugar ring and pseudouridylation which forms a 5’-ribosyl isomer of uridine named pseudouridine or Ψ, respectively, to the rRNA (Figure 2) (Decatur & Fournier 2002). Post-transcriptional modifications are found in highly conserved rRNA regions which are often functionally important (Decatur & Fournier 2002). ', '', 'Figure . Schematic representation of pseudouridylation and 2’-O-methylation of RNA. ', 'The pseudouridine (Ψ) post-transcriptional modification is a rotation isomer of uridine (U) converted by pseudouridine synthetases. 2’O-methylation post-transcriptional modification is the addition of a methyl group at the 2’-O-position of the RNA backbone by 2’-O-methyltransferases. Figure from (Karijolich & Yu 2010).', 'snRNA transcription by RNA polymerase II and maturation', 'RNA polymerase II transcribes nascent pre-mRNAs which obtain a 5’-guanine-N7 methyl (m7G) cap co-transcriptionally linked via an inverted 5’-5’ triphosphate bridge to the first nucleoside of the nascent transcript (Figure 3a) (reviewed in Ghosh and Lima, 2010). The m7G cap co-transcriptionally added to nascent mRNA has multiple functions, including protection against 5’-3’ exonucleases, efficient splicing of precursor mRNAs, polyadenylation at the 3’-end, nuclear export and efficient translation (Lewis & Izaurflde 1997; Pabis et al. 2013). In addition to transcription of mRNAs, RNA polymerase II transcribes a large number of short non-coding RNAs, including small nuclear (snRNA) and small nucleolar (snoRNAs) (Guiro & Murphy 2017). Well-characterized RNA polymerase II transcribed non-coding snRNAs include the spliceosomal snRNAs U1, U2, U4 and U5. Mature RNA polymerase II transcribed snRNA transcripts contain a 2,2,7-trimethylguanosine (m3G) cap which is formed by further dimethylation of the m7G cap at the N2 guanine position of the m7G cap structure (Figure 3b) (Ghosh & Lima 2010). In addition to modification of the 5’-cap, snRNAs obtain two types of post-transcriptional modification including 2’-O-methylation and pseudouridylation which are guided by a specific subset of snoRNAs located in the Cajal bodies, named small Cajal body-specific RNAs (scaRNAs) (Karijolich & Yu 2010; Jády & Kiss 2001). Post-transcriptional modifications found in spliceosomal snRNAs are highly conserved throughout evolution and are located in functionally important regions for splicing, including regions of snRNA/snRNA and pre-mRNA/snRNA hybridization (Figure 4) (Karijolich & Yu 2010). ', '', 'Figure . Synthesis of m7G and m3G-cap formation in RNA polymerase II transcripts.', 'a Transcripts produced by RNA polymerase II co-transcriptionally obtain a 5’-guanine-N7 methyl (m7G) through linkage via an inverted 5’-5’ triphosphate bridge to the first nucleoside. b A subset of non-coding transcripts are further post-transcriptionally processed to obtain a 2,2,7-trimethylguanosine (m3G) cap structure which is obtained by further dimethylation of the m7G cap at the N2 guanine position of the m7G cap structure. Transcripts containing a m3G cap structure include RNA polymerase II transcribed snRNAs and some snoRNAs. c The γ-methylphosphate cap is formed through transfer of a methyl group on the primary transcript by γ-methyltransferases. Transcripts containing a γ-methylphosphate cap include RNA polymerase III transcripts U6 snRNA and 7SK RNA. Figure adapted from (Ghosh & Lima 2010).', '', 'The spliceosome consists of five uridine-rich snRNAs, U1, U2, U4, U5 and U6 (Figure 4), and 100s of proteins (Jurica & Moore 2003). While RNA polymerase II transcribes U1, U2, U4 and U5, the U6 snRNA is transcribed by RNA polymerase III and contains a γ-monomethyl cap (Singh & Reddy 1989). Methylation of the γ-phosphate of the 5’-triphosphate cap is catalyzed by the γ-methyltransferase transferring a methyl group from the S-adenosylmethionine to a γ-phosphate oxygen at the nascent 5’-end of the RNA polymerase III transcript (Figure 3b) (Gupta et al. 1990; Shimba & Reddy 1994). Likewise, the 7SK RNA contains a γ-monomethyl cap structure (Gupta et al. 1990).', '', '', 'Figure legend on next page', 'Figure . Schematic representation of the secondary structures of spliceosomal snRNAs. ', 'The secondary structures of the spliceosomal small nuclear (snRNAs) are shown. Post-transcriptional modifications including pseudouridylation and 2’-O methylations are highlighted by a rectangle or circle surrounding the individual nucleotide, respectively. The Sm-binding sites are highlighted with dark gray boxes. Figure adapted from (Karijolich & Yu 2010).', '', 'snoRNA transcription by RNA polymerase II', 'snoRNAs are also transcribed by RNA polymerase II and are found abundantly in the nucleolus within ribonucleoprotein (RNP) complexes named snoRNPs which function as guide RNAs in post-transcriptional modification of rRNA, snRNA and tRNAs (Kiss 2002). Two major classes of snoRNAs exist based on sequence and structural elements, including box C/D and box H/ACA snoRNAs. Box C/D snoRNAs guides 2’-O-methylation and box H/ACA guided pseudouridylation. ', 'Different eukaryotes employ different mechanisms of snoRNA transcription, including transcription from independent genes, polycistronic transcription and processing from mRNA introns (Kufel & Grzechnik 2019) (Figure 5). For example, the yeast S. cerevisiae expresses 76 snoRNAs (Piekna-Przybylska et al. 2007) and mostly employs transcription from independent genes. However, a small number of snoRNAs are found within mRNA introns (8 snoRNAs) and are organized in polycistrons (5 clusters encoding 17 snoRNAs) (Piekna-Przybylska et al. 2007). In contrast, humans encode the majority of snoRNA genes within non-coding genes or introns of mRNAs in the same orientation with only a small number of human snoRNAs transcribed by RNA polymerase II as independent genes. Host genes encoding snoRNAs are often 5’-terminal oligo pyrimidine (5’-TOP) mRNAs which encode for highly expressed ribosomal proteins or proteins involved in ribosome biogenesis or translation (Ojha et al. 2020; Smith & Steitz 1998; Tycowski et al. 1993). snoRNAs are transcribed as precursors and are required to undergo maturation at the 5’ and 3’-ends. The fully processed snoRNAs lack a poly(A) tail and are between 60 and 300 nucleotides long (reviewed in Falaleeva & Stamm 2013).', '', 'Figure . Genomic organization of small nucleolar RNA (snoRNA) genes in different eukaryotic species. ', 'Genomic organization of snoRNA transcription generally follows a similar mechanism for species within a certain branch of the evolutionary tree. The yeast S. cerevisiae for example encodes the majority of snoRNAs as independent genes under regulation of its own promotor and terminator, while Drosophila melanogaster and Caenorhabditis elegans are generally transcribed as a polycistronic transcript. In vertebrates, including humans, the majority of snoRNAs are encoded within the introns of mRNAs or long non-coding RNAs (lncRNAs). P: promotor, T: terminator. Figure from (Kufel & Grzechnik 2019).', '', 'RNA polymerase III transcription of tRNAs and other non-coding transcripts', 'RNA polymerase III transcribes short transcripts that make up approximately 15% (Turowski & Tollervey 2016) of the total cellular RNA and include many different RNAs involved in protein synthesis (i.e. 5S rRNA and tRNAs), pre-rRNA and pre-tRNA processing factors (i.e. RNase MRP and RNase P RNAs), co-translational translocation of nascent polypeptide chains during endoplasmic reticulum (ER) translation (i.e. 7SL RNA), transcription regulation of RNA polymerase II elongation factor P-TEFb (i.e. 7SK RNA) and pre-mRNA splicing (i.e. U6 snRNA) (reviewed in Dieci et al., 2007). In some species, certain transcripts are transcribed by RNA polymerase III instead of RNA polymerase II. A few examples include the telomerase (TER) RNA functioning in telomeric DNA extension in ciliates, the U3 snoRNA in plants which is a component of the rRNA processing complex and a small number of snoRNAs in S. cerevisiae, A. thaliana, C. elegans and D. melanogaster (reviewed in Dieci et al., 2007). ', 'The most abundantly transcribed gene products include 5S rRNA and tRNAs. The 5S rRNA is a part of the large ribosomal subunit (Klinge & Woolford 2019), while tRNAs function as adaptors between the mRNA codon sequence and corresponding amino acid post-transcriptionally linked to the 3’-CCA sequence (O’Donoghue et al. 2018). The RNA polymerase III complex consists of a DNA-binding factor, a transcription initiation factor and the polymerase core. RNA polymerase III transcription of tRNAs generally uses intragenic promotors as transcription initiation elements, which are the A Box and B Box localized within the genomic tRNA sequence and the distance between these two elements is variable depending on identity of the tRNA (Figure 6) (Arimbasseri & Maraia 2016) Termination occurs upon encountering the genomic stretch of As functioning as the termination signal for RNA polymerase III, leading to incorporation of a 3’-terminal uridylate stretch in every nascent RNA polymerase III transcript (Arimbasseri et al. 2013). La proteins are the first nascent RNA polymerase III transcripts through association with the high-affinity 3’-uridylate RNA binding site (see later) (Fairley et al. 2005). Another typical RNA polymerase III transcript feature is an uncapped 5’-end (i.e. 5’-triphosphate sequence) (Yan et al. 2022) (see (Figure 3a). ', '', 'Figure 6. Schematic representation of a tRNA gene. ', 'Nascent pre-tRNA transcription is initiated through direct binding of DNA-binding factor TFIIIC (not shown) to A box and B box, followed by transcription initiation factor TFIIIB (not shown) binding to the TATA-box. This results in recruitment of RNA polymerase III, transcription elongation and termination. The RNA transcription termination signal is encoded in the genome as a stretch of adenylates. This results in stuttering of the RNA polymerase and as a result, a stretch of Us is produced at the 3’-end of each RNA polymerase III transcript. Figure adapted from (Porter et al. 2021).', '', 'tRNAs are encoded at multiple copies in the genome', 'The human genome consists of more than 400 nuclear-encoded tRNA genes and are grouped into 20 families based on the amino acid added to the post-transcriptionally 3’-terminal CCA sequence (Chan and Lowe, 2016 - http://gtrnadb.ucsc.edu GRCh37/hg19). The groups of tRNAs containing different anticodons that can decode the same amino acid are named isoacceptors. For example, the alanine amino acid (codons GCU, GCG, GCC, GCA,) can be decoded with the isoacceptors tRNAs AlaAGC, AlaCGC and AlaUGC, while the phenylalanine amino acid (codons TTC and TTT) is decoded by only one isodecoder: PheGAA (Chan and Lowe, 2016 - http://gtrnadb.ucsc.edu GRCh37/hg19). Typically, amino acids are decoded by one or more tRNA isodecoders, ranging up to five different isodecoders for arginine and leucine amino acids in humans. The approximately 400 human tRNAs genes are divided among 48 different isoacceptor families decoding the 61 sense codons into the 20 different amino acids. For example, the above mentioned tRNAs AlaAGC, AlaCGC and AlaUGC decode GCU, GCG, GCA synonymous codons as alanine, respectively. The GCC codon is also translated as alanine by one of these tRNA isoacceptors because of the degeneracy of the genetic code, particularly at the wobble position of the tRNA anticodon (position 34 in the tRNA) allowing non-cognate RNA base pairing following Watson-Crick wobble rules (Chan & Lowe 2016; Pinkard et al. 2020). ', 'Various isodecoders correspond to tRNAs that contain the same anticodon sequence but with different nucleotide composition in the body of the tRNA. For example, LeuCAG is encoded by 9 copies in the genome, however, two mature tRNA LeuCAG isodecoders exist (Chan and Lowe, 2016 -  GRCh37/hg19). Regardless of the tRNA isodecoder sequence variability, the corresponding mature tRNA is required to retain the ability to fold into the typical tRNA structure to function as a transfer molecule in translation. Certain isodecoders genes with nucleotide changes in the D-arm or T-arm could have effects on the transcription efficiency since these sequences are the RNA polymerase III promoter A- and B-box, respectively (Geslain & Pan 2010). Expression of isodecoders can vary between different tissues (Pinkard et al. 2020). ', 'Mature tRNAs adopt a classic cloverleaf secondary structure', 'The maturation process of a tRNA includes multiple steps, starting with transcription of the nascent tRNA by RNA polymerase III. Next, the nascent pre-tRNA is processed into a mature tRNA through removal of 5’-leader and 3’-trailer sequences, intron splicing, and CCA addition at the 3’-end by nucleotidyltransferase (Figure 7a) (Maraia & Lamichhane 2011). Throughout the maturation process, post-transcription modifications are added to the tRNA that enhance correct folding of the tRNA into the classic cloverleaf secondary and L-shaped tertiary structure (Helm 2006). Mature tRNA molecules are extremely stable with a half-life around 100 h (Choe & Taylor 1972). The length of mature tRNAs is approximately between 74 and 93 nucleotides, and despite overall lack of sequence conservation, generally contain four or five stems that are formed through internal base pairing, including the acceptor stem, D-stem, anticodon stem, T-stem and, depending on the tRNA, a variable stem (not present in the example) (Figure 7b) (Berg et al. 2019). Additionally, the tRNA structure contains a conserved dihydrouridine (D)-loop, anticodon stem loop and TΨC (T)-loop (Figure 7b). Formation of the four stem structures in the tRNA require sufficient Watson–Crick A-T and C-G or G-U wobble pairs between opposing nucleotides (Berg et al. 2019).', 'The discriminator base is the 3’-most terminal nucleotide and is found as a single nucleotide overhang that serves as the preceding nucleotide for addition of the CCA sequence by the nucleotidyltransferase (Figure 7b). The post-transcriptionally added CCA sequence serves as the attachment site for amino acids transferred by aminoacyl-tRNA synthetases (Pang et al. 2014). The conserved tRNA structure is not only important for interactions with translation factors, ribosomes, and mRNAs during translation, but also for interactions with the aminoacyl-tRNA synthetases that deliver the correct amino acid to corresponding tRNA isotype (Carter & Wolfenden 2015). ', '', 'Figure . Processing of nascent pre-tRNAs into mature tRNAs. ', 'a Schematic representation of a nascent pre-tRNA. RNA polymerase III transcription results in 5’- and 3’-terminal extensions shown in yellow and blue, respectively. The La protein associates with the 3’-terminal stretch of uridylates shown in dark blue. Processing of a La-bound nascent pre-tRNA involves the endonuclease RNase P for processing of the 5’-leader and endonuclease RNase Z for removal of the 3’-trailer sequence. The La-independent pathway has a different processing order (see Figure 8). Following 3’-end processing, the pre-tRNA obtains a post-transcriptionally added -CCA sequence (see later). A subset of nascent pre-tRNAs contain an intervening intron sequence (shown in orange) that requires to be processed (see later). b The mature tRNA secondary structure adopts a typical cloverleaf fold and the tertiary structure a L-shape. Different structural regions include loops and stems which are formed as a result of internal base pairing. The color code highlights the different regions of a mature tRNA, including the acceptor stem in light green, the D-arm in cyan, the anticodon stem in orange, the anticodon triplet in red, the variable loop in light green, the T-arm in cyan and discriminator base at position N73 upstream of the post-transcriptionally added -CCA sequence (shown in yellow). During the maturation process of the nascent pre-tRNA, multiple post-transcriptional modifications are obtained (see later). Figure adapted from (Porter et al. 2021).', '', 'While tRNAs are composed of a small number of RNA base building blocks (A, U, C and G) that form the basis for formation of secondary and tertiary structures through base pairing, many alternate secondary and tertiary structure with similar levels of free energy could also form. These misfolded tRNAs can become kinetically trapped in this alternate conformation leading to non-functional tRNAs. This phenomenon is described as the RNA folding problem and can be resolved by RNA chaperones that prevent tRNAs getting trapped by unfolding and refolding and also by stabilizing correct secondary and tertiary folds through binding (reviewed in Herschlag, 1995). ', 'Processing of nascent pre-tRNAs in eukaryotes', 'In order to obtain a fully functional mature aminoacyl-tRNA, multiple processing and quality control steps are required. The nascent pre-tRNA contains 3’-trailer and 5’-leader extensions and in some cases an intervening intron sequence which need to be processed accurately (see Figure 7a) (Maraia & Lamichhane 2011). In addition, post-transcriptional modifications are obtained throughout the maturation process to enhance accurate folding of the mature tRNA (Pan 2018). These pre-tRNA processing steps are described in detail in the following sections.', 'In addition, following 5’- and 3’-end maturation, eukaryotic pre-tRNAs are substrates for the ATP(CTP):tRNA nucleotidyltransferase that catalyzes the synthesis of the CCA sequence at the 3’-end of the transcript (Tomita & Yamashita 2014). Following addition of the CCA, the next step involves charging of the tRNA with the appropriate amino acid by amino acetyltransferases (Pang et al. 2014). However, these processing steps are beyond the scope of this thesis.', 'Processing of the 5’-leader by endonuclease RNase P ', 'In all three domains of life, processing of the 5’-leader is completed by the endonuclease RNase P through phosphodiester hydrolysis resulting in a 5’-monophosphate terminus (Gößringer et al. 2012). The RNase P enzyme consists of multiple proteins and the ribonuclease P RNA. The RNA subunit possesses catalytic activity is highly conserved between bacteria, Archaea and eukaryotes (Chen & Pace 1997). Generally, processing of pre-tRNAs by RNase P is the final step in 5’-end maturation. One major exception to standard pre-tRNA processing has been described in the parasitic archeon Nanoarchaeum equitans (Randau et al. 2008) in which tRNA transcription does not start upstream resulting in absence of the 5’-leader extensions. As a result, these tRNAs do not require 5’ processing by RNaseP and therefore produce mature tRNAs containing a 5’-triphosphate. So far, this is the only identified species to not encode any of the components of the RNase P complex (Randau et al. 2008).', 'Processing of the 3’-trailer by either endonuclease RNase Z or exonuclease Rex1 is determined by the La protein', 'Once the 5’-leader has been processed by RNase P, the endonuclease RNase Z proceeds to cleave the 3’-trailer to remove the extension sequence up to the discriminator base (N73, see Figure 7). This processing step results in a 3’ end-matured pre-tRNA and concomitantly, release of the La protein from the tRNA which was bound to the 3’-terminal uridylate stretch. Generally, nascent pre-tRNA processing follows these consecutive steps in a system where La is associated with the 3’-trailer of the tRNA, termed the La-dependent pre-tRNA processing pathway (Figure 8 – pathway 1). Here, the 5’-leader extension is removed from the nascent pre-tRNA by RNase P prior to processing of 3’-end extensions by RNase Z (Van Horn et al. 1997; Yoo & Wolin 1997). Interestingly, in situations where La is deleted or limiting and the 3’-trailer sequences of the nascent pre-tRNA remain unprotected, the 3’ to 5’ RNA exonuclease-1 (Rex1) rapidly processes the 3’-trailer sequence prior to 5’-trailer cleavage by RNase P (Figure 8 – pathway 2) (Copela et al. 2008). This process is termed the La-independent pre-tRNA processing pathway.', '', 'Figure legend on next page', 'Figure . The La protein assists in processing of nascent pre-tRNAs. ', 'RNA polymerase III transcribes pre-tRNAs and produces a stretch of 3’-uridylates at the end of each nascent transcript as a result of the dAs stretch encoded in the genome that functions as a transcription termination signal. The La protein binds the 3’-UUU-OH sequence provided that the stretch of uridylates is of a sufficient length. Normal pre-tRNAs are processed either by a La-dependent (# 1) or La-independent (#2) pathway resulting in end-processed tRNAs. The main difference between normal pre-tRNA processing in presence or absence of La is the order of 5’- and 3’-end processing and 3’-end processing by an endo- or exonuclease. The endonuclease RNase Z functions in the La-dependent pathway, while the exonuclease Rex1 functions in the La-independent pathway because there is no 3’-end protection of nascent pre-tRNA by La binding. In a La-independent pathway (#4), structurally impaired pre-tRNAs are targeted for degradation by the nuclear surveillance: the misfolded pre-tRNAs are polyadenylated by the TRAMP complex followed by exosome degradation. The presence of a functional La protein enhances pre-tRNA folding and processing (#3), resulting in end-processed tRNAs. Figure adapted from (Maraia & Lamichhane 2011).', '', 'As mentioned before, transcription termination by RNA polymerase III occurs after encountering a stretch of dA’s in the template DNA strand resulting in production of a 3’-terminal stretch of uridylate RNA (Figure 8). The minimal number of encoded dA’s for efficient RNA polymerase III transcription termination varies between eukaryotes, for example H. sapiens requires only 4 dAs, while S. pombe requires at least 5 dAs and S. cerevisiae 6 dAs or more (Arimbasseri et al. 2013). Typically, La proteins require between 3-6 uridylates for high affinity binding depending on the RNA polymerase III termination stretch in different species with 3 uridylates for human La (Mathews & Francoeur 1984; Stefano 1984) and 4-5 uridylates for S. pombe Sla1p (Huang et al. 2005). The number of encoded dAs correlates with the minimal number of uridylates on the nascent pre-tRNA for high affinity La protein binding. After RNA polymerase III transcription, a heterogenous population of pre-tRNAs are present in the nucleus with varying 3’-uridylate tail lengths as a result of varying termination signal length, i.e. the number of dAs encoded in the genome and stuttering of the polymerase upon encountering the dA termination signal. Since La proteins have a higher affinity for longer 3’-uridylate tails, nascent pre-tRNAs with a relatively shorter uridylate stretch will remain unbound when the La protein is limiting (Stefano 1984). As a result of this preferential binding, both the La-dependent and La-independent nascent pre-tRNA processing pathway can take place in simultaneously (Figure 8 – pathways 1 and 2) and both result in the formation of a mature tRNA (Maraia & Lamichhane 2011). ', 'Nascent pre-tRNAs require an appropriate fold in order to be processed by the endonucleolytic tRNA processing enzymes that remove the 5’-leader and 3’-trailer. Folding of nascent pre-tRNAs takes place co-transcriptionally (Leamy et al. 2019), however, multiple different conformations could be obtained and occasionally leading to misfolded pre-tRNAs trapped in a misfolded structure (Vakiloroayaei et al. 2017). These transcripts are targeted for degradation by the nuclear surveillance (Figure 8 – pathways 4), but can be rescued by the RNA chaperone activity from La that functions in correct folding of pre-tRNAs (Figure 8 – pathways 3) (discussed in more detail in a later section) (Chakshusmathi et al. 2003). Another important function of the La protein is nuclear retention of tRNAs and other RNA polymerase III transcripts. The La protein contains a nuclear localization signal (NLS) and retains the transcript in the nucleus through direct interactions avoiding premature export into the cytoplasm of incomplete or incorrectly processed transcripts (Intine et al. 2002; Bayfield et al. 2007).', 'In conclusion, the presence of the La protein on the 3’-terminal uridylates on a nascent pre-tRNA determines the usage of the 3’-end exonucleolytic or endonucleolytic maturation process, with accessibility of the 3’-end inhibited by La limiting access of the 3’-exonuclease Rex1 to the 3’-end and therefore 3’-maturation takes place through the endonucleolytic RNase Z pathway. In contrast, absence of the La protein at the 3-terminus of the nascent pre-tRNA allows 3’-exonucleases to process the 3’-trailer sequence. Both the exonucleolytic and endonucleolytic pre-tRNA processing pathway results in functional mature tRNAs, indicating that these enzymes function redundantly in processing of pre-tRNA 3’-extensions (Yoo & Wolin 1997). The Lhp1 protein in Saccharomyces cerevisiae is not essential and in Lhp1 deletion strains, the 3’-trailer sequences of nascent pre-tRNAs are processed by the 3’-exonuclease pathway (Yoo & Wolin 1994) since the RNase Z endonuclease processing pathway is dependent on presence of Lhp1 on the 3’-uridylate extension (Yoo & Wolin 1997). ', 'Post-transcriptional modifications', 'Post-transcriptional modification involves the enzymatic covalent attachment of a chemical to these standard RNA bases to alter their properties in multiple ways (ref). As part of the tRNA maturation process, tRNAs obtain post-transcriptional modifications at different steps during processing (ref). Modified nucleotides have been found on La-associated pre-tRNAs, indicating that post-transcriptional modifications also occur in early steps of pre-tRNA processing (Hendrick et al. 1981). Nuclear-encoded tRNAs have the highest density of post-transcriptional modifications with an average of 13 modifications per molecule, while mt-tRNAs only contain 5 modifications on average (Pan 2018). tRNAs also contain the most diverse nucleotide modifications with more than 90 different types of modifications identified across archaea, bacteria, and eukaryotes (Cantara et al. 2011). Post-transcriptional modifications of tRNAs have several functions, including stability of the tRNA structure and appropriate decoding of the mRNA (Lorenz et al. 2017; El Yacoubi et al. 2012). ', 'While the presence of modification in tRNAs is advantageous for folding and stability of the tRNA, reverse transcriptase-based sequencing methods are severely hindered by the stable and extensive secondary structure and the presence of many post-transcriptional modifications in tRNAs. Multiple methods have been developed to circumvent the block generated by modifications for cDNA extension by reverse transcriptase. First, AlkB-facilitated RNA methylation sequencing (ARM-seq) utilizes an E. coli AlkB enzyme which removes methyl groups from tRNA to avoid generation of truncated cDNA (Cozen et al. 2015; Zheng et al. 2015). Second, a thermostable group II intron reverse transcriptase (TGIRT) possesses high processivity during cDNA generation from highly stable RNA structures and thereby enhances production of full-length cDNA transcripts (Zheng et al. 2015). ', 'Splicing of intron-containing tRNAs', 'A subset of tRNAs contain intervening intron sequences that must be spliced before the tRNA can be a substrate for amino acid charging and therefore function as an adapter molecule in translation. The intron is typically found one nucleotide 3’-terminal to the anticodon in yeasts and vertebrates and typically contains a sequence that is complementary to the anticodon and base pairing between the intron and anticodon determines the secondary structure of the pre-tRNA (Johnson & Abelson 1983) (see Figure 7). Correct intron splicing is important to produce a full set of isoacceptors that are required to provide the translational machinery with all 20 amino acids required for polypeptide synthesis. Importantly, tRNAs retaining the intervening intron sequences are incapable of adopting an appropriate mature tRNA fold and are therefore defective in translation (ref). ', 'La proteins assist in correct folding and processing of nascent pre-tRNA through two independent binding modes', 'The La protein, also known as Sjogren’s Syndrome antigen B (SSB), was first identified as an autoantigen in patients suffering from systemic lupus erythematosus and Sjogren’s syndrome (Mattioli & Reichlin 1974a; Alspaugh & Tan 1975). Patient antibodies were used for immunoprecipitation and revealed that La is an RNA-binding protein (Hendrick et al. 1981; Lerner et al. 1981) that in particular interacts with nascent polymerase III transcripts still containing the typical 3’-terminal uridylate sequence (encoded in the genome as an adenylate stretch at the end of the RNA polymerase III gene) (see earlier) (Stefano 1984; Mathews & Francoeur 1984). Homologues of the La protein have been identified in many eukaryotes, ranging from yeasts (Yoo & Wolin 1994; Van Horn et al. 1997) to metazoans (Park et al. 2006; Arhin, Shen, Pérez, et al. 2005; Scherly et al. 1993; Pellizzoni et al. 1996) and from insects (Yoo & Wolin 1994; Pardigon & Strauss 1996; Bai & Tolias 2000) to higher plants (Fleurdépine et al. 2007), where La is essential in metazoans and higher plants but dispensable in yeast (Yoo & Wolin 1994; Fleurdépine et al. 2007; Van Horn et al. 1997; Park et al. 2006). ', 'The best conserved sequence of the genuine La protein is found in the N-terminal domain, whereas the C-terminal domain is more variable between species. The conserved N-terminal domain contains the La module, an RNA-binding unit composed of two juxtaposed RNA-binding domains, the La motif (LaM) and an RNA recognition motif (RRM) separated by a short linker (see Figure 9 – top panel) (Teplova et al. 2006). In higher eukaryotes, the C-terminal domain of the La protein contains a second RRM (RRM2), a disordered region and a nuclear localization signal (NLS) (Figure 9) (Jacks et al. 2003). ', '', 'Figure legend on next page', 'Figure . Schematic representation of conservation of the La module (LaM + RRM1) in genuine La and La-related protein (LARP) family. ', \"Domain organization of the genuine La proteins from Homo sapiens, Dictyostelium discoideum, Trypanosoma brucei, Arabidopsis thaliana, Schizosaccharomyces pombe (Sla1p), Saccharomyces cerevisiae (Lhp1) and Tetrahymena thermophila (Mlp1) compared to LARP7 proteins from Homo sapiens and Tetrahymena thermophila (p65) and other La-related proteins from Homo sapiens (LARP1, LARP4 and LARP6). The La module is found in the N-terminal region and contains the LaM (shown in black) and the RNA-recognition motif-1 (RRM1) (shown in light grey). A second RRM is found in the C-terminal region of genuine La proteins in higher eukaryotes and LARP7 proteins (shown in dark grey). LARP-1, -4, and -6 have obtained additional domains with specialized functions, including DM15: domain for binding of the 5' terminal oligopyrimidine (TOP) containing mRNAs; PAM2: PABP-interacting motif-2; PBM: PABP binding motif; RIR: RACK1- interaction region; LSA: LaM and S1-associated motif. \", 'Uridylate-specific binding of La results in 3’-end protection', 'For sequence-specific binding of the 3’-terminal uridylate sequence both RNA-binding domains in the La module (LaM + RRM1) are involved in formation of a binding pocket at the interdomain interface around these uridylates, without using the canonical nucleic acid interaction surfaces of the La module: the winged helix-face of the LaM and the antiparallel β-sheet surface on RRM1 (Figure 10a) (Teplova et al. 2006; Kotik-Kogan et al. 2008). Functional studies have demonstrated that the La module is indispensable and sufficient for binding (Goodier et al. 1997; Ohndorf et al. 2001), and structural studies confirm these findings and reveal that the La module forms a hydrophobic binding pocket at the interdomain interface of the LaM and RRM1 around the uridylate sequence (Figure 10a) (Teplova et al. 2006; Kotik-Kogan et al. 2008). Important residues lining the hydrophobic pocket that interact with the uridylate RNA using their side chains are Q20, Y23, Y24, D33, F35, and F55 and two additional LaM residues, N56 and R57, make contact with the uridylate RNA only by using their backbone structure (Figure 10b – U1 and U2 interactions). In contrast to the many interactions made by the LaM, RRM1 only makes a single contact with the uridylate RNA using the backbone of I140 (Figure 10b – U2 interactions). In addition to contacts between RNA and amino acid residues, some interdomain interactions between the LaM and RRM1 have also been observed Figure 10b – LaM-RRM1 interactions). Through binding of the La module (LaM + RRM1) surrounding the 3’-uridylate RNA, the 3’-end of the transcript is protected from 3’-exonucleases (Yoo & Wolin 1997). ', '', '', '', 'Figure legend on next page', 'Figure . The human La protein binds 3’-uridylate RNA through cooperative binding between the LaM and RRM1.', 'a Structure of the human La protein in complex with 5’-AUAUUUU-3’ (PDB: 2VOD). For clarity, only three uridylates at the 3’-end are shown and a negative number system was adopted with the most 3’-terminal uridylate named U-1. The penultimate uridylate (U-2) is found to be pointing towards the inside of the binding pocket and the 3’-terminal uridylate (U-1) is pointing more towards the outside in a stacking formation with U-3. Secondary structures are shown as ribbons with β-sheets colored in pink, the canonical winged-helix-fold and RRM domain α-helices shown in dark blue and La protein-specific α-helices show in light blue. RNA is shown as a stick model with carbon atoms colored in yellow. Other atoms are colored by type: oxygen in red, nitrogen in blue and phosphorous in orange. b Magnified views of La-U-1 RNA interactions: amino acids important for binding of the 3’-terminal uridylate U-1 are located exclusively in the LaM and interactions are strictly made to sugar-phosphate backbone features on the nucleotide (except for stacking interactions of the pyrimidine ring and the ribose with F35 and F55, respectively). D33 specifically interacts with 2’-OH and 3’-OH groups. Hydrogen bonds are formed with the phosphate group of U-1 by the sidechain of Y24 and both N56 and R57 use the amino acid backbone structures. Magnified view of La-U-2 RNA interactions: amino acids important for binding of U-2 are found in both the LaM and RRM1 and hydrogen-bonds are made with the nucleotide base of U-2. Q20 makes a hydrogen bond with U-2 using its sidechain and I140 contacts U-2 with its backbone. The pyrimidine ring of U-2 is also stacked directly on Y23. Magnified view of interdomain interactions between the LaM and RRM1: interactions between amino acids are strictly formed with amino-acid side chains. An interdomain contact exists between N139 in the RRM1 and Y23 in the LaM. Another interdomain interaction exists between RRM1 residue D125 and LaM residue R57, which in turn also interacts with Q20 through an intradomain interaction.', '', '\\n', 'RNA chaperone activity of La assist in appropriate tRNA folding and maturation', 'The function of many non-coding RNA molecules, including tRNAs, often depends on correct folding into their precise functional conformations. RNA chaperones are proteins that can facilitate formation of functional conformations by resolving misfolds through unfolding and folding of an RNA by direct interactions (Lorsch 2002). The La protein is an RNA chaperone that in addition to protecting the 3’-uridylate ends of pre-tRNAs from exonucleases, can also assist in RNA folding (Bayfield & Maraia 2009; Hussain et al. 2013; Naeeni, Contes, et al. 2012). The typical RNA-binding sequences in the human La protein RRM1 β-sheet have been found to make uridylate-independent contacts to the main body of pre-tRNAs and also possess RNA chaperone activity by assisting with tRNA folding and maturation (Bayfield & Maraia 2009). Another contact between the hLa protein and pre-tRNA is made though interaction between the short basic motif (SBM) (localized in the C-terminal part) and the 5’-triphosphate end of nascent pre-tRNA (Fan et al. 1998). The combined interaction of the hLa protein with the uridylate sequence on the 3’-end of nascent pre-tRNAs and additional binding to the body and 5’-end of the pre-tRNA helps the hLa protein identify precursor tRNAs rather than mature tRNA (Bayfield & Maraia 2009). A proposed function of the 3’ uridylate dependent binding mode is to initiate binding via this sequence specific stretch of uridylates and thereby to recruit La-associated RNA chaperone function to these transcripts. ', 'For example, human La has been characterized as an RNA chaperone through an in vitro assay where La assists in folding of a misfolded, self-splicing intron into a functional form (Belisova et al. 2005; Bayfield & Maraia 2009). Many in vivo studies have been completed in yeasts because the La protein is non-essential in these systems. However, deletion of the La protein in both the budding yeast Schizosaccharomyces cerevisiae (Lhp1) (Yoo & Wolin 1994) and fission yeast Schizosaccharomyces pombe (Sla1) (Van Horn et al. 1997) has a lethal phenotype in combination with either loss of function of certain tRNA modification enzymes (Copela et al. 2006) or with a mutation in the anticodon stem of the essential tRNA SerCGA, the only tRNA isoacceptor for UCG codons in Schizosaccharomyces cerevisiae (Yoo & Wolin 1997). The lethal phenotype could be rescued by a compensatory mutation that restored base pairing in the anticodon stem, indicating that Lhp1 is essential after introduction of mutations that would cause misfolding of the pre-tRNA (Yoo & Wolin 1997). ', 'These observations provide the basis for the tRNA mediated suppression assay in S. pombe where RNA chaperone function of La proteins from different species can be tested. In this red/white assay, La proteins can be transformed into S. pombe ySH9 strains lacking its genuine La homolog (sla1-∆) and containing a suppressor tRNASerUCA and the ade6-704 mRNA (Figure 11). When the suppressor tRNASerUCA is functional, as a result of appropriate folding and maturation by RNA chaperone function of transformed La, translation of the full-length ADE6 enzyme can take place and as a result leads to alleviation of the accumulation of the red intermediate. In other words, yeast colonies have a white color in the presence of functional La chaperone activity and a red color in the absence thereof. The tRNA mediated suppression assay has been used previously with La proteins from different species and RNA chaperone activity has been established for all La proteins analysed (Porat & Bayfield 2020). ', '', 'Figure . tRNA-mediated suppression assay to study RNA chaperone function. ', 'a Representation of the suppressor tRNASerUCA with lower case letters indicating nucleotides that are going to be removed during pre-tRNA processing (5’- and 3’-end maturation and intron removal). Mutated nucleotides compared to the wild type tRNASerUGA are indicated in red. The anticodon mutation enables the tRNA to decode a UGA stop codon by inserting a serine into the growing peptide chain, but it also results in a mismatch between the anticodon and intron sequence resulting in misfolding of the pre-tRNA and as a result degradation. Another mutation in the body of the pre-tRNA can be introduced to increase the dependency on RNA chaperone activity for correct folding of the suppressor pre-tRNA. b The ade6-704 mRNA contains an early stop codon UGA that can be suppressed by a functional suppressor tRNASerUCA to produce the full-length ADE6 enzyme that has a role in adenine biosynthesis. In the presence of an RNA chaperone (in this example the La protein was transformed into S. pombe ySH9) the suppressor tRNASerUCA is functional and a full-length ADE6 enzyme is produced. As a result, a red intermediate metabolite (formed in the adenine synthesis pathway) can be converted into colorless products. In contrast, when RNA chaperone function is absent, the suppressor tRNA is not functional and no ADE6 enzyme is produced leading to accumulation of a red intermediate metabolite. Figure adapted from (Porat & Bayfield 2020) (a) and figure generated using BioRender (b).', 'La and La-related proteins (LARPs) conserved structural arrangements', 'The La protein is a member of the La-related proteins (LARPs), a superfamily of eukaryotic RNA-binding proteins that contain five different families: LARP1, LARP3 (genuine La), LARP4, LARP6 and LARP7 (Bousquet-Antonelli & Deragon 2009). A common feature of the LARP family is the La module in the N-terminal part of the proteins consisting of two RNA-binding domains: the LaM and the RNA recognition motif (RRM) (see Figure 9) (Alfano et al. 2004; Kenan & Keene 2004; Dong et al. 2004). While LARPs typically have a high degree of homology with the genuine La protein in the LaM-RRM arrangement, many LARPs have adopted additional functions by sequence and structure variations within the La module and by acquiring additional domains with specialized functions (Dock-Bregeon et al. 2019). Within the LARP group, LARP7 proteins are most closely related to La proteins when comparing sequence identity and function. However, in contrast to human La that binds all nascent RNA polymerase III transcripts on their 3’-terminal uridylate tail, human LARP7 interacts almost exclusively with the non-coding RNA 7SK (Krueger et al. 2008; He et al. 2008). The 7SK RNA is an RNA polymerase III transcript and is initially bound by the genuine La protein (Chambers et al. 1983). During 7SK RNP assembly, the La protein is replaced by LARP7 which recognizes the 3’-terminal UUU-3’OH motif of 7SK by formation of a hydrophobic binding pocket around the uridylate sequence using the LaM and RRM1 (Figure 12b, left) (He et al. 2008; Muniz et al. 2013; Uchikawa et al. 2015), very similar to the binding pocket formed by La upon binding of uridylate RNAs (Figure 12a, left) (Kotik-Kogan et al. 2008; Teplova et al. 2006). Immunoprecipitation of the La protein co-purifies the methylphosphate capping enzyme (MePCE) responsible for addition of the m3G-cap on 7SK and U6 snRNA (Muniz et al. 2013). Transcripts containing the modified m3G-cap instead of the nascent RNA polymerase III 5’-triphosphate cap have lower binding affinity to the human La protein indicating that the post-transcriptional modification of the cap is required for replacement of the La protein (Bhattacharya et al. 2018). ', 'In addition to 3’-uridylate interactions, LARP7 also makes specific contacts with the 3’ hairpin of 7SK by using the α3 helix on RRM2 (Figure 12b, right) (Eichhorn et al. 2016). In T. thermophila, the LARP7 ortholog p65 was until recently the only characterized LARP in this species (Bousquet-Antonelli & Deragon 2009; Deragon 2020). p65 can interact with telomerase RNA in a similar way as human LARP7 interactions with 7SK RNA: the disordered region C-terminal of RRM2 α3 forms an α-helical extension upon binding of stem IV of telomerase RNA (Figure 12c). Telomerase RNA is transcribed by RNA polymerase III in this species and the La module has also been proposed to interact with the 3’-UUU-OH motif of telomerase RNA similarly as human La and LARP7 (Singh et al. 2012; Cash & Feigon 2017; Prathapam et al. 2005; O’Connor & Collins 2006; Bayfield et al. 2010). The related p43 protein found in the ciliated protozoa Euplotes is similarly critical for the assembly, nuclear retention and functionality of the telomerase complex (Aigner & Cech 2004; Aigner et al. 2003; Möllenbeck et al. 2003). The LARP7 ortholog in fission yeast, Pof8, has also been described as a constitutive component of the telomerase complex (Páez-Moscoso et al. 2018). In conclusion, both structural arrangement and different RNA binding modes are similar between p65, LARP7 and La. ', '', '', 'Figure legend on next page', '', 'Figure . Structural conservation between different member of the La-related protein family. ', 'a Structures of the three different RNA-binding domains from the genuine La protein in H. sapiens. The LaM and RRM1 bind uridylate RNA inside the hydrophobic pocket formed between the two domains (PDB: 2VOD) and the structure for RRM2 (PDB: 1OWX). b Structures of the three different RNA-binding domains from the LARP7 protein in H. sapiens. Similar binding of uridylate RNA by the LaM and RRM1 forming a hydrophobic pocket around the RNA (PDB: 4WKR) and the RRM2 binding the 3’-terminal hairpin of the non-coding 7SK RNA (PDB: 6D12). c Structures of the RRM2 from p65 in T. thermophila. The C-terminal disordered region of α3 forms a helical extension upon binding of the non-coding telomerase RNA (PDB: 4EYT, 4ERD). ', 'A novel type of La protein identified in alveolates lacking an RRM domain and containing a C-terminal DUF3223', 'A recent study has identified another protein from the LARP superfamily in T. thermophila and based on the primary sequence conservation of the LaM it has been grouped with the genuine La proteins (Deragon 2020) and will be referred to as Macronucleus localized protein of unknown function (Mlp1) (Arthur et al. 2017). Interestingly, Mlp1 only contains a highly conserved LaM, unlike previously identified La proteins in other eukaryotes containing the typical LaM-RRM arrangement. This atypical La protein has been identified in several members of the alveolates and is the first eukaryotic group in which a genuine La protein has been described that contains a LaM without an adjacent RRM1 (type IV) (Figure 13) (Deragon 2020). While this is very unusual for genuine La proteins, some proteins in the LARP superfamily are known to only contain a LaM. For example, LARP1 orthologs in the fungi S. pombe (Slr1p) and S. cerevisiae (Sro9p and Slf1p) are RNA-binding proteins that contain a LaM without an adjacent RRM (Sobel & Wolin 1999; Bousquet-Antonelli & Deragon 2009). More recent, the human LARP1 was also described to contain a conserved LaM without presence of the adjacent RRM1 (Kozlov et al. 2022). However, the appearance of LARPs without an RRM immediately following the LaM is rather uncommon (Bousquet-Antonelli & Deragon 2009). The absence of the RRM1 in alveolates genuine La protein is highly unexpected because in La proteins described in other eukaryotes, both RNA-binding domains in the La module (LaM + RRM1) are required for high-affinity binding of the typical uridylate RNA (see Figure 10). In other words, deletion of either the LaM or RRM1 results in loss of binding of uridylate RNA (Dong et al. 2004; Goodier et al. 1997). Another unusual feature of this new type of La proteins is the presence of a domain of unknown function-3223 (DUF3223) in the C-terminal region. The DUF3223 has also been identified in the C-terminal region of plant-specific RNA polymerases IV/V subunits (NRPD1 and NRPE1) (Wang & Ma 2015) and in plant proteins involved in rRNA processing (DOMINO1, DeCL and DeCL-like) (Lahmy et al. 2004; Bellaoui et al. 2003). The DUF3223 is also found many in bacteria and protists but is not found is in animals and fungi (Wang & Ma 2015). ', '', '', '\\n', '\\t', '', '', 'Figure legend on next page', 'Figure . Structural organization of the La protein in eukaryotes. ', 'a The different types of genuine La proteins present in eukaryotic species. LaM: La motif, RRM1: RNA recognition motif-1, RRM2: RNA recognition motif-2, DUF3223: domain of unknown function-3223, GK/R: glycine, lysine/arginine-rich motif. Figure adapted from (Deragon 2020). b Overview of La proteins typically found in eukaryotic lineages. Numbering is used to indicate the number of genuine La proteins encoded in the genome. c Phylogenetic tree constructed using full-length La protein sequences (except for Alveolates, where the LaM alone was used). La proteins corresponding to different eukaryotic groups are color-coded corresponding to the legend on the right.', ' ', 'Statement of purpose', 'The discovery that formed the basis of this Ph.D. thesis was the identification of a novel LaM-containing and RRM1-lacking La protein in alveolates that challenges the expectation of the structural composition of genuine La proteins. The well-described 3’-terminal uridylate high-affinity La protein binding site was though to be cooperatively bound through formation of a UUU-3’OH specific binding pocket between both RNA-binding domains found in the La module, the LaM and RRM1. We posed the question if a La protein that is structurally missing an important conserved RNA binding domain could still function as a genuine La protein in pre-tRNA binding and processing. We confirmed that Mlp1 indeed functions as a genuine La protein by demonstrating preferential binding of pre-tRNAs compared to mature tRNAs, uridylate-specific binding through highly conserved residues located in the LaM and RNA chaperone activity promoting processing and maturation of misfolded pre-tRNAs using pre-tRNA mediated suppression assay. In addition, we studied tRNA processing in the yeast S. pombe and in T. thermophila and discovered that Mlp1 is linked to alternate tRNA processing compared to other eukaryotes. We also described that the architecture of pre-tRNAs in T. thermophila is variable from other eukaryotes, in that the 3’-trailer sequences are absent and the mature tRNA sequence immediately continues into the 3’-uridylate stretch. The discovery of an altered pre-tRNA processing pathway in the highly studied eukaryote T. thermophila, driven by a LaM-containing and RRM1-lacking La protein, is an interesting contribution to both the pre-tRNA processing and the La field. ', 'In addition to lacking the RRM1 RNA-binding domain, Mlp1 also contain a C-terminal DUF3223. We demonstrated that this domain possesses RNA chaperone activity by tRNA mediated suppression assay. We considered that this La protein containing two addition domains, the middle domain and C-terminal DUF3223, could have addition functions. We further investigated the function of this variant La protein by identifying other non-coding RNA binding partners and found that Mlp1 interacts with mature snRNAs. In addition, we identified direct protein interactions with a protein component of the functionally active snRNPs indicating a function for Mlp1 in splicing. We confirmed that Mlp1 indeed affects the efficiency of splicing of introns from pre-mRNAs. This is the first study describing the involvement of a genuine La protein in pre-mRNA splicing. ', 'We also found that Mlp1 interacts with mature snoRNAs and protein components of the snoRNP complex. We investigated if Mlp1 would function in snoRNP assembly, however, we were not able to assign a function for Mlp1 in assembly. We also studied the effect of Mlp1 depletion on post-transcriptional modifications in rRNA without observing a difference. However, we described that Mlp1 interacts with all previously described snoRNAs and we have used this metric to identify novel snoRNAs in T. thermophila. This could elucidate the snoRNA field in T. thermophila further.   ', '', '', '', '', '', '', '', 'Chapter II: \\nAltered tRNA processing is linked to a distinct and unusual La protein in Tetrahymena thermophila', '\\n', 'Author contributions', '', 'Altered tRNA processing is linked to a distinct and unusual La protein in Tetrahymena thermophila \\nKerkhofs, Kyra1; Garg, Jyoti1; Fafard-Couture, Étienne2; Abou Elela, Sherif3; Scott, Michelle2; Pearlman, Ronald E.1; Bayfield, Mark Allan1', \"1 Department of Biology, Faculty of Science, York University, Toronto, Ontario N3J 1P3, Canada, \\n2 Département de Biochimie et de Génomique Fonctionnelle (LGDP), Faculté de Médecine et des Sciences de la Santé, Université de Sherbrooke, Sherbrooke, Québec, J1E 4\\u2009K8, Canada,\\xa0\\n3 Département de Microbiologie et d'Infectiologie, Faculté de Médecine et des Sciences de la Santé, Université de Sherbrooke, Sherbrooke, Québec, J1E 4\\u2009K8, Canada.\\xa0\", 'Manuscript accepted in', 'Nature Communications ', '2022 October 10th, ', 'Author contributions:', 'Jyoti Garg transformed, selected, and characterized the partial Mlp1 knockout (KO) strain in Tetrahymena thermophila by Southern blotting in Supplementary Figure 7b (original manuscript: Supplementary Figure S7b) and performed previous versions of immunofluorescent staining in Supplementary Figure 7c (original manuscript: Supplementary Figure S7c). Étienne Fafard-Couture performed TGIRT-sequencing analysis generating transcripts per million (TPM) counts and generated a custom python script to obtain counts per million (CPM) for pre-tRNAs and mature tRNAs. This data was used to generate Figure 15b (original manuscript: Figure 2b) and Figure 18c (original manuscript: Figure 5c).', 'Abstract', 'Nascent pre-tRNAs are transcribed by RNA polymerase III and immediately bound by La proteins on the UUU-3’OH sequence, using a tandem arrangement of the La motif and an adjacent RNA recognition motif-1 (RRM1), resulting in protection from 3’-exonucleases and promotion of pre-tRNA folding. The Tetrahymena thermophila protein Mlp1 has been classified as a genuine La protein, despite the predicted absence of the RRM1. We found that Mlp1 functions as a La protein through binding of pre-tRNAs and affecting processing in Tetrahymena thermophila and when expressed in fission yeast. However, unlike in other examined eukaryotes, depletion of Mlp1 results in 3’-trailer stabilization. We also observed that 3’-trailers in Tetrahymena thermophila are uniquely short relative to other examined eukaryotes, and that 5’-leaders have evolved to disfavour pre-tRNA leader/trailer pairing. Our data indicate that this variant Mlp1 architecture is linked to an altered, novel mechanism of tRNA processing in Tetrahymena thermophila.', '', '\\n', 'Introduction', 'RNA polymerase III transcription of nascent pre-tRNAs terminates after the synthesis of a stretch of uridylates on a 3’-trailer extension (UUU-3’OH). La is the first protein to bind these nascent pre-tRNAs on the uridylate stretch (Fairley et al. 2005) and assists with pre-tRNA folding (Chakshusmathi et al. 2003; Copela et al. 2006; Bayfield & Maraia 2009) in a manner that can protect pre-tRNAs from 3’- to 5’exonuclease digestion (Copela et al. 2008) and rescue defective pre-tRNAs from nuclear surveillance (Huang et al. 2006). Once the nascent pre-tRNA has acquired a tRNA-like structure, the endonuclease RNase P removes the 5’-leader of La-bound pre-tRNA, followed by endonucleolytic cleavage of the 3’-trailer by RNase Z (Van Horn et al. 1997; Yoo & Wolin 1997). As a result of 3’-end cleavage, the La protein no longer associates with the tRNA and can be recycled for processing of new nascent pre-tRNAs (Bayfield & Maraia 2009). In addition to the La-dependent pathway, an alternative La-independent pre-tRNA processing pathway exists but the order of pre-tRNA processing is reversed: without 3’-trailer protection by La, the exonuclease Rex1 digests the 3’-trailer of the pre-tRNA rapidly before RNase P endonucleolytic cleavage of the 5’-leader (Copela et al. 2008), and misfolded pre-tRNAs are also more prone to degradation through nuclear surveillance (Huang et al. 2006). La binding to pre-tRNAs is therefore hypothesized to establish and determine the order of 5’-leader versus 3’-trailer processing (Yoo & Wolin 1997). Once the 5’-leader and 3’-trailer sequences are processed, tRNA nucleotidyltransferase adds a CCA sequence to the discriminator base at the 3’-end of the mature tRNA which will serve as the site of amino acid charging (Tomita & Yamashita 2014).', 'Genuine La proteins are members of the La-related proteins (LARPs) (Bousquet-Antonelli & Deragon 2009). Most LARP family members, and all genuine La proteins, contain an N-terminal La module consisting of two adjacent RNA-binding domains: a La motif (LaM) and an RNA recognition motif-1 (RRM1) (Figure 14a) (Alfano et al. 2004; Dong et al. 2004; Kenan & Keene 2004). In Tetrahymena thermophila, the LARP7 ortholog p65 was until recently the only characterized LARP in this species (Bousquet-Antonelli & Deragon 2009; Deragon 2020), but a recent study has grouped the T. thermophila Macronucleus localized protein of unknown function (Mlp1) (Arthur et al. 2017) with the genuine La proteins, based on its primary sequence conservation in the LaM (Deragon 2020). Interestingly, Mlp1 only contains a highly conserved LaM, unlike all previously studied La proteins which contain the tandem LaM-RRM arrangement. This atypical La protein has been identified in several members of the alveolates (Deragon 2020). Utilizing the interesting and important model organism, the alveolate ciliate protozoan\\xa0T. thermophila\\xa0(reviewed in (Orias et al. 2011)), the macronucleus/somatic nucleus localization and structural characteristics of TtMlp1 could provide insight into the observation that despite the apparent lack of the RRM1, Mlp1 functions as a genuine La protein in tRNA processing in the nucleus.', '', '', '', '', '', 'Figure legend on next page', 'Figure . Alveolate La proteins are predicted to have a La motif (LaM) but not an RNA recognition motif-1 (RRM1).', 'a Schematic representation of RNA-binding domains found in La and La-related protein-7 (LARP7) from different eukaryotes. The LaM and RRM together form the La module responsible for uridylate binding through formation of a hydrophobic binding pocket between the two domains. LaM: La motif, RRM: RNA-recognition motif, DUF: Domain of Unknown Function. b Primary amino acid sequence alignments from different eukaryotic lineages showing conservation of uridylate binding residues (highlighted, bottom). A dark grey background indicates identical residues, light grey conserved residues and white indicates no conservation. Color coded legend is shown in a. Full LaM and RRM1 domains are shown in Supplementary Figure 1a,b. c Secondary structure predictions of LaM and RRM1 from different eukaryotic lineages. Predicted β-sheets shown in red and predicted α-helices in blue (dark blue: typical α-helices found in the winged-helix fold and classic RRM, light blue: inserted α-helices found in La proteins specifically) are compared against the secondary structure motif of the hLa protein crystal structure on the top (PDB: 2VOD). Location of the conserved amino acid residues important for uridylate binding are shown at the bottom. Color coded legend is shown in a.', '', 'The absence of the RRM1 in the La module of a genuine La protein in alveolates is highly unexpected, due to the mechanism by which La proteins bind the uridylate containing 3’-trailer. Structural studies of the human La (hLa) LaM-RRM bound to UUU-3’OH revealed that both domains sandwich this RNA, and functional studies have demonstrated that the La module is indispensable and sufficient for uridylate binding (Alfano et al. 2004; Ohndorf et al. 2001), as deletion of either the LaM or RRM1 results in complete loss of binding (Alfano et al. 2004; Dong et al. 2004; Kotik-Kogan et al. 2008; Teplova et al. 2006). Another unusual feature of this new type of La protein is the presence of a domain of unknown function-3223 (DUF3223) in the C-terminal region (Deragon 2020). Thus, questions remain as to whether Mlp1 functions as a genuine La protein, and if so, whether it uses a mode of RNA binding dissimilar to previously studied La proteins, with possible associated changes in how Mlp1 directs pre-tRNA processing.', 'Here, we present evidence that despite the apparent lack of the RRM1, Mlp1 functions as a genuine La protein. Using ribonucleoprotein (RNP) immunoprecipitation (RIP)-Seq of Mlp1, we show association with UUU-3’OH containing pre-tRNAs in vivo, and preferential binding of pre-tRNA substrates over mature tRNA substrates in vitro. In order to assess whether this altered architecture might have associated consequences in pre-tRNA processing, we tested Mlp1 function in a well-established model system and demonstrated that heterologous Mlp1 expression in Schizosaccharomyces pombe promotes pre-tRNA processing and tRNA mediated suppression, but without typical La-associated 3’-end protection. Consistent with this, genetic depletion of Mlp1 in T. thermophila reveals that in contrast to previously studied La proteins, Mlp1 destabilizes pre-tRNA 3’-ends, thus acting as a factor that accelerates 3’-end processing. In addition, when comparing pre-tRNAs found in T. thermophila with other eukaryotic species, we found that the 3’-trailer sequences are considerably shorter, and that 5’-leader sequences have evolved accordingly to disfavor base pairing at these shortened 3’-trailers. Our data are consistent with a model in which Mlp1 fulfills many expected functions of a genuine La protein but uses alternate RNA binding modes to promote a pre-tRNA processing pathway in T. thermophila that differs from those found in other studied eukaryotic systems.', '\\n', 'Results', 'Mlp1 is predicted to lack an RRM adjacent to the LaM ', 'A recent phylogenetic study in eukaryotes of all kingdoms identified Mlp1 as a new atypical genuine La protein in alveolates, containing a highly conserved LaM without an adjacent RRM (Deragon 2020). To investigate primary amino acid sequence conservation, multiple sequence alignments were conducted for La proteins from different eukaryotic lineages. We confirm that residues important for uridylate binding in the LaM are mostly conserved in T. thermophila (Figure 14b, Supplementary Figure 1,c-d). In contrast, we found little to no conservation of residues in the region of the adjacent domain where the RRM1 would be located (Supplementary Figure 1b) (Kelley et al. 2015). When comparing primary sequences of the La module between different LARPs, the presence of an RRM1 domain could often only be inferred from secondary structure predictions (Bousquet-Antonelli & Deragon 2009). Therefore, we compared the secondary structure predictions for T. thermophila La with secondary structures for different eukaryotes (Figure 14c). These consistently predicted an RRM fold immediately C-terminal to the LaM in all species examined, except for candidate alveolate La proteins. ', 'Mlp1 preferentially binds pre-tRNAs in Tetrahymena thermophila ', 'We hypothesized that should Mlp1 function as a genuine La protein, it should bind and promote the processing of RNA polymerase III transcripts, as has been demonstrated in budding and fission yeast (Van Horn et al. 1997; Yoo & Wolin 1997). We immunoprecipitated Mlp1-associated RNAs followed by detection by Northern blot (Figure 15a, Supplementary Figure 2a). Using probes specific for pre-tRNA 3’-extensions, we confirmed that Mlp1 immunoprecipitation enriched pre-tRNA species relative to mature tRNAs, which were relatively more abundant in input RNA fractions. We also observed lesser enrichment of the U5 small nuclear RNA (snRNA), which is consistent with data indicating association of La proteins with processing intermediates of U5 in Saccharomyces cerevisiae (Xue et al. 2000). We conclude that based on its expected RNA target cohort, Mlp1 behaves as expected for a genuine La protein. ', '', '', '', '', '', '', '', '', '', 'Figure legend on next page', 'Figure . Mlp1 preferentially binds pre-tRNAs partially via the 3’-terminal uridylates.', 'a Northern blot analysis of Mlp1 ribonucleoprotein-immunoprecipitated (RIP) samples from T. thermophila. Mlp1 enriches pre-tRNAs more efficiently than mature tRNAs (n=3 biologically independent samples). Pre-tRNA IleUAU and LeuUAA are both recognized by the 3’-trailer probe used. 5.8S rRNA was used as a non-binding loading control. FT: flowthrough, IgG: immunoglobin G control antibody. Western blot confirming Mlp1-specific immunoprecipitation is shown in Supplementary Figure 2a. b Next generation sequencing data of immunoprecipitated pre-tRNAs (rows represent biologically independent replicates), shown as a heatmap of log2+1 transformed fold enrichment calculated by taking ratios of normalized Mlp1-immunoprecipitated tRNA and input tRNA counts per million (CPM) for different tRNA isotypes (top). Next generation tRNA sequencing data from Gogakos et al. analyzed similarly for hLa (bottom). The log2+1 transformed fold enrichment was calculated by taking ratios of normalized La-immunoprecipitated tRNA and the averaged input tRNAs CPM. The data were split between premature and mature tRNAs based on the 3’-end of the transcript (-CCA ending mature tRNAs and U-ending pre-tRNAs). Reproducibility between replicates for T. thermophila was determined by correlation plots shown in Supplementary Figure 3. c Schematic representation of 32P-labeled T. thermophila pre-tRNA intermediates containing 5’-leader and 3’-trailer, 5’-leader only, 3’-trailer only and mature tRNA used in d-i,k, Supplementary Figure 2d,e and Fig. 3a,b. d-i Binding curves from EMSAs comparing 32P-labeled LeuAAG 5’-leader, 3’-trailer containing pre-tRNA and mature LeuAAG tRNA (n≥2 independent experiments). Native gels are shown in Supplementary Figure 4a and Kd quantification in Table 1. j Binding curves comparing binding of hLa, Mlp1 and Mlp1 mutants to 32P-labeled CUGCUGUUUU-3’OH RNA (n≥2 independent experiments). Native gels are shown in Supplementary Figure 4a and Kd quantification in Table 1. k Binding curves from competition EMSA between 32P-labeled 5’-leader containing pre-tRNA containing a 5’-triphosphate (+5’PPP) and unlabelled competitors +5’PPP as a positive control and 5’-triphosphate removed pre-tRNA (-5’PPP) (n=2 independent experiments). Native gels are shown in Supplementary Figure 4d,e. Source data are provided as a Source Data file.', '', '\\n', 'To further investigate Mlp1-associated RNAs, we sequenced Mlp1-immunoprecipitated RNAs after removing RNAs > 300 nt by gel electrophoresis. As expected for a genuine La protein, we observed an enrichment of reads mapping to pre-tRNA genes carrying 3’-uridylate extensions relative to their normalized abundance in size-matched input samples (Figure 15b – Mlp1). In contrast, mature tRNAs are not enriched in Mlp1-bound samples (Figure 15b – Mlp1; averaged enrichment values for tRNA isotypes presented in Supplementary Figure 2b). We compared our dataset with a previous study in which hLa-bound tRNA reads were obtained by photoactivatable crosslinking and immunoprecipitation (PAR-CLIP) and where tRNA sequencing of input and immunoprecipitated RNA was done using hydro-tRNAseq (Gogakos et al. 2017) and found a similar enrichment for 3’-uridylate containing pre-tRNAs (Figure 15b – hLa; averaged enrichment values for tRNA isotypes presented in Supplementary Figure 2c). The relative enrichment of pre-tRNAs over mature tRNAs, are strongly indicative of Mlp1 functioning as a genuine La protein in T. thermophila.', 'Mlp1 preferentially binds pre-tRNA substrates in vitro', 'The hLa protein preferentially binds pre-tRNA substrates through engagement of multiple sites, including the pre-tRNA body, 3’-trailer and 5’-leader (Bayfield & Maraia 2009; Fan et al. 1998). To determine whether enrichment of Mlp1 associated pre-tRNAs relative to mature tRNAs correlated with changes in affinity for such ligands, we compared binding affinity of Mlp1 for in vitro transcribed mature tRNA versus pre-tRNA substrates using electrophoretic mobility shift assays (EMSAs), as well as versions of these that included either or both of 5’-leader or 3’-trailer extensions (Figure 15c). We found that Mlp1 preferentially binds pre-tRNAs containing both 5’- and 3’-extensions, followed by pre-tRNA containing the 3’-trailer. Lowest binding affinities were consistently found for mature tRNAs, confirming the in vivo preferential binding of pre-tRNAs (Supplementary Figure 2d,e, Supplementary Table 1).', 'To further investigate Mlp1 tRNA binding, we compared the affinity of full-length Mlp1 or Mlp1 mutants to hLa for radioactively labeled tRNA targets by EMSA (Figure 15d-i, Supplementary Figure 4a, Table 1). To test the importance of 3’-uridylate binding, we compared Mlp1 to an Mlp1 mutant in which conserved amino acids predicted to recognize the UUU-3’OH motif were substituted (Q11A/Y14A)(Dong et al. 2004; Teplova et al. 2006) (see Figure 14b, Supplementary Figure 1d, Q20/Y23 numbering in hLa), as well as a mutant in which the entire LaM was deleted (Mlp1 95-340). We found that the increased affinity for pre-tRNAs was lost in the Mlp1 Q11A/Y14A mutant (compare Figure 15e and f) as well as for the LaM deletion mutant (compare Figure 15e and g). In contrast, the C-terminal DUF3223 is not important to discriminate between pre-tRNA and mature tRNAs since removal of this domain (Mlp1 1-250) maintains the binding affinity difference (Figure 15h), while the Q11A/Y14A mutations in the context of the deleted DUF3223 (Mlp1 1-250 Q11A/Y14A) again resulted in decreased affinity for pre-tRNAs (Figure 15i). To investigate UUU-3’OH binding more directly, we compared these Mlp1 mutants in protein-RNA binding experiments using a 3’-trailer sequence CUGUGUUUU-3’OH and found that the predicted uridylate binding residues located in the LaM (Q11A/Y14A) were required for binding (Figure 15j, Supplementary Figure 4a, Table 1). Additionally, binding of 3’-uridylate RNA occurs with the same affinity for Mlp1 1-250 compared to full length Mlp1 (Figure 15j), whereas affinity for uridylate RNA is lost when both domains are used individually (Mlp1 1-95 and Mlp1 95-250) (Supplementary Figure 4b). These data suggest that despite the apparent lack of the RRM1, Mlp1 functions in a similar manner as the hLa protein in preferentially binding pre-tRNA substrates, and that predicted, conserved uridylate binding residues in the LaM promote higher affinity binding associated with the UUU-3’OH motif. ', 'Table . Kd values from EMSAs determining binding of hLa, Mlp1 and Mlp1 mutants to typical La protein RNA targets. ', 'Kd: equilibrium dissociation constant, S.D.: standard deviation, N.D.: binding affinity not determined, N.A.: relative binding calculation not applicable.', '', \"The hLa protein is known to also interact with the 5’-triphosphate containing end of the nascent pre-tRNA through a short basic motif located in the C-terminal part of the protein (Fan et al. 1998). We tested the affinity of the 5’-triphosphate for Mlp1 by competition EMSA, using a 5’-leader containing, 3’-trailer processed in vitro transcribed radioactively labeled tRNA (+5’PPP) and unlabelled competitor tRNAs with the 5’-triphosphate removed after phosphatase treatment (-5’PPP; +/- 5’PPP demonstrated in Supplementary Figure 4c). The unlabelled –5'PPP substrate competed poorly relative to an unlabelled +5’PPP substrate for the radioactive +5’PPP substrate on hLa (Figure 15k – compare -5'PPP and +5’PPP hLa, Supplementary Figure 4d), but the difference in competition between these RNAs on Mlp1 was smaller (Figure 15k – compare -5'PPP and +5’PPP Mlp1, Supplementary Figure 4e). These data are consistent with a lesser degree of 5’-leader discrimination for 5’PPP in Mlp1 relative to hLa. \", 'Mlp1 binding to 3’-trailer RNAs is different from hLa', 'To further study the distinct binding modes, we performed competition experiments between 32P-labeled uridylate RNA (U10) and unlabelled competitor pre-tRNA and mature tRNA substrates. We used hLa as a control and found that, as expected from previous work (Bayfield & Maraia 2009), pre-tRNAs were a stronger competitor for the uridylate binding pocket than mature tRNAs (Figure 16a, Supplementary Figure 5a). In contrast, uridylate RNA was competed off Mlp1 using only low amounts of either pre-tRNA or mature tRNA (Figure 16b, Supplementary Figure 5a), suggesting that binding of uridylates on Mlp1 is weaker compared to hLa. ', '', '', '', '', '', 'Figure legend on next page', 'Figure . Mlp1 does not discriminate 3’-uridylates as stringently as hLa.', 'a,b Binding curves from competition EMSA between 32P-labeled uridylate RNA (U10) and unlabelled competitor U10 RNA (control), 5’-leader and 3’-trailer containing T. thermophila pre-tRNA LeuAAG and mature tRNA LeuAAG for hLa (a) and Mlp1 (b). Error bars represent the mean +/- standard deviation (Mlp1 U10: n=4, tRNA: n=2, hLa U10: n=2, tRNA: n=1 independent experiments). Native gels are shown in Supplementary Figure 5a. c Three-dimensional representation of the hLa protein in complex with uridylate RNA with the last three terminal nucleotides shown (UUU-3’OH ) (PDB: 2VOD). The penultimate uridylate U-2 is positioned in between the LaM and RRM1, while the 3’-terminal uridylate U-1 is positioned more towards the outside in stacking formation with U-3. RNA is shown in yellow; β-sheets are shown in red and α-helices shown in blue (dark blue: typical α-helices found in the winged-helix fold and classic RRM, light blue: inserted α-helices specifically found in La proteins). Image generated using PyMOL. d,e Binding curves from competition EMSAs using P32-labeled wild type CUGCUGUUUU (4U) and unlabelled 4U and mutant RNAs: CUGCUGUUUC (U-1C), CUGCUGUUCU (U-2C), CUGCUGUCUU (U-3C) and CUGCUGCCCC (4C) for hLa (d) or Mlp1 (e) (n=2 independent experiments). Native gels are shown in Supplementary Figure 5c,d. f Magnified views of hLa and 3’-terminal uridylate U-1 interactions (PDB: 2VOD) demonstrating the importance of the 3’-end. Protein carbon backbones have the same color coding as in c, oxygen is shown in red, nitrogen is shown in blue and phosphorous is shown in orange. RNA is shown in yellow. Image generated using PyMOl. g,h Binding curves from competition EMSA between P32-labeled U10-3’OH and unlabelled U10-3’OH (control) and U10-3’-P for hLa (g) and Mlp1 (h) (n=2 independent experiments). Native gels are shown in Supplementary Figure 5e. Source data are provided as a Source Data file.', '', 'Previous high-resolution structural characterization of hLa bound to UUU-3’OH established that the penultimate uridylate (UUU-3’OH) has the greatest importance for sequence specific, high affinity binding (Kotik-Kogan et al. 2008; Teplova et al. 2006). Notably, this uridylate is the only residue in the UUU-3’OH motif that contacts the RRM1 in hLa (Figure 16c). Previous work established that hLa has no binding affinity for 10-mers C10, G10 or A10, however, 20-mer versions of G20 and A20 have a considerable binding affinity for hLa (Vinayak et al. 2018). To compare the importance of the position of the penultimate uridylate (two nucleotides from the 3’-end: U-2), we performed competition EMSAs for hLa and Mlp1 using the radioactively labeled UUU-3’OH containing RNA CUGCUGUUUU-3’OH RNA (hence referred to as wild type 4U) and unlabelled RNAs carrying specific U to C variations to the terminal uridylates in this sequence, as previous work indicated the U to C substitution had the greatest effect on RNA substrate discrimination by hLa (Teplova et al. 2006), and we confirmed complete lack of binding affinity of Mlp1 for C10 (Supplementary Figure 5b). As expected, mutating the penultimate uridylate (U-2) into a cytidylate (U-2C – CUGCUGUUCU-3’OH) resulted in this RNA functioning as a poor competitor with hLa, whereas changes at the most 3’-terminal position (U-1C) and the third last position (U-3C) has a lesser effect on competition capability (Figure 16d, Supplementary Figure 5c). Mutating the four last positions (4C – CUGCUGCCCC-3’OH) resulted in the inability to compete with the wild type 4U RNA (Figure 16d, Supplementary Figure 5c). In contrast, competition for Mlp1 between radioactively labeled 4U and any unlabelled variant RNA competitor (U-1C, U-2C and U-3C) showed similar competition levels, indicating that the position of the penultimate uridylate U-2 is not as important for interactions between Mlp1 and 3’-trailer sequences (Figure 16e, Supplementary Figure 5d). Interestingly, unlabelled competitor 4C was more capable of competition with wild type 4U RNA for binding on Mlp1. These data are also consistent with Mlp1 discrimination for uridylate RNA not occurring as strongly as for hLa, and that the specificity of binding to the penultimate uridylate is weaker or absent for Mlp1.', 'The previous high-resolution work on hLa in complex with UUU-3’OH containing RNA also revealed the importance of the 2’- and 3’-hydroxyls (-OH) for high affinity binding in the hydrophobic binding pocket between LaM and RRM1 (Kotik-Kogan et al. 2008; Teplova et al. 2006), with two hydrogen bonds formed with an aspartate (Figure 16f) that is conserved in Mlp1 (see Figure 14b, Supplementary Figure 1d, D33 numbering in hLa). To investigate the importance of the free 3’-terminal hydroxyl groups for Mlp1 binding, we used competition EMSAs comparing a regular U10 and a 3’-phosphorylated unlabeled competitor RNA. We found that for both hLa and Mlp1 the phosphorylated substrate makes a poor competitor, confirming that the presence of a 3’-hydroxyl end is important for interactions between Mlp1 and uridylates (Figure 16g,h, Supplementary Figure 5e). Together, these results demonstrate that Mlp1 shows the same preference and LaM amino acid dependence for pre-tRNAs and UUU-3’OH binding as hLa, but that the altered architecture correlates with diminished discrimination for these substrates relative to hLa.', 'RNA chaperone function of Mlp1 in an S. pombe based heterologous system ', 'Our computational and biochemical work suggested that Mlp1 differs from other examined La proteins in its binding and discrimination of RNA targets. We have previously used a well-established, heterologous tRNA-mediated suppression system in S. pombe to investigate the function of both yeast and human La in the promotion of La-dependent pre-tRNA processing. This system can report on La-dependent 3’-end protection of nascent pre-tRNAs from exonucleases, as well as RNA chaperone activity to enhance correct folding of nascent pre-tRNAs through the ability to rescue a misfolded suppressor tRNA in vivo (Bayfield & Maraia 2009; Hussain et al. 2013; Naeeni, Contes, et al. 2012; Porat & Bayfield 2020). To test whether the altered Mlp1 architecture correlated with differences in pre-tRNA processing, we transformed S. pombe La protein Sla1p, full-length Mlp1 and multiple Mlp1 mutants into a sla1- S. pombe strain (ySH9) which encodes a defective stop codon UGA-decoding suppressor tRNA (tRNA-SerUCA) as well as the ade6-704 allele, which is decoded by a tRNA-SerUCA to suppress red pigment accumulation during growth on low adenine. Successful suppression in this system relies on the presence of a La protein or other suitable RNA chaperone, resulting in white colonies versus red colonies in unsuppressed cells. ', 'When comparing Sla1p-transformants to full-length Mlp1-transformants, we found that Mlp1 can stabilize the defective suppressor pre-tRNA similar to Sla1p (Figure 17a). Maturation of the defective suppressor tRNA can occur via 3’-terminal protection of uridylates from exonucleases, or general RNA chaperone activity, or a combination of these to assist with folding of pre-tRNAs (Huang et al. 2006). To test the importance of the uridylate binding residues for suppression, we compared the Mlp1 Q11A/Y14A mutant and the LaM deletion mutant (Mlp1 95-340) to wild type Mlp1 and observed a pink phenotype indicating intermediate suppression levels despite equal levels of protein expression (Figure 17a, Supplementary Figure 6a), suggesting that these uridylate binding residues function in maturation of the pre-tRNA in vivo. Next, we studied the function of the C-terminal DUF3223 and found that removal of this domain (Mlp1 1-250) led to near complete loss of suppression (Figure 17a), and combination of DUF3223 removal and uridylate binding inactivation (Mlp1 1-250 Q11A/Y14A) resulted in a complete loss of suppression (Figure 17a). These results indicate that to obtain complete tRNA mediated suppression, both the conserved uridylate binding residues (Q11 and Y14) in the LaM and the C-terminal DUF3223 are required. ', '', '', 'Figure legend on next page\\n', 'Figure . Mlp1 promotes tRNA mediated suppression in the absence of protection of 3’-trailers.', 'a tRNA-mediated suppression assay of Schizosaccharomyces pombe ySH9 strain transformed with pRep4 encoded Sla1p (positive control; S. pombe La protein), Mlp1 or indicated Mlp1 mutants (n=3 biologically independent samples). Protein expression levels are shown by Western blot in Supplementary Figure 6a. Levels of suppressor tRNA stabilization is based on colour of the colonies and annotated as follows: full suppression (+++), near full suppression (++), intermediate suppression (+), no suppression (-). Top: diagram showing domain architecture of Mlp1. b Northern blot to determine 3’-end protection of suppressor pre-tRNA-SerUCA in ySH9 transformants shown in a (n=2 biologically independent samples). Accumulation of suppressor pre-tRNA SerUCA was determined using an intron complementary probe and mature tRNA using a tRNA body probe leading to detection of both pre-tRNA and mature tRNA. The asterisk indicates a 3’-processed pre-tRNA intermediate. Excess unlabeled probe complementary to endogenous SerUGA was added to avoid cross-reaction between suppressor tRNA and endogenous tRNA. U5: loading control. c Northern blot to determine 3’-end protection of endogenous pre-tRNA LysCUU in ySH9 transformants shown in a, with suppressor tRNA stabilization levels (see Fig. 4a) shown above the blot (n=3 biologically independent samples). Accumulation of pre-tRNA LysCUU intermediates was determined using an intron complementary probe which detects unprocessed 5’-leader and 3’-trailer containing pre-tRNA (band a), 5’-leader processed pre-tRNA (band b) and both 5’-leader and 3’-trailer processed pre-tRNA (band c) in sla1 transformants (positive control; lane 2). The 3’-processed, 5’-leader containing pre-tRNA intermediate is annotated as band d. The same blot was probed with a 3’-trailer complementary probe, a 5’-leader complementary probe and a mature tRNA complementary probe. Black boxes correspond to a marker size of 100 nt. U5: loading control. d Sanger sequencing of clonal isolates corresponding to cDNAs derived from 3’-terminal sequences from Sla1p- and Mlp1-immunoprecipitated pre-tRNAs in ySH9 as transformed in a-c (n=2 biologically independent samples). Western blot of Sla1p- and Mlp1-immunoprecipitations shown in Supplementary Figure 6b. Source data are provided as a Source Data file.', '', '\\n', \"To investigate 3’-end protection more directly, we extracted total RNA from Mlp1- and Sla1p-expressing strains and analysed suppressor pre-tRNA SerUCA processing by Northern blot (Figure 17b). We found that higher levels of suppressor pre-tRNA stabilization in Sla1p- and Mlp1-transformed strains (lanes 2, 3) corresponded to more mature suppressor tRNA which correlates with the white phenotype observed in the tRNA-mediated suppression results (see Figure 17a), as opposed to the lack of pre-tRNA stabilization and subsequent mature suppressor tRNA in pRep4 control (lane 1) resulting in a red phenotype (Figure 17a,b). Interestingly, the most abundant pre-tRNA species was slightly smaller in Mlp1 transformed cells relative to Sla1p (see asterisk). We also detected endogenous pre-tRNAs LysCUU processing intermediates using an intron, 5’-leader and a 3’-trailer probe (Figure 17c). The presence of the three La-dependent pathway pre-tRNA intermediates are detected after transformation of the positive control Sla1p, as have been described previously (Intine et al. 2000) (Figure 17c – intron probe, compare lanes 1 and 2, indicated by a, b and c). Notably, we observed the appearance of a distinct pre-tRNA intermediate in Sla1 expressing cells when using a probe specific for the 5’-leader of pre-tRNA LysCUU that did not co-migrate with any of the major species observed using the intron or 3’-trailer probe (Figure 17c – 5'-leader, indicated by d, beneath the full length pre-tRNA band). We hypothesize that this species represents a subset of pre-tRNAs that are not stably bound by Sla1p and whose 3’-ends have been nibbled by a 3’-exonuclease, yet have retained their 5’-leaders as has been described during La-independent pre-tRNA processing. As expected from the tRNA-mediated suppression assay, Mlp1 also stabilized endogenous precursor pre-tRNA LysCUU, however, this precursor tRNA co-migrated with the 5’-leader containing, 3’-trailer exonuclease processed intermediate observed in Sla1p -transformants (Figure 17c – 5’-leader, band d, compare lane 2 and 3), consistent with Mlp1 stabilizing a 5’-leader containing, 3’-processed or nibbled pre-tRNA intermediate. As expected, all Mlp1 mutants defective in tRNA-mediation suppression were defective in stabilizing pre-tRNA intermediates. Together, these data are consistent with Mlp1 engaging pre-tRNAs and promoting tRNA-mediated suppression in S. pombe, but with impaired protections of pre-tRNA 3’-ends relative to Sla1p.\", 'To further compare Mlp1 and Sla1p function in the processing of pre-tRNA intermediates by Mlp1 in S. pombe, we immunoprecipitated Sla1p and Mlp1 RNP-complexes (Supplementary Figure 6b) and sequenced the 3’-ends of their associated pre-tRNA LysCUU and pre-tRNA TyrGUA by 3’-rapid amplification of cDNA ends (3’RACE). While Sla1p-associated pre-tRNAs were enriched for species containing primarily four and five nucleotide long uridylate 3’-trailers, as has been described previously (Huang et al. 2005), Mlp1-associated pre-tRNAs were largely depleted for 3’-trailer containing species (Figure 17d), consistent with the Northern blotting results (Figure 17b,c), with a minority of Mlp1 immunoprecipitated pre-tRNAs ending in 3’-CCA, indicating Mlp1 can also bind pre-tRNAs after CCA addition but prior to nuclear export and intron removal. These data are consistent with our Mlp1 RNA binding data in vitro, indicating that Mlp1 promotes pre-tRNA processing but with impaired 3’ discrimination and lower protection of the 3’-ends of associated pre-tRNAs relative to other examined genuine La proteins.', 'Mlp1 depletion in T. thermophila leads to impaired 3’-trailer processing', 'To investigate the effect of Mlp1 depletion on pre-tRNA processing in T. thermophila, we generated a partial MLP1 knockout strain and confirmed genomic integration of the selection marker by Southern blot and PCR (Supplementary Figure 7a,b and data not shown) and reduced protein expression of Mlp1 by Western blot (Figure 18a) and indirect immunofluorescent staining (Supplementary Figure 7c). Complete deletion of the MLP1 locus in T. thermophila macronuclei through increasing drug selection (phenotypic assortment (Merriam & Bruns 1988)) was not achieved, indicating that Mlp1 is likely essential in this system, similar to Mus musculus, Drosophila melanogaster, Trypanosoma brucei and Arabidopsis thaliana (Arhin, Shen, P, et al. 2005; Bai & Tolias 2000; Fleurdépine et al. 2007; Park et al. 2006). We extracted total RNA followed by pre-tRNA intermediate detection by Northern blot for pre-tRNA IleUAU, LeuUAA and ValAAC. In the absence of Mlp1, a 3’-trailer containing pre-tRNA intermediate was stabilized (Figure 18b – intron probe, bottom band). These data provide evidence for T. thermophila being the first eukaryote in which La protein levels correlate inversely with 3’-trailer stabilization. These results are also consistent with our heterologous expression data in S. pombe indicating impaired protection of pre-tRNA 3’-trailers by Mlp1. These data suggest that the variant domain architecture of Mlp1 relative to other genuine La proteins is associated with an altered pre-tRNA processing pathway in this system', '. ', '', '', 'Figure legend on next page', 'Figure . Depletion of Mlp1 results in altered pre-tRNA processing in T. thermophila with normal levels of mature tRNAs.', \"a Western blot confirming decreased protein expression of Mlp1 in the partial Mlp1 knockout (KO) strain compared to the wild type (WT) strain (n=3 biologically independent samples). Loading control: histone H3 and β-actin. Purified 6X-His-Mlp1 was analysed to control for antibody specificity. b Northern blot detecting tRNAs IleUAU, LeuUAA and ValCAC pre-tRNA intermediates with an intron, 5’-leader, and 3’-trailer specific probe (n=3 biologically independent samples). Pre-tRNA IleUAU and LeuUAA are both recognized by 3’-trailer and 5’-leader probe used. Black boxes correspond to a marker of the same size. U5 snRNA and/or 5.8S rRNA were used as loading controls. c Next generation sequencing data shown as a heatmap of log10 transformed normalized cumulative pre-tRNA counts for each uridylate tail length (n=3 biologically independent replicates). For clarity, the average is plotted and the longest uridylate tail shown is U6 (UUUUUU-3'OH). d Quantification of uridylate tail length from next generation sequencing data (n=3 biologically independent samples) in c. Error bars represent the mean +/- standard deviation. A Two-way ANOVA with Bonferroni’s correction for multiple comparisons was performed to determine statistical significance: **** P < 0.0001, * P = 0.0491. For each tRNA isodecoder, the presence of a uridylate tail length (U1, U2, etc.) was scored as present if its abundance in CPM was greater than 0.3; log10(2). e,f Scatterplots of log2 transformed normalized tRNA counts (CPM) from next generation sequencing data from WT strains (e) and partial Mlp1 KO strains (f) plotted against the number of genes per tRNA isotype encoded in the genome (see Additional Data 1). The correlation was calculated, and a positive correlation was observed for both WT and KO strains (r = 0.746 and r = 0.731, respectively) indicating that more tRNA gene copies result in higher tRNA expression. g Plotting of log2 transformed normalized WT and partial Mlp1 KO counts (CPM) gives a strong positive correlation (r = 0.970). h Northern blot analysis detecting mature tRNA TyrGUA and ArgUCU levels using a probe specific for the spliced mature tRNA (n=3 biologically independent samples). U5 was used as a loading control on both membranes. Source data are provided as a Source Data file.\", '', '', '', \"To study the effect of Mlp1 depletion on a transcriptome-wide scale, we performed total RNA-sequencing of small RNA-enriched samples, similar to our Mlp1 RIP-Seq (see Figure 15b), in which counts for each pre-tRNA isoacceptor were determined for each unique uridylate tail length at the 3’-end of the tRNA. We found that the average length of the 3'-uridylate tail increased for the majority of tRNA species as a result of Mlp1 depletion (Figure 18c,d). These findings indicate that Mlp1 promotes removal of the 3’-trailer and that when Mlp1 is limiting, the 3’-trailer sequences are not processed as efficiently. Together, these data are consistent with Mlp1 accelerating 3’-end processing, relative to other examined species in which genuine La proteins stabilize 3’-trailers.\", 'Previous work has demonstrated that in eukaryotes, the number of genomic tRNA gene copies correlates strongly with the expression of mature tRNA transcripts and thus can function as a predictor for tRNA expression levels (Behrens et al. 2021; Tuller et al. 2010). We observed that in T. thermophila the number of genomic tRNA genes correlates with the expression levels of tRNAs (R2 = 0.55, r = 0.75) (Figure 18e). To investigate if depletion of Mlp1 results in changes of total tRNA levels, we performed the same analysis for our Mlp1 partial knockout strain (R2 = 0.53, r = 0.73) (Figure 18f) and found that when comparing tRNA expression between WT and partial Mlp1 knockout strains the correlation was very strong (R2 = 0.94, r = 0.97) (Figure 18g), indicating that Mlp1 depletion did not influence mature tRNA expression. Northern blot analysis for representative tRNAs confirmed similar amounts of mature tRNA between wild type and partial Mlp1 knockout strains (Figure 18h), reminiscent of minimal differences in mature tRNA abundance (despite changes to pre-tRNA processing) in conditional mouse La knockout cells (Gaidamakov et al. 2014).', 'The 3’-trailer lengths are shorter in T. thermophila relative to other eukaryotes', 'To explore tRNAs and pre-tRNA processing in T. thermophila more extensively, we compared their predicted pre-tRNA architecture to those from other eukaryotic species. We compared 3’-trailer lengths of each pre-tRNA, as determined by the number of nucleotides between the discriminator base and the genomic stretch of at least four consecutive thymines (Bogenhagen & Brown 1981) that give rise to the 3’UUU-OH motif in the nascent transcript. We found that T. thermophila has shorter 3’-trailer lengths than any other species analyzed, with the most common 3’-trailer length being zero nucleotides (Figure 19a, Table 2, Additional Data 2). Identical trends were observed for genomic stretches/candidate terminators of five, six or seven consecutive thymines (Supplementary Table 2, Additional Data 2). A similar analysis for the total length of the mature tRNA revealed that, as expected, the average length of mature tRNAs is the same as other examined species (Figure 19b, Table 2). We previously noted differences in enrichment of pre-tRNAs isotypes (Supplementary Figure 2b – pre-tRNAs) in our Mlp1 RIP-seq results and hypothesized that the 3’-trailer length could be a determinant for binding affinity. We plotted fold enrichment for pre-tRNAs between Mlp1-immunoprecipitated and input tRNAs against 3’-trailer length and found that these are moderately negatively correlated (R2 = 0.14, r = -0.47) (Supplementary Figure 8a), suggesting that nascent pre-tRNAs containing a longer 3’-trailer sequence may have a slightly lower binding affinity for Mlp1. When comparing this to the hLa data, we found a lesser correlation (R2 = 0.05, r = -0.23) (Supplementary Figure 8b). ', '', '', '', 'Figure legend on next page', '\\n', 'Figure . The 3’-trailer length of T. thermophila tRNAs is considerably shorter compared to other eukaryotes and affects the N-1 composition of the 5’-leader.', 'a Genome-wide determination of 3’-trailer lengths as determined by the number of nucleotides between the discriminator base (the most 3’-terminal nucleotide in the mature tRNA upstream of the posttranscriptional added CCA) and a genomic stretch of four Ts of different eukaryotes. Violin plots shows the median as a full purple line and the quartiles as full black lines. Data summary shown in Table 2. b Genome-wide analysis of mature tRNA length for different eukaryotes. Violin plots show the median as a full purple line and the quartiles as full black lines. No statistical significance (P < 0.05) was observed using a one-way ANOVA and Tukey’s multiple comparison test. Data summary shown in Table 2. c Schematic representation of a pre-tRNA. The mature tRNA sequence is shown as a black rectangle and written in uppercase (UCGA) in the tRNA cartoon. Pre-tRNA specific sequences, including the 5’-leader and 3’-trailer are shown as a black line and written in lowercase (ucga) in the tRNA cartoon. The discriminator base (N73) is the most 3’-terminal nucleotide of the mature tRNA sequence highlighter in dark green. The most 3’-terminal nucleotide of the 5’-leader sequence is highlighted with a light green box (N-1). d,e Logo analysis of 5’-leader sequences of pre-tRNAs in T. thermophila, S. pombe and H. sapiens (d). The same analysis split by the discriminator base identity (e). The number of pre-tRNAs representing each condition is shown underneath each logo. Source data are provided as a Source Data file.', '', 'Table . Summary of 3’-trailer and mature tRNA sequence lengths in different eukaryotic species.', 'S.D.: standard deviation, nt: nucleotides.', '', 'Processing of the 5’-leader by RNase P is based on tertiary structure recognition of the tRNA followed by endonucleolytic cleavage at the –1/+1 position (Leontis et al. 1988; Wu et al. 2018). Previous studies demonstrated structural conservation of the catalytically active RNA in RNase P in T. thermophila, indicating that processing occurs in a similar manner (True & Celander 1996; Wu et al. 2018). Optimal efficiency of RNase P cleavage is dependent on the presence of a bulge that includes the last nucleotide of the 5’-leader (the N–1 position; Figure 19c), as extensive base pairing between 5’-leader and 3’-trailer sequences inhibits 5’-leader cleavage (Lee et al. 1997). We hypothesized that T. thermophila tRNAs should have adapted 5’-leader sequences to ensure the generation of a mismatched bulge against the shorter 3’-trailer sequences. We determined the most common 5’-leader sequences by logo generation for T. thermophila, S. pombe and Homo sapiens and found that the most frequent nucleotide at the most 3’-residue of the 5’-leader sequence (N-1) in T. thermophila is an adenine (± 75%) (Figure 19d, Supplementary Figure 8c). In contrast, the distribution of nucleotides at this position is more diverse in other species (Figure 19d, Supplementary Figure 8c). At the 3’-end of the mature transcript, the discriminator base (N73) is mostly adenine (± 50%), followed by guanine as second most frequent nucleotide (± 25%) (Supplementary Figure 8d) which is common for all eukaryotes in this study. ', 'We then split our data based on the identity of the discriminator base (N73) (Figure 19e, Supplementary Figure 8e), which can pair with the first nucleotide in the 5’-leader (N-1). We observed a strong lack of Watson–Crick base pairing for T. thermophila between the opposing nucleotides in the discriminator base pairing with the typical adenine (A73 – A-1; C73 – A-1 and G73 – A-1). Conversely, tRNAs containing a thymine as discriminator base (T73) strongly avoided an adenine as the N-1 5’-leader base to avoid base pairing. This partitioned discrimination is not evident for S. pombe and H. sapiens (Figure 19e). Since shortened 3’-trailers in T. thermophila should have a greater dependence on leader-trailer bulges occurring through the discriminator base, these data are thus consistent with the 3’ most nucleotide of the 5’-leader having a greater evolutionary pressure to avoid base pairing with the discriminator base which is seen most strongly for T. thermophila (Supplementary Figure 8f) relative to other examined species.', 'Discussion', 'Previously studied genuine La proteins contain a conserved tandem arrangement of a LaM and RRM1 collectively referred to as a La module. Previous phylogenetic predictions (Deragon 2020) and our continued computational analysis have indicated that the previously characterized T. thermophila protein Mlp1 may group with the genuine La proteins, despite the predicted lack of the La module’s RRM1 domain. In addition to the absence of the RRM1, Mlp1 is predicted to have a previously uncharacterized domain of unknown function (DUF3223), which is absent in all other examined members of the LARP superfamily. Since this arrangement of a genuine La protein is unprecedented, it was not clear how this protein might perform La associated functions and what effects this might have, if any, on the processing of La RNA targets.', 'We present data consistent with Mlp1 functioning as a genuine La protein. Using in vitro binding assays, we demonstrate that the identity of the Mlp1 residues Q11 and Y14, analogous to UUU-3’OH binding residues in hLa, are also required for high affinity binding of 3’-uridylate containing trailer sequences. We demonstrate that binding of uridylate RNA is retained when both the predicted LaM (Mlp1 1-95) and middle domain (Mlp1 95-250) are included, however, uridylate binding does not occur when either the LaM (Mlp1 1-95) or the middle domain (Mlp1 95-250) are tested in isolation. Given the relative paucity of contacts from the RRM relative to the LaM during hLa binding to UUU-3’OH, it is likely that the 95-250 region might serve an analogous function, making contacts important for UUU-3’OH binding despite the absence of the RRM fold. ', 'While the Mlp1 LaM and middle domain combine to support UUU-3’OH binding similar to the classic La module, other key differences exist. Competition experiments indicated that short UUU-3’OH containing trailers were more easily displaced from Mlp1 by pre-tRNA and mature tRNA substrates, relative to hLa, suggesting that Mlp1 binding to UUU-3’OH may be less stable compared to La proteins with the classic LaM-RRM1 arrangement. The previous hLa-UUU-3’OH co-crystals revealed that RRM1 makes only a single contact with the penultimate uridylate RNA (U-2) during UUU-3’OH binding, and it is this uridylate that is recognized with the greatest specificity. Unlike hLa, the U-2C RNA was no less effective a competitor against the 4U RNA than the U-3C and U-1C RNAs when Mlp1 was tested. The impaired ability of Mlp1 to discriminate the U-2 residue, as well as uridylates more generally, suggest that the lack of the RRM1 results in a relatively lower ability of Mlp1 to differentiate UUU-3’OH containing RNAs.', 'Previous work has demonstrated that the La module (LaM+RRM1) of different LARPs, and more specifically their RRM regions, possess RNA chaperone activity in the absence of 3’-uridylate protection from exonucleases (Hussain et al. 2013; Naeeni, Contes, et al. 2012). Using tRNA-mediated suppression in a La null strain (sla1-) of S. pombe, we demonstrated that while the uridylate binding residues in the LaM of Mlp1 are important for suppression, the DUF3223 region also promotes the correct folding of defective pre-tRNAs. These data raise the possibility that the DUF3223 region of Mlp1 may serve an analogous function in RNA chaperone activity previously associated with the RRM1 and other C-terminal regions of genuine La proteins.', 'Genuine La proteins are however well known to also stabilize 3’-trailer containing intermediates through high-affinity binding of the 3’-uridylate tail and thereby providing protection against 3’-exonucleases. Using Northern blotting and RIP-3’-RACE, we demonstrated that while S. pombe La stabilizes 5’-leader, 3’-trailer, and intron-containing pre-tRNA intermediates, Mlp1 stabilized a 5’-leader and intron containing, but 3’-trailer lacking, pre-tRNA species, suggesting that Mlp1 may accelerate 3’-processing of nascent pre-tRNAs, relative to other La proteins. This could imply that binding of Mlp1 to the 3’-ends of the nascent pre-tRNAs is not as strong as compared to Sla1p, leaving the RNA exposed to 3’-exonucleases. Alternatively, Mlp1 could increase access of the 3’-trailer to the endonuclease RNase Z.', 'To better understand how the altered architecture of Mlp1 might influence pre-tRNA processing, we generated a partial MLP1 knockout strain in T. thermophila and performed RNA-Seq and Northern blots of endogenous tRNA species. We found that, consistent with our Mlp1 data from S. pombe, Mlp1 appears to promote the removal of 3’-trailers, as depletion of Mlp1 in vivo leads to a greater relative abundance of UUU-3’OH trailer extensions. In yeast, depletion of La leads to exonucleolytic nibbling and a lower abundance of UUU-3’OH ends, while in the presence of La the 3’-end is stabilized until endonucleolytic cleavage by RNase Z directly 3’ to the discriminator base (N73) (Figure 20). ', 'Previous work has demonstrated that hLa impedes access by RNase Z, delaying the 3’-endonucleolytic and resulting in 5’-leader processing proceeding 3’-trailer processing (Nashimoto et al. 2001). We hypothesize that the presence of an RRM1 in hLa could be contributing to this function, seeing that in Mlp1, a La protein lacking the RRM1, the 3’-processing appears not to be blocked. Thus, the apparent destabilization of 3’-trailers by Mlp1 in T. thermophila suggests that the function of La in this system, differing from other examined La proteins, may be to increase access of the pre-tRNA 3’-trailer to the tRNA processing machinery (Figure 20). The difference in processing of pre-tRNAs is likely caused by Mlp1 since the 3’-exonuclease Rex1p and 3’-endonuclease RNase Z are conserved in T. thermophila (Supplementary Figure 9a,b). An alternate but not mutually exclusive hypothesis may be that reduction of Mlp1 levels may alter access of pre-tRNA 3’-ends to the LSm2-8 complex, and that this may be linked to 3’-trailer accumulation relative to wild-type cells, as previous work in yeast has also linked the LSm2-8 complex to the processing of pre-tRNA 3’-ends (Kufel & Tollervey 2003; Kufel et al. 2002). To confirm that the LSm2-8 complex functions similar as in other eukaryotic systems, we compared primary amino acid sequence alignments and found that the LSm2-8 complex in T. thermophila appears to be conserved, indicating similar functionality (Supplementary Figure 9c,d). We also observed that Mlp1 depletion did not lead to changes in mature tRNA expression levels, suggesting that an Mlp1-independent tRNA maturation pathway also likely exists in T. thermophila, similar to budding and fission yeast.', '', 'Figure . Model for pre-tRNA processing in T. thermophila.', 'During La-dependent processing (top, left) in previously studied eukaryotes such as yeast, the La protein is the first protein to associate with pre-tRNAs on the 3’-stretch of uridylates generated by RNA polymerase III transcription termination. Binding of La provides protection from degradation by 3’-exonucleases, assists with tRNA folding through RNA chaperone activity and stabilizes the nascent pre-tRNA. The next step in tRNA processing is endonucleolytic removal of the 5’-leader by RNase P, followed by an endonucleolytic cut by RNase Z resulting in removal of the 3-trailer sequence and the La protein bound to the uridylate stretch. In contrast, during La-independent processing of pre-tRNAs (top, right), the 3’-trailer is rapidly removed first by 3’-exonucleases such as Rex1p, followed by RNase P processing resulting in an end-matured tRNA. Our data from T. thermophila point towards a pre-tRNA processing model in which Mlp1-dependent processing (bottom, left) 3’-trailers are processed efficiently. Mlp1-independent processing results in accumulation of pre-tRNAs containing unprocessed 3’-trailer sequences, indicating that Mlp1 is required for efficient 3’-end processing unlike other eukaryotes. Image created with BioRender.com. The Publication License is provided as a Source Data file. ', 'Along with the alternate Mlp1-associated tRNA processing in T. thermophila, we observed several differences in features of pre-tRNA 5’-leaders and 3’-trailers compared to other eukaryotic species. Our genome-wide 3’-trailer length analysis in eukaryotes revealed that T. thermophila pre-tRNAs have very short 3’-trailer sequences. Since 3’-trailers in T. thermophila are dominated by the uridylate stretch, we studied the prevalence of the 3’-most terminal residue of the 5’-leader (N-1) and found that T. thermophila pre-tRNAs appear to have evolved to avoid perfect matches with the discriminator base (N73) (preceding the uridylate tail), ensuring a lack of complete base pairing to enhance RNase P cleavage in the context of a minimized 3’-trailer sequence. Further upstream in the 5’-leader (N-2) is often an adenosine, but then typically uridylates, which would not be expected to pair with the 3’UUU-OH motif. The 5’-leaders and 3’-trailers in T. thermophila may therefore have evolved to have minimal base pairing between pre-tRNA 5’-leaders and 3’-trailers, compared to other eukaryotes in which a bulge proximal to the mature tRNA ends is often followed by a paired region closer to the 5’-leader and 3’-trailer extremities. For example, human pre-tRNAs have an approximately equal distribution of N-1 nucleotide for discriminator bases A, T and G. The most prevalent N-1 nucleotide for discriminator bases A and T, are T and A respectively, indicating that human pre-tRNAs, which generally contain a longer 3’-trailer sequence, have more flexibility in N-1 sequence to introduce a mismatch bulge at the N-1 site for optimal 5’-leader processing by RNase P. It will be interesting to further investigate whether this alternate arrangement is linked to the altered functional roles described here for Mlp1 in this system. ', 'Methods', 'Conservation analysis. ', 'Accession numbers used to obtain primary amino acid sequences from the National Center for Biotechnology Information (NCBI) for La, Rex1p, RNase Z/ELAC 2 and LSm2-8 complex are shown in Supplementary Table 3 and primary amino acid alignments were obtained using Clustal Omega (EMBL-EBI) (Sievers et al. 2011), followed by analysis using a custom python script to annotate identical and conserved amino acids using human protein sequences as a reference. Amino acids were grouped as conserved based on side chain diversity: (1) Asp (D), Glu (E), Asn (N), Gln (Q); (2) Lys (K), Arg (R), His (H); (3) Phe (F), Trp (W), Tyr (Y); (4) Val (V), Ile (I), Leu (L), Met (M); and (5) Ser (S), Thr (T). ', 'Secondary structure predictions were obtained using Phyre2 (Kelley et al. 2015) and color coded based on predicted β-sheet or α-helices. High-resolution structures of the LaM of H. sapiens, T. brucei, and D. discoideum La proteins were obtained from PDB: 2VOD, 1S29, and 2M5W, respectively. Structure of the La module (LaM+RRM1) in complex with uridylate RNA of H. sapiens La was obtained from PDB: 2VOD. Structure prediction of the LaM of T. thermophila was obtained using the lomets2 tool (Zheng et al. 2019). Structure of the LSm2-8 complex in complex with uridylate RNA of S. pombe and S. cerevisiae were obtained from PDB: 6PNN and 4M7D, respectively. Modeling was done using PyMOL. ', '', 'DNA constructs. ', 'The MLP1 sequence encoding T. thermophila La protein (NCBI Reference Sequence: XP_001019287.2) was obtained from the Genome Database  (TTHERM_00384860) (Stover et al. 2012), and codon optimized for expression in E. coli. gBlocks of the optimized codon sequence were ordered from integrated DNA technologies (IDT) (sequence can be found in Additional Data 3). ', 'Mlp1 and Mlp1 mutants used for in vitro electromobility shift assays (EMSAs) were cloned in the NheI and BamHI restriction enzyme sites of the pET28a vector using the plasmid encoded N-terminal 6XHis-tag for protein purification. pET28a hLa was previously cloned in the NcoI and BamHI sites including a reverse primer-encoded 6X-His tag. Mlp1 and Mlp1 mutants used for S. pombe transformations were cloned in the SalI and BamHI sites in pRep4 plasmid (ura+) incorporating a 6X-His tag N-terminally in the forward primer during PCR amplification. All primers are listed in Additional Data 3.', 'T7 DNA templates for in vitro transcription were generated by PCR amplification using pre- or mature tRNA specific primers (listed in Additional Data 3) to obtain DNA templates containing an upstream T7 promotor. The DNA template was gel purified using 7 M urea denaturing 10% polyacrylamide gel and DNA was eluted from the gel by overnight rotation in 150 mM sodium acetate in 50% phenol:chloroform:isoamylalcohol (25:24:1) at 4°C. The aqueous layer obtained by centrifugation at 20,000 g for 10 minutes at 4°C was ethanol precipitated overnight at -80°C.', '', 'T. thermophila cultivation and knockout strain generation.', 'Liquid cultures were grown to mid-log phase (0.1 - 1 x 106 cells/mL) at 30°C shaking at 90 RPM in SPP media (1% proteose peptone, 0.1% yeast extract, 0.2% glucose, 0.003% Fe-EDTA) (Fillingham et al. 2001). Cell pellets were harvested by centrifugation for 3 minutes at 1000g. Following two washes in 10 mM Tris-HCl, pH 7.4, pellets were stored at -80°C. ', 'PCR-amplification from T. thermophila genomic DNA to obtain flanking 5’- and 3’-regions was completed using primers containing KpnI/XhoI and NotI/SacI, respectively. The flanking regions of the MLP1 gene (http://www.ciliate.org/ - TTHERM_00384860) were cloned into pNeo4 plasmid (Mochizuki 2008) flanking the paromomycin (Neo4) drug resistance cassette using restriction enzyme sites KpnI XhoI and SacI NotI, respectively. The Neo4 cassette is located downstream of the CdCl2 inducible metallothionein (MTT1) promoter. The resulting pNeo4 MLP1 knockout plasmid DNA construct was linearized using the ScaI restriction enzyme prior to transformation. Biolistic transformation of T.thermophila was performed as described previously (Chalker 2012). Integration of the DNA construct is based on homologous recombination and transformants were grown under increasing concentration of paromomycin starting at 60 μg/mL to a final concentration of 1000 μg/mL (phenotypic assortment) (Merriam & Bruns 1988). Correct integration was determined using Southern blot and PCR. Partial MLP1 knockout strain available on request.', '', 'Protein isolation from T. thermophila.', 'T. thermophila cell pellets were resuspended in 10% Trichloroacetic acid (TCA) in 1X PBS, followed by incubation at –20°C overnight to enhance protein precipitation. The white fluffy protein pellet was collected by centrifugation for 15 minutes at 10,000g at 4°C, then washed twice with 100% ice-cold acetone. The pellet was air-dried and resuspended in 50μL/mL culture 2.5X SDS loading dye [5X SDS loading dye: 5% β-mercaptoethanol (v/v), 0.02% bromophenol blue (w/v), 30% glycerol (v/v), 10% sodium dodecyl sulfate (SDS) (w/v), 250 mM Tris-HCl, pH 6.8]. ', 'RNA-immunoprecipitation from T. thermophila.', 'T. thermophila cell pellets from 100 mL cultures were collected at log phase (0.1 - 1 x 106 cells/mL) and washed twice with 1X PBS. The cells were cross-linked with 1% formaldehyde for 10 minutes followed by quenching with 0.25 M glycine for 5 minutes at room temperature. Cells were washed twice in 1X PBS before lysis in 2 mL buffer A [30 mM Tris-HCl pH 7.4, 150 mM NaCl, 20 mM KCl, 2 mM MgCl2, 0.1% Triton-X100, 1 mM phenylmethylsulfonyl fluoride (PMSF), 1X Halt Protease Inhibitor Cocktail (PIC) (ThemoFisher Scientific), 100U SUPERase In RNase Inhibitor (ThermoFisher Scientific)] followed by sonication (25% amplification with 15s intervals for 4 minutes). The lysate was clarified by centrifugation at 20,000 g for 45 minutes at 4°C and treated with 10 U TurboDNase. The lysates were pre-cleared using rabbit isotype immunoglobin G (IgG) control-bound Protein G Dynabeads rotating for 1 hour at 4°C (12 μg / 50 μL beads slurry). Immunoprecipitations were performed using an affinity purified rabbit anti-Mlp1 antibody (ThermoFisher Scientific – Custom Antibodies) and a rabbit IgG control coupled to Protein G Dynabeads rotating 1 hour at 4°C (12 μg / 50 μL beads slurry). The Protein G Dynabeads were washed 5 times using buffer A (without PIC, PMSF and RNase inhibitor). Input and eluted RNA was isolated following reverse cross-linking for 45 minutes at 70°C in buffer B [50 mM Tris-HCl pH 7.4, 5 mM EDTA pH 8.0, 10 mM DTT, 1% SDS], followed by Trizol extraction.', 'tRNA library preparation and TGIRT sequencing and analysis.', 'Total and Mlp1-associated RNAs were size-selected (20-300 nucleotides) on a 7 M urea denaturing 10% polyacrylamide gel. RNA was eluted from the gel by overnight rotation in 150 mM sodium acetate, pH 5.1 in 50% phenol:chloroform:isoamylalcohol (25:24:1) at 4°C. The aqueous layer obtained by centrifugation at 20,000g for 10 minutes at 4°C was ethanol precipitated overnight at -80°C. The pellet was washed once with 70% ethanol, air dried and resuspended in RNase-free H2O. To promote sequencing of highly structured tRNA species carrying modifications that are inhibitory to cDNA synthesis, we used TGIRT-III reverse transcriptase (InGex) template-switching to prepare cDNA libraries as described previously (Fafard-Couture et al. 2021; Nottingham et al. 2016). Libraries were sequenced on a NextSeq500 platform (Illumina) (2x 75 bp). ', 'tRNA-Seq processing pipeline.']\n"
     ]
    }
   ],
   "source": [
    "# open connection to Word Document\n",
    "doc = docx.Document(r\"C:\\Users\\bruno.henderyckx\\Desktop\\Thesis\\Thesis - v9 final draft.docx\")\n",
    " \n",
    "# read in each paragraph in file\n",
    "result = [p.text for p in doc.paragraphs]\n",
    "print(result[0:500])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "3fa42840",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-03T16:14:49.809652Z",
     "start_time": "2022-11-03T16:14:49.791701Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing of non-coding RNAs by Mlp1 in Tetrahymena thermophilaKyra KerkhofsA dissertation submitted to the faculty of graduate studies in partial fulfillment of the requirements for the degree of Doctor of Philosophy Graduate Program in Biology\n",
      "York University\n",
      " Toronto, OntarioDecember 2022© Kyra Kerkhofs 2022AbstractIn eukaryotes, many RNA species are transcribed as precursor molecules and require post-transcriptional processing to become a functional mature RNA. This includes the very abunda\n",
      "['Processing', 'of', 'non-coding', 'RNAs', 'by', 'Mlp1', 'in', 'Tetrahymena', 'thermophilaKyra', 'KerkhofsA', 'dissertation', 'submitted', 'to', 'the', 'faculty', 'of', 'graduate', 'studies', 'in', 'partial', 'fulfillment', 'of', 'the', 'requirements', 'for', 'the', 'degree', 'of', 'Doctor', 'of', 'Philosophy', 'Graduate', 'Program', 'in', 'Biology', 'York', 'University', '', 'Toronto,', 'OntarioDecember', '2022©', 'Kyra', 'Kerkhofs', '2022AbstractIn', 'eukaryotes,', 'many', 'RNA', 'species', 'are', 'transcribed', 'as', 'precursor', 'molecules', 'and', 'require', 'post-transcriptional', 'processing', 'to', 'become', 'a', 'functional', 'mature', 'RNA.', 'This', 'includes', 'the', 'very', 'abundant', 'cytoplasmic', 'transfer', 'RNA', '(tRNAs)', 'and', 'ribosomal', 'RNA', '(rRNAs)', 'that', 'function', 'in', 'translation', 'of', 'messenger', 'RNAs', '(mRNAs).', 'In', 'addition,', 'other', 'non-coding', 'RNAs', 'also', 'require', 'extensive', 'processing', 'and', 'assembly', 'into', 'ribonucleoproteins', '(RNPs),', 'including', 'small', 'nuclear', 'RNAs', '(snRNAs)', 'that', 'function', 'in', 'pre-mRNA', 'splicing', 'and', 'small', 'nucleolar', 'RNAs', '(snoRNAs)', 'involved', 'in', 'rRNA', 'and', 'non-coding', 'RNA', 'processing', 'and', 'modification.', 'The', 'ciliate', 'Tetrahymena', 'thermophila', 'is', 'a', 'highly', 'studied', 'and', 'well-characterized', 'model', 'organism', 'with', 'many', 'groundbreaking', 'discoveries.', 'However,', 'many', 'important', 'non-coding', 'RNA', 'processing', 'pathways', 'remain', 'elusive', 'in', 'this', 'organism.', 'We', 'used', 'molecular', 'biology,', 'biochemistry,', 'cell', 'biology', 'and', 'bioinformatics', 'to', 'investigate', 'the', 'role', 'of', 'a', 'novel', 'La', 'protein', 'named', 'Macronucleus', 'Localized', 'Protein', 'of', 'unknown', 'function', '(Mlp1)', 'in', 'pre-tRNA,', 'snoRNA', 'and', 'snRNA', 'biogenesis.', 'RNA', 'polymerase', 'III', 'transcribes', 'nascent', 'pre-tRNA', 'resulting', 'in', 'the', 'formation', 'of', 'the', '5’-leader,', '3’-trailer', 'and', 'typical', '3’-terminal', 'uridylate', 'sequence']\n"
     ]
    }
   ],
   "source": [
    "results_string = \"\".join(result)\n",
    "\n",
    "print(results_string[0:500])\n",
    "      \n",
    "print(results_string[0:1500].replace('\\n',\" \").split(\" \"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "c7d94a07",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-03T16:15:18.199016Z",
     "start_time": "2022-11-03T16:15:18.171091Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Processing': 6, 'of': 1338, 'non-coding': 40, 'RNAs': 61, 'by': 404, 'Mlp1': 459, 'in': 1134, 'Tetrahymena': 38, 'thermophilaKyra': 1, 'KerkhofsA': 1, 'dissertation': 1, 'submitted': 1, 'to': 582, 'the': 1880, 'faculty': 1, 'graduate': 1, 'studies': 10, 'partial': 58, 'fulfillment': 1, 'requirements': 1, 'for': 450, 'degree': 3, 'Doctor': 1, 'Philosophy': 1, 'Graduate': 1, 'Program': 1, 'Biology': 1, 'York': 4, 'University': 1, 'Toronto,': 4, 'OntarioDecember': 1, '2022©': 1, 'Kyra': 1, 'Kerkhofs': 4, '2022AbstractIn': 1, 'eukaryotes,': 12, 'many': 13, 'RNA': 451, 'species': 32, 'are': 309, 'transcribed': 77, 'as': 274, 'precursor': 17, 'molecules': 7, 'and': 1429, 'require': 8, 'post-transcriptional': 47, 'processing': 137, 'become': 2, 'a': 685, 'functional': 32, 'mature': 140, 'RNA.': 16, 'This': 43, 'includes': 6, 'very': 8, 'abundant': 6, 'cytoplasmic': 7, 'transfer': 7, '(tRNAs)': 2, 'ribosomal': 21, '(rRNAs)': 1, 'that': 347, 'function': 88, 'translation': 13, 'messenger': 4, '(mRNAs).': 1, 'In': 101, 'addition,': 21, 'other': 68, 'also': 83, 'extensive': 10, 'assembly': 65, 'into': 60, 'ribonucleoproteins': 3, '(RNPs),': 1, 'including': 50, 'small': 54, 'nuclear': 34, '(snRNAs)': 6, 'pre-mRNA': 17, 'splicing': 45, 'nucleolar': 12, '(snoRNAs)': 7, 'involved': 27, 'rRNA': 83, 'modification.': 2, 'The': 262, 'ciliate': 3, 'thermophila': 174, 'is': 388, 'highly': 46, 'studied': 17, 'well-characterized': 1, 'model': 10, 'organism': 1, 'with': 415, 'groundbreaking': 1, 'discoveries.': 1, 'However,': 25, 'important': 37, 'pathways': 4, 'remain': 5, 'elusive': 1, 'this': 97, 'organism.': 1, 'We': 114, 'used': 60, 'molecular': 4, 'biology,': 1, 'biochemistry,': 1, 'cell': 21, 'biology': 1, 'bioinformatics': 4, 'investigate': 26, 'role': 12, 'novel': 52, 'La': 297, 'protein': 241, 'named': 7, 'Macronucleus': 3, 'Localized': 1, 'Protein': 27, 'unknown': 7, '(Mlp1)': 4, 'pre-tRNA,': 6, 'snoRNA': 64, 'snRNA': 127, 'biogenesis.': 4, 'polymerase': 167, 'III': 69, 'transcribes': 13, 'nascent': 96, 'pre-tRNA': 148, 'resulting': 27, 'formation': 32, '5’-leader,': 6, '3’-trailer': 94, 'typical': 25, '3’-terminal': 51, 'uridylate': 101, 'sequence.': 12, 'Pre-tRNA': 7, 'has': 72, 'been': 49, 'well-described': 3, 'humans,': 5, 'yeasts': 4, 'eukaryotic': 35, 'where': 32, '5’-': 18, '3’-extensions': 5, '(i.e.': 12, '5’-leader': 63, '3’-trailer)': 1, 'removed': 9, 'sequentially.': 1, 'proteins': 144, 'have': 78, 'high': 19, 'binding': 202, 'affinity': 44, 'sequences': 79, 'through': 94, 'concomitant': 2, 'between': 111, 'conserved': 81, 'domains': 19, 'motif': 45, '(LaM)': 7, 'RNA-recognition': 5, 'motif-1': 7, '(RRM1).': 1, 'Therefore,': 8, 'first': 21, 'associate': 15, 'pre-tRNAs.': 8, 'next': 38, 'step': 15, 'involves': 15, 'endonucleolytic': 17, 'followed': 88, '3’-trailer.': 4, 'absence': 22, 'protein,': 14, 'pathway': 21, 'reversed': 1, 'rapidly': 4, 'trimmed': 4, '3’-exonuclease,': 2, 'processing.': 9, 'Chapter': 14, 'II,': 1, 'we': 163, 'characterize': 1, 'lacking': 11, 'RNA-binding': 28, 'domain': 37, 'required': 32, 'performed': 57, 'assays': 11, 'confirm': 16, 'performs': 2, 'functions': 36, 'including,': 1, 'preferential': 7, 'pre-tRNAs': 102, 'compared': 52, 'tRNAs,': 7, 'uridylate-specific': 3, 'residues': 39, 'LaM': 64, 'chaperone': 36, 'activity': 23, 'promote': 8, 'maturation': 29, 'misfold': 2, 'contrast': 4, 'depletion': 51, 'results': 33, 'stabilization': 10, 'instead': 9, 'rapid': 2, 'trimming': 3, '3’-exonucleases,': 2, 'indicating': 40, 'linked': 15, 'fundamentally': 1, 'different': 67, 'mechanism': 7, 'tRNA': 174, 'T.': 197, 'thermophila.': 27, 'shown': 97, 'II': 79, 'transcripts': 50, 'association': 21, 'forms': 6, 'spliceosomal': 22, 'snRNAs': 85, 'ending': 7, '3’-uridylate': 33, 'sequences.': 8, 'III,': 4, 'found': 131, 'associates': 35, 'site': 39, 'core': 26, 'component': 9, 'snRNP.': 3, 'More': 10, 'specifically,': 4, 'interact': 8, 'U4/U6': 51, 'di-snRNP': 31, 'complex,': 10, 'which': 127, 'formed': 16, 'during': 31, 'reaction,': 1, 'affect': 3, 'complex.': 8, 'diminished': 3, 'efficiency': 8, 'pre-mRNAs,': 1, 'genuine': 51, 'chapter': 1, 'IV,': 2, 'described': 55, 'all': 37, 'previously': 66, 'snoRNAs': 132, 'components': 20, 'snoRNPs': 10, 'guide': 15, 'modification': 27, 'rRNA,': 10, 'tRNAs.': 12, 'metric': 4, 'predict': 4, 'validate': 4, 'expression.': 4, 'snoRNP': 45, 'led': 7, 'us': 7, 'if': 41, 'snoRNPs.': 2, 'did': 11, 'not': 84, 'find': 2, 'striking': 1, 'difference': 10, 'upon': 20, 'depletion.': 3, 'could': 37, 'indicate': 16, 'redundantly': 4, 'another': 7, 'only': 35, 'minor': 4, 'effects': 5, 'on': 229, 'function.': 6, 'Acknowledgements': 1, 'Table': 31, 'Contents': 1, 'List': 5, 'FiguresChapter': 2, 'I:': 3, 'Literature': 3, 'ReviewChapter': 2, 'II:': 6, 'Altered': 6, 'distinct': 15, 'unusual': 10, 'thermophilaChapter': 7, 'III:': 5, 'Genuine': 6, 'affects': 25, 'IV:': 5, 'assemblyChapter': 3, 'V:': 3, 'Conclusion': 3, 'future': 5, 'perspectives': 2, 'Supplementary': 60, 'TablesChapter': 2, 'AbbreviationsChapter': 1, 'Review': 1, 'Multiple': 5, 'noncoding': 4, 'synthesis': 5, 'achieved': 2, 'three': 20, 'multi-subunit': 1, 'polymerases': 2, 'eukaryotes': 23, '(Carter': 2, '&': 180, 'Drouin': 1, '2009).': 14, 'coding': 14, '(mRNAs)': 2, '(Guiro': 3, 'Murphy': 4, '2017).': 5, 'I': 11, 'produces': 2, 'single': 13, '47S': 2, '(rRNA)': 1, '(35S': 1, 'yeast)': 1, 'processed': 30, '18S,': 4, '5.8S': 19, '28S': 4, 'rRNAs': 6, '(18S,': 1, '25S': 1, 'yeast),': 1, 'large': 14, 'subunits': 9, '(Klinge': 4, 'Woolford': 4, '2019),': 4, 'short': 12, 'transcripts,': 8, '5S': 11, 'part': 10, 'subunit': 9, '(Dieci': 3, 'et': 389, 'al.': 382, '2007).': 7, 'similar': 27, 'they': 3, 'produce': 6, 'relatively': 6, 'number': 27, 'expressed': 14, '(Paule': 1, 'White': 1, '2000),': 2, 'while': 27, '20,000': 10, 'genes': 33, 'approximately': 12, '50%': 4, 'transcription': 79, 'contributing': 2, 'production': 4, 'mRNAs': 11, 'encoding': 12, '(Warner': 1, '1999).': 6, 'Ribosome': 2, 'biogenesis': 9, '(1)': 4, 'transcription,': 5, 'processing,': 7, 'snoRNAs,': 21, '(2)': 4, 'snRNAs,': 10, 'export,': 1, 'import': 3, '(RPs),': 1, '(3)': 4, '(40S': 1, '–': 63, 'consisting': 7, '18S': 3, '33': 1, '(RPS))': 1, '(60S': 1, '28S,': 1, '46': 1, '(RPL)),': 1, '(4)': 3, 'export': 5, 'cytoplasm': 9, '(Figure': 172, '1)': 5, '(Henras': 3, '2015;': 3, 'Lafontaine': 1, '2015).': 11, 'Polypeptide': 1, 'relies': 2, 'coordinated': 1, 'types': 5, 'must': 2, 'be': 63, 'tightly': 1, 'regulated': 1, 'at': 239, 'levels': 33, 'ensure': 3, 'normal': 6, 'growth': 2, 'proliferation': 1, '(Moir': 1, 'Willis': 1, '2013).': 9, 'Translation': 2, 'an': 126, 'intricate': 1, 'process': 7, 'each': 29, 'puzzle': 1, 'piece': 1, 'needs': 2, 'fit': 2, 'exactly': 1, 'right': 1, 'place': 12, 'order': 11, 'maintain': 1, 'homeostasis.': 1, 'focus': 1, 'thesis': 2, 'U6': 83, '(snRNA)': 2, 'U1,': 26, 'U2,': 33, 'U4': 49, 'U5': 46, 'snoRNAs.': 13, 'Figure': 159, 'legend': 28, 'pageFigure': 16, '1.': 4, 'polymerases.': 1, 'nucleus': 14, 'multiple': 42, 'such': 10, '(not': 6, 'shown)': 4, '(snoRNAs).': 2, 'factors': 6, 'translated': 2, 'imported': 1, 'ribosome': 4, 'assembly.': 11, 'located': 29, 'nucleolus': 14, 'polycistronic': 9, 'transcript,': 3, 'pre-rRNA,': 1, 'rRNA.': 12, 'Further': 3, 'occurs': 9, 'modifications': 42, '60S.': 1, '40S': 2, '60S,': 1, 'respectively,': 6, 'assembled': 10, 'transported': 3, 'proteins.': 13, 'adapted': 14, 'from': 213, '(Lafontaine': 1, 'Transcription': 6, 'RNAsRNA': 1, 'takes': 7, 'DNA': 43, '(rDNA)': 1, 'encoded': 26, 'copies': 13, 'throughout': 4, 'genome.': 4, 'For': 27, 'example,': 16, 'yeast': 41, 'S.': 75, 'cerevisiae': 38, 'encodes': 4, '150': 19, 'per': 21, 'haploid': 1, 'genome': 21, 'humans': 5, '300': 8, '400': 3, 'diploid': 1, 'I,': 2, 'containing': 85, '(see': 33, 'external': 3, '(5’-ETS': 2, '3’-ETS)': 2, 'internal': 13, 'spacers': 1, '(ITS-1': 2, 'ITS-2)': 2, 'whereas': 7, 'later)': 2, 'pre-rRNA': 8, 'initiated': 3, 'co-transcriptionally': 5, 'complex': 80, 'endo-': 3, 'exonucleolytic': 6, 'steps': 9, '(reviewed': 14, 'Henras': 1, 'al.,': 3, 'addition': 31, 'step,': 1, 'obtain': 24, 'nucleotide': 30, 'up': 8, '200': 5, '(Lestrade': 1, 'Weber': 1, '2006).': 16, 'Most': 6, '2017)': 4, 'base': 47, 'pairing': 17, 'target': 11, 'Two': 3, 'classes': 5, 'box': 130, 'C/D': 62, 'H/ACA': 48, 'direct': 16, '2’-hydroxyl-methylation': 1, '(2’-O-methylation)': 1, 'backbone': 6, '2’-O': 2, 'position': 14, 'sugar': 1, 'ring': 7, 'pseudouridylation': 10, '5’-ribosyl': 1, 'isomer': 2, 'uridine': 3, 'pseudouridine': 5, 'or': 87, 'Ψ,': 1, '2)': 5, '(Decatur': 6, 'Fournier': 6, '2002).': 25, 'Post-transcriptional': 7, 'regions': 20, 'often': 6, 'functionally': 6, '.': 69, 'Schematic': 13, 'representation': 16, '2’-O-methylation': 21, '(Ψ)': 2, 'rotation': 3, '(U)': 1, 'converted': 3, 'synthetases.': 1, '2’O-methylation': 3, 'methyl': 8, 'group': 10, '2’-O-position': 1, '2’-O-methyltransferases.': 1, '(Karijolich': 4, 'Yu': 6, '2010).snRNA': 1, 'maturationRNA': 1, 'pre-mRNAs': 6, '5’-guanine-N7': 2, '(m7G)': 5, 'cap': 36, 'via': 5, 'inverted': 3, '5’-5’': 2, 'triphosphate': 2, 'bridge': 2, 'nucleoside': 1, 'transcript': 29, '3a)': 1, 'Ghosh': 1, 'Lima,': 1, '2010).': 9, 'm7G': 13, 'added': 19, 'mRNA': 23, 'functions,': 3, 'protection': 21, 'against': 21, '5’-3’': 1, 'exonucleases,': 4, 'efficient': 5, 'mRNAs,': 2, 'polyadenylation': 1, '3’-end,': 1, '(Lewis': 1, 'Izaurflde': 1, '1997;': 12, 'Pabis': 1, 'RNAs,': 4, 'Well-characterized': 1, 'include': 14, 'U5.': 2, 'Mature': 10, 'contain': 37, '2,2,7-trimethylguanosine': 3, '(m3G)': 5, 'further': 27, 'dimethylation': 3, 'N2': 2, 'guanine': 3, 'structure': 70, '3b)': 2, '(Ghosh': 2, 'Lima': 2, '5’-cap,': 1, 'two': 42, 'guided': 2, 'specific': 30, 'subset': 5, 'Cajal': 7, 'bodies,': 1, 'body-specific': 1, '(scaRNAs)': 1, '2010;': 1, 'Jády': 1, 'Kiss': 2, '2001).': 9, 'evolution': 2, 'splicing,': 6, 'snRNA/snRNA': 1, 'pre-mRNA/snRNA': 1, 'hybridization': 4, '4)': 1, 'Synthesis': 5, 'm3G-cap': 25, 'transcripts.a': 1, 'Transcripts': 4, 'produced': 11, 'linkage': 1, 'nucleoside.': 1, 'b': 29, 'A': 52, 'post-transcriptionally': 6, 'obtained': 51, 'structure.': 4, 'm3G': 7, 'some': 13, 'c': 20, 'γ-methylphosphate': 2, 'primary': 19, 'γ-methyltransferases.': 1, '7SK': 11, '2010).The': 1, 'spliceosome': 17, 'consists': 8, 'five': 8, 'uridine-rich': 2, 'U4,': 6, '4),': 2, '100s': 1, '(Jurica': 1, 'Moore': 2, '2003).': 11, 'While': 10, 'U5,': 1, 'contains': 32, 'γ-monomethyl': 6, '(Singh': 5, 'Reddy': 7, '1989).': 2, 'Methylation': 1, 'γ-phosphate': 2, '5’-triphosphate': 15, 'catalyzed': 3, 'γ-methyltransferase': 1, 'transferring': 2, 'S-adenosylmethionine': 2, 'oxygen': 7, '5’-end': 4, '(Gupta': 2, '1990;': 5, 'Shimba': 2, '1994).': 3, 'Likewise,': 1, '1990).Figure': 1, 'secondary': 27, 'structures': 18, 'snRNAs.': 15, 'shown.': 2, 'methylations': 1, 'highlighted': 32, 'rectangle': 2, 'circle': 1, 'surrounding': 5, 'individual': 4, 'nucleotide,': 3, 'respectively.': 14, 'Sm-binding': 7, 'sites': 19, 'dark': 23, 'gray': 1, 'boxes.': 1, '2010).snoRNA': 1, 'IIsnoRNAs': 1, 'abundantly': 2, 'within': 40, 'ribonucleoprotein': 6, '(RNP)': 4, 'complexes': 10, 'tRNAs': 64, '(Kiss': 7, 'major': 8, 'exist': 5, 'based': 33, 'sequence': 99, 'structural': 11, 'elements,': 2, 'Box': 6, 'guides': 1, 'pseudouridylation.': 2, 'Different': 3, 'employ': 3, 'mechanisms': 3, 'independent': 27, 'genes,': 4, 'introns': 14, '(Kufel': 6, 'Grzechnik': 3, '2019)': 2, '5).': 5, 'expresses': 1, '76': 1, '(Piekna-Przybylska': 2, '2007)': 1, 'mostly': 4, 'employs': 1, 'genes.': 8, '(8': 1, 'snoRNAs)': 2, 'organized': 1, 'polycistrons': 1, '(5': 1, 'clusters': 2, '17': 1, 'contrast,': 13, 'encode': 4, 'majority': 10, 'same': 21, 'orientation': 1, 'human': 36, 'Host': 1, '5’-terminal': 2, 'oligo': 1, 'pyrimidine': 3, '(5’-TOP)': 1, '(Ojha': 1, '2020;': 1, 'Smith': 1, 'Steitz': 6, '1998;': 3, 'Tycowski': 3, '1993).': 2, 'precursors': 3, 'undergo': 6, '5’': 8, '3’-ends.': 3, 'fully': 6, 'lack': 20, 'poly(A)': 3, 'tail': 15, '60': 3, 'nucleotides': 26, 'long': 4, 'Falaleeva': 2, 'Stamm': 4, '2013).Figure': 1, 'Genomic': 6, 'organization': 5, '(snoRNA)': 1, 'species.': 10, 'generally': 6, 'follows': 2, 'certain': 7, 'branch': 4, 'evolutionary': 3, 'tree.': 1, 'example': 3, 'under': 9, 'regulation': 4, 'its': 9, 'own': 1, 'promotor': 3, 'terminator,': 1, 'Drosophila': 2, 'melanogaster': 4, 'Caenorhabditis': 1, 'elegans': 2, 'transcript.': 4, 'vertebrates,': 2, '(lncRNAs).': 1, 'P:': 1, 'promotor,': 1, 'T:': 1, 'terminator.': 1, '2019).RNA': 1, 'transcriptsRNA': 1, 'make': 4, '15%': 2, '(Turowski': 1, 'Tollervey': 2, '2016)': 5, 'total': 27, 'cellular': 10, 'tRNAs),': 1, 'RNase': 79, 'MRP': 2, 'P': 26, 'RNAs),': 1, 'co-translational': 1, 'translocation': 1, 'polypeptide': 2, 'chains': 2, 'endoplasmic': 1, 'reticulum': 1, '(ER)': 1, '7SL': 1, 'RNA),': 4, 'elongation': 2, 'factor': 10, 'P-TEFb': 1, 'RNA)': 1, 'snRNA)': 1, 'Dieci': 2, 'species,': 4, 'II.': 2, 'few': 8, 'examples': 2, 'telomerase': 7, '(TER)': 1, 'functioning': 7, 'telomeric': 1, 'extension': 10, 'ciliates,': 1, 'U3': 12, 'plants': 3, 'cerevisiae,': 2, 'A.': 4, 'thaliana,': 3, 'C.': 1, 'D.': 9, 'most': 32, 'gene': 13, 'products': 9, 'adaptors': 1, 'codon': 8, 'corresponding': 19, 'amino': 44, 'acid': 38, '3’-CCA': 1, '(O’Donoghue': 1, '2018).': 15, 'DNA-binding': 3, 'factor,': 1, 'initiation': 3, 'core.': 2, 'uses': 3, 'intragenic': 1, 'promotors': 1, 'B': 11, 'localized': 3, 'genomic': 31, 'distance': 1, 'these': 70, 'elements': 3, 'variable': 8, 'depending': 3, 'identity': 6, '6)': 1, '(Arimbasseri': 3, 'Maraia': 13, 'Termination': 1, 'encountering': 3, 'stretch': 29, 'As': 16, 'termination': 46, 'signal': 8, 'leading': 21, 'incorporation': 2, 'every': 4, 'high-affinity': 11, '(Fairley': 2, '2005).': 5, 'Another': 10, 'feature': 5, 'uncapped': 1, 'sequence)': 1, '(Yan': 1, '2022)': 6, '3a).': 1, '6.': 2, 'gene.': 3, 'Nascent': 3, 'TFIIIC': 1, 'box,': 3, 'TFIIIB': 1, 'TATA-box.': 1, 'recruitment': 3, 'termination.': 4, 'adenylates.': 1, 'stuttering': 2, 'result,': 3, 'Us': 1, '3’-end': 47, '(Porter': 2, '2021).tRNAs': 1, 'genomeThe': 1, 'more': 37, 'than': 27, 'nuclear-encoded': 1, 'grouped': 7, '20': 19, 'families': 2, 'CCA': 7, '(Chan': 5, 'Lowe,': 3, '2016': 3, '-': 11, 'http://gtrnadb.ucsc.edu': 2, 'GRCh37/hg19).': 3, 'groups': 6, 'anticodons': 1, 'can': 33, 'decode': 3, 'isoacceptors.': 1, 'alanine': 2, '(codons': 2, 'GCU,': 2, 'GCG,': 2, 'GCC,': 1, 'GCA,)': 1, 'decoded': 4, 'isoacceptors': 3, 'AlaAGC,': 2, 'AlaCGC': 2, 'AlaUGC,': 1, 'phenylalanine': 1, 'TTC': 1, 'TTT)': 1, 'one': 14, 'isodecoder:': 1, 'PheGAA': 1, 'Typically,': 2, 'acids': 14, 'isodecoders,': 1, 'ranging': 4, 'isodecoders': 5, 'arginine': 1, 'leucine': 1, 'humans.': 2, 'divided': 5, 'among': 1, '48': 1, 'isoacceptor': 5, 'decoding': 3, '61': 1, 'sense': 1, 'codons': 3, 'acids.': 2, 'above': 2, 'mentioned': 2, 'AlaUGC': 1, 'GCA': 1, 'synonymous': 1, 'alanine,': 1, 'GCC': 1, 'because': 7, 'degeneracy': 1, 'genetic': 2, 'code,': 1, 'particularly': 1, 'wobble': 3, 'anticodon': 14, '(position': 1, '34': 1, 'tRNA)': 1, 'allowing': 2, 'non-cognate': 1, 'following': 28, 'Watson-Crick': 2, 'rules': 1, 'Lowe': 3, '2016;': 4, 'Pinkard': 1, '2020).': 15, 'Various': 1, 'correspond': 3, 'but': 32, 'composition': 4, 'body': 7, 'tRNA.': 7, 'LeuCAG': 2, '9': 2, 'genome,': 1, 'however,': 23, 'Regardless': 1, 'isodecoder': 2, 'variability,': 1, 'retain': 3, 'ability': 5, 'fold': 29, 'molecule': 5, 'translation.': 3, 'Certain': 1, 'changes': 9, 'D-arm': 2, 'T-arm': 2, 'since': 12, 'promoter': 2, 'A-': 1, 'B-box,': 1, 'respectively': 7, '(Geslain': 1, 'Pan': 1, 'Expression': 7, 'vary': 1, 'tissues': 1, '(Pinkard': 1, 'adopt': 3, 'classic': 10, 'cloverleaf': 3, 'structureThe': 1, 'steps,': 2, 'starting': 3, 'III.': 2, 'Next,': 28, 'removal': 14, 'sequences,': 3, 'intron': 34, 'nucleotidyltransferase': 4, '7a)': 2, '(Maraia': 5, 'Lamichhane': 4, '2011).': 13, 'Throughout': 1, 'process,': 3, 'post-transcription': 1, 'enhance': 8, 'correct': 14, 'folding': 25, 'L-shaped': 1, 'tertiary': 7, '(Helm': 1, 'extremely': 1, 'stable': 4, 'half-life': 1, 'around': 7, '100': 13, 'h': 5, '(Choe': 1, 'Taylor': 1, '1972).': 1, 'length': 20, '74': 2, '93': 1, 'nucleotides,': 1, 'despite': 10, 'overall': 2, 'conservation,': 2, 'four': 17, 'stems': 2, 'pairing,': 2, 'acceptor': 2, 'stem,': 3, 'D-stem,': 1, 'T-stem': 1, 'and,': 1, 'tRNA,': 5, 'stem': 19, 'present': 13, 'example)': 1, '7b)': 1, '(Berg': 2, '2019).': 6, 'Additionally,': 2, 'dihydrouridine': 1, '(D)-loop,': 1, 'loop': 15, 'TΨC': 1, '(T)-loop': 1, '7b).': 2, 'Formation': 3, 'sufficient': 6, 'Watson–Crick': 2, 'A-T': 1, 'C-G': 1, 'G-U': 1, 'pairs': 3, 'opposing': 2, '2019).The': 1, 'discriminator': 24, '3’-most': 2, 'terminal': 6, 'overhang': 1, 'serves': 2, 'preceding': 2, 'attachment': 2, 'transferred': 7, 'aminoacyl-tRNA': 2, 'synthetases': 2, '(Pang': 2, '2014).': 3, 'interactions': 50, 'factors,': 1, 'ribosomes,': 1, 'translation,': 2, 'deliver': 1, 'isotype': 10, 'Wolfenden': 1, 'pre-tRNA.': 4, 'extensions': 5, 'yellow': 6, 'blue,': 11, 'uridylates': 28, 'blue.': 2, 'La-bound': 2, 'endonuclease': 11, 'Z': 18, 'La-independent': 9, '8).': 2, 'Following': 13, 'obtains': 6, '-CCA': 3, 'later).': 3, 'intervening': 4, '(shown': 11, 'orange)': 3, 'requires': 5, 'adopts': 4, 'L-shape.': 1, 'loops': 3, 'result': 17, 'pairing.': 3, 'color': 6, 'code': 1, 'highlights': 1, 'light': 25, 'green,': 2, 'cyan,': 1, 'orange,': 5, 'triplet': 1, 'red,': 3, 'cyan': 1, 'N73': 1, 'upstream': 17, 'yellow).': 2, 'During': 6, '2021).While': 1, 'composed': 2, 'building': 1, 'blocks': 1, '(A,': 2, 'U,': 1, 'C': 11, 'G)': 1, 'form': 15, 'basis': 3, 'alternate': 8, 'free': 5, 'energy': 1, 'form.': 1, 'These': 23, 'misfolded': 10, 'kinetically': 1, 'trapped': 3, 'conformation': 3, 'non-functional': 1, 'phenomenon': 1, 'problem': 2, 'resolved': 1, 'chaperones': 4, 'prevent': 1, 'getting': 1, 'unfolding': 2, 'refolding': 1, 'stabilizing': 3, 'folds': 2, 'Herschlag,': 1, '1995).': 2, 'eukaryotesIn': 1, 'aminoacyl-tRNA,': 1, 'quality': 2, 'control': 37, 'required.': 2, 'cases': 1, 'need': 5, 'accurately': 1, 'accurate': 1, '(Pan': 2, 'detail': 2, 'sections.In': 1, 'maturation,': 1, 'substrates': 15, 'ATP(CTP):tRNA': 1, 'catalyzes': 3, '(Tomita': 2, 'Yamashita': 2, 'CCA,': 1, 'charging': 3, 'appropriate': 9, 'acetyltransferases': 1, 'beyond': 1, 'scope': 1, 'thesis.Processing': 1, 'life,': 1, 'completed': 14, 'phosphodiester': 1, 'hydrolysis': 1, '5’-monophosphate': 1, 'terminus': 1, '(Gößringer': 2, '2012).': 23, 'enzyme': 27, 'ribonuclease': 1, 'possesses': 5, 'catalytic': 9, 'bacteria,': 2, 'Archaea': 1, '(Chen': 2, 'Pace': 1, '1997).': 17, 'Generally,': 4, 'final': 6, 'maturation.': 1, 'One': 3, 'exception': 1, 'standard': 7, 'parasitic': 1, 'archeon': 1, 'Nanoarchaeum': 1, 'equitans': 1, '(Randau': 2, '2008)': 3, 'does': 5, 'start': 1, 'extensions.': 2, 'do': 4, 'RNaseP': 1, 'therefore': 11, '5’-triphosphate.': 1, 'So': 1, 'far,': 1, 'identified': 50, 'any': 14, '2008).Processing': 1, 'either': 16, 'exonuclease': 5, 'Rex1': 5, 'determined': 28, 'proteinOnce': 1, 'P,': 3, 'proceeds': 2, 'cleave': 1, 'remove': 5, '(N73,': 1, 'see': 1, '7).': 4, '3’': 12, 'end-matured': 2, 'concomitantly,': 1, 'release': 3, 'was': 240, 'bound': 16, 'stretch.': 3, 'consecutive': 4, 'system': 11, 'associated': 30, 'termed': 2, 'La-dependent': 9, '8': 5, '1).': 6, 'Here,': 1, 'prior': 9, '(Van': 4, 'Horn': 7, 'Yoo': 4, 'Wolin': 17, 'Interestingly,': 9, 'situations': 1, 'deleted': 4, 'limiting': 3, 'unprotected,': 2, 'exonuclease-1': 1, '(Rex1)': 1, 'processes': 2, '5’-trailer': 2, 'cleavage': 17, '(Copela': 7, '2008).': 8, 'pathway.Figure': 1, 'assists': 6, '3’-uridylates': 2, 'end': 13, 'dAs': 5, 'signal.': 2, 'binds': 14, '3’-UUU-OH': 2, 'provided': 17, 'length.': 2, 'Normal': 1, '(#': 1, '(#2)': 1, 'end-processed': 2, 'main': 4, 'presence': 30, 'exonuclease.': 1, 'pathway,': 2, 'there': 1, 'no': 49, 'binding.': 4, '(#4),': 1, 'structurally': 4, 'impaired': 6, 'targeted': 2, 'degradation': 6, 'surveillance:': 1, 'polyadenylated': 3, 'TRAMP': 1, 'exosome': 1, 'degradation.': 2, 'enhances': 3, '(#3),': 1, '2011).As': 1, 'before,': 1, 'after': 13, 'dA’s': 2, 'template': 3, 'strand': 4, 'minimal': 4, 'varies': 1, 'H.': 17, 'sapiens': 14, '4': 9, 'dAs,': 1, 'pombe': 37, 'least': 8, '5': 40, '6': 3, '3-6': 1, '3': 11, '(Mathews': 2, 'Francoeur': 5, '1984;': 6, 'Stefano': 4, '1984)': 1, '4-5': 1, 'Sla1p': 10, '(Huang': 7, 'correlates': 5, 'After': 1, 'heterogenous': 1, 'population': 1, 'varying': 2, 'lengths': 9, 'length,': 1, 'i.e.': 2, 'dA': 1, 'Since': 9, 'higher': 13, 'longer': 6, 'tails,': 2, 'shorter': 5, 'will': 10, 'unbound': 5, 'when': 16, '(Stefano': 5, '1984).': 8, 'binding,': 8, 'both': 47, 'take': 4, 'simultaneously': 1, '1': 44, 'enzymes': 5, 'Folding': 1, '(Leamy': 1, 'conformations': 2, 'occasionally': 1, '(Vakiloroayaei': 1, 'surveillance': 4, 'rescued': 2, '3)': 3, '(discussed': 1, 'later': 1, 'section)': 1, '(Chakshusmathi': 2, 'retention': 6, 'transcripts.': 7, 'localization': 11, '(NLS)': 2, 'retains': 2, 'avoiding': 1, 'premature': 2, 'incomplete': 3, 'incorrectly': 1, '(Intine': 3, '2002;': 8, 'Bayfield': 11, '2007).In': 1, 'conclusion,': 4, 'determines': 2, 'usage': 1, 'accessibility': 6, 'inhibited': 1, 'access': 5, '3’-exonuclease': 6, '3’-maturation': 1, 'pathway.': 1, '3-terminus': 1, 'allows': 2, '3’-exonucleases': 9, 'Both': 6, '(Yoo': 12, 'Lhp1': 14, 'Saccharomyces': 4, 'essential': 10, 'deletion': 14, 'strains,': 3, '1994)': 2, 'dependent': 3, 'modificationsPost-transcriptional': 1, 'enzymatic': 1, 'covalent': 1, 'chemical': 1, 'bases': 3, 'alter': 2, 'their': 16, 'properties': 1, 'ways': 1, '(ref).': 3, 'Modified': 1, 'La-associated': 3, 'pre-tRNAs,': 6, 'occur': 3, 'early': 3, '(Hendrick': 3, '1981).': 2, 'Nuclear-encoded': 1, 'highest': 3, 'density': 1, 'average': 6, '13': 1, 'molecule,': 1, 'mt-tRNAs': 1, 'diverse': 2, '90': 3, 'across': 1, 'archaea,': 1, '(Cantara': 1, 'several': 10, 'stability': 4, '(Lorenz': 1, '2017;': 2, 'El': 1, 'Yacoubi': 1, 'advantageous': 1, 'reverse': 16, 'transcriptase-based': 1, 'sequencing': 18, 'methods': 3, 'severely': 1, 'hindered': 1, 'developed': 5, 'circumvent': 1, 'block': 2, 'generated': 23, 'cDNA': 16, 'transcriptase.': 1, 'First,': 7, 'AlkB-facilitated': 1, 'methylation': 4, '(ARM-seq)': 1, 'utilizes': 2, 'E.': 5, 'coli': 3, 'AlkB': 1, 'removes': 2, 'avoid': 7, 'generation': 13, 'truncated': 2, '(Cozen': 1, 'Zheng': 1, 'Second,': 2, 'thermostable': 1, 'transcriptase': 2, '(TGIRT)': 1, 'processivity': 1, 'thereby': 8, 'full-length': 8, '(Zheng': 3, 'Splicing': 2, 'intron-containing': 3, 'tRNAsA': 1, 'spliced': 9, 'before': 6, 'substrate': 14, 'adapter': 4, 'typically': 8, 'vertebrates': 3, 'complementary': 19, '(Johnson': 1, 'Abelson': 1, '1983)': 1, 'Correct': 2, 'full': 10, 'set': 2, 'provide': 7, 'translational': 1, 'machinery': 2, 'synthesis.': 2, 'Importantly,': 1, 'retaining': 3, 'incapable': 1, 'adopting': 2, 'defective': 9, 'assist': 4, 'modesThe': 1, 'known': 9, 'Sjogren’s': 3, 'Syndrome': 1, 'antigen': 1, '(SSB),': 1, 'autoantigen': 2, 'patients': 2, 'suffering': 2, 'systemic': 2, 'lupus': 2, 'erythematosus': 2, 'syndrome': 2, '(Mattioli': 2, 'Reichlin': 2, '1974a;': 1, 'Alspaugh': 2, 'Tan': 1, '1975).': 2, 'Patient': 1, 'antibodies': 14, 'were': 182, 'immunoprecipitation': 25, 'revealed': 19, '1981;': 2, 'Lerner': 2, '1981)': 2, 'particular': 3, 'interacts': 36, 'still': 4, '(encoded': 1, 'adenylate': 2, 'gene)': 1, 'earlier)': 1, 'Mathews': 3, 'Homologues': 1, '1994;': 4, 'Van': 3, '1997)': 3, 'metazoans': 2, '(Park': 1, '2006;': 9, 'Arhin,': 1, 'Shen,': 2, 'Pérez,': 1, '2005;': 5, 'Scherly': 1, '1993;': 1, 'Pellizzoni': 1, '1996)': 3, 'insects': 1, 'Pardigon': 1, 'Strauss': 1, '1996;': 4, 'Bai': 2, 'Tolias': 2, '2000)': 2, '(Fleurdépine': 1, '2007),': 1, 'dispensable': 1, 'Fleurdépine': 2, '2007;': 3, 'Park': 2, 'best': 1, 'N-terminal': 8, 'domain,': 4, 'C-terminal': 28, 'module,': 3, 'unit': 1, 'juxtaposed': 1, 'domains,': 3, 'recognition': 12, '(RRM)': 3, 'separated': 15, 'linker': 1, 'top': 7, 'panel)': 1, '(Teplova': 6, 'second': 6, 'RRM': 18, '(RRM2),': 1, 'disordered': 3, 'region': 24, '9)': 2, '(Jacks': 1, 'conservation': 20, 'module': 21, '(LaM': 5, '+': 6, 'RRM1)': 4, 'La-related': 7, '(LARP)': 1, 'family.': 2, 'Domain': 2, 'Homo': 4, 'sapiens,': 5, 'Dictyostelium': 1, 'discoideum,': 1, 'Trypanosoma': 3, 'brucei,': 2, 'Arabidopsis': 2, 'Schizosaccharomyces': 6, '(Sla1p),': 1, '(Lhp1)': 2, 'LARP7': 14, '(p65)': 1, '(LARP1,': 1, 'LARP4': 1, 'LARP6).': 1, 'black)': 1, '(RRM1)': 4, 'grey).': 2, 'LARP-1,': 1, '-4,': 1, '-6': 1, 'additional': 12, 'specialized': 3, 'DM15:': 1, \"5'\": 1, 'oligopyrimidine': 1, '(TOP)': 1, 'mRNAs;': 1, 'PAM2:': 1, 'PABP-interacting': 1, 'motif-2;': 1, 'PBM:': 1, 'PABP': 1, 'motif;': 1, 'RIR:': 1, 'RACK1-': 1, 'interaction': 14, 'region;': 1, 'LSA:': 1, 'S1-associated': 1, 'motif.': 6, 'Uridylate-specific': 1, 'protectionFor': 1, 'sequence-specific': 1, 'pocket': 13, 'interdomain': 6, 'interface': 2, 'uridylates,': 2, 'without': 19, 'using': 184, 'canonical': 3, 'nucleic': 1, 'surfaces': 2, 'module:': 1, 'winged': 3, 'helix-face': 1, 'antiparallel': 4, 'β-sheet': 10, 'surface': 4, 'RRM1': 39, '10a)': 2, 'Kotik-Kogan': 4, 'Functional': 1, 'demonstrated': 23, 'indispensable': 2, '(Goodier': 2, 'Ohndorf': 3, '2001),': 2, 'findings': 3, 'reveal': 2, 'hydrophobic': 7, 'Important': 2, 'lining': 1, 'side': 6, 'Q20,': 1, 'Y23,': 1, 'Y24,': 1, 'D33,': 1, 'F35,': 1, 'F55': 2, 'residues,': 9, 'N56': 2, 'R57,': 2, 'contact': 5, '10b': 3, 'U1': 12, 'U2': 9, 'interactions).': 3, 'made': 6, 'LaM,': 3, 'makes': 7, 'I140': 2, 'contacts': 13, 'observed': 25, 'LaM-RRM1': 2, 'Through': 2, 'RNA,': 3, 'protected': 1, 'cooperative': 3, 'RRM1.a': 1, 'Structure': 6, '5’-AUAUUUU-3’': 1, '(PDB:': 22, '2VOD).': 4, 'clarity,': 3, 'negative': 15, 'adopted': 2, 'U-1.': 2, 'penultimate': 8, '(U-2)': 3, 'pointing': 2, 'towards': 9, 'inside': 2, '(U-1)': 1, 'outside': 3, 'stacking': 3, 'U-3.': 2, 'Secondary': 7, 'ribbons': 1, 'β-sheets': 4, 'colored': 11, 'pink,': 2, 'winged-helix-fold': 1, 'α-helices': 13, 'blue': 23, 'protein-specific': 1, 'show': 5, 'stick': 1, 'carbon': 2, 'atoms': 10, 'yellow.': 7, 'Other': 1, 'type:': 5, 'nitrogen': 8, 'phosphorous': 6, 'orange.': 7, 'Magnified': 5, 'views': 4, 'La-U-1': 1, 'interactions:': 2, 'U-1': 5, 'exclusively': 2, 'strictly': 3, 'sugar-phosphate': 1, 'features': 3, '(except': 2, 'ribose': 3, 'F35': 2, 'F55,': 1, 'respectively).': 3, 'D33': 2, 'specifically': 11, '2’-OH': 2, '3’-OH': 3, 'groups.': 1, 'Hydrogen': 4, 'bonds': 6, 'phosphate': 4, 'sidechain': 2, 'Y24': 1, 'R57': 1, 'use': 4, 'structures.': 1, 'view': 5, 'La-U-2': 2, 'U-2': 7, 'hydrogen-bonds': 1, 'U-2.': 1, 'Q20': 2, 'hydrogen': 3, 'bond': 1, 'backbone.': 1, 'stacked': 1, 'directly': 7, 'Y23.': 1, 'RRM1:': 2, 'amino-acid': 1, 'chains.': 1, 'An': 6, 'exists': 4, 'N139': 1, 'Y23': 1, 'LaM.': 2, 'residue': 4, 'D125': 1, 'turn': 3, 'intradomain': 1, 'interaction.': 1, 'maturationThe': 1, 'molecules,': 1, 'depends': 2, 'precise': 1, 'conformations.': 1, 'facilitate': 1, 'resolving': 1, 'misfolds': 1, '(Lorsch': 1, 'protecting': 1, 'ends': 5, '(Bayfield': 9, '2009;': 6, 'Hussain': 2, '2013;': 4, 'Naeeni,': 5, 'Contes,': 3, 'uridylate-independent': 2, 'possess': 4, 'assisting': 2, 'hLa': 39, 'though': 2, 'basic': 2, '(SBM)': 1, '(localized': 1, 'part)': 1, '(Fan': 2, '1998).': 7, 'combined': 2, 'helps': 1, 'identify': 17, 'rather': 9, 'proposed': 3, 'mode': 4, 'initiate': 1, 'recruit': 2, 'characterized': 7, 'vitro': 14, 'assay': 17, 'misfolded,': 1, 'self-splicing': 1, '(Belisova': 1, 'Many': 1, 'vivo': 5, 'non-essential': 2, 'systems.': 2, 'budding': 4, 'fission': 5, '(Sla1)': 1, 'lethal': 5, 'phenotype': 7, 'combination': 8, 'loss': 8, '2006)': 3, 'mutation': 8, 'SerCGA,': 2, 'UCG': 1, 'compensatory': 1, 'restored': 1, 'introduction': 1, 'mutations': 5, 'would': 19, 'cause': 1, 'misfolding': 2, 'observations': 4, 'mediated': 17, 'suppression': 31, 'tested.': 2, 'red/white': 1, 'assay,': 2, 'transformed': 17, 'ySH9': 10, 'strains': 30, 'homolog': 2, '(sla1-∆)': 1, 'suppressor': 24, 'tRNASerUCA': 5, 'ade6-704': 4, '11).': 2, 'When': 5, 'functional,': 1, 'La,': 3, 'ADE6': 4, 'leads': 7, 'alleviation': 1, 'accumulation': 7, 'red': 11, 'intermediate.': 3, 'words,': 2, 'colonies': 4, 'white': 17, 'thereof.': 1, 'established': 13, 'analysed': 3, '(Porat': 3, 'tRNA-mediated': 10, 'study': 26, 'Representation': 1, 'lower': 10, 'case': 1, 'letters': 1, 'going': 8, '(5’-': 1, 'removal).': 1, 'Mutated': 1, 'wild': 24, 'type': 29, 'tRNASerUGA': 1, 'indicated': 11, 'red.': 3, 'enables': 1, 'UGA': 2, 'stop': 3, 'inserting': 1, 'serine': 1, 'growing': 1, 'peptide': 3, 'chain,': 1, 'it': 9, 'mismatch': 2, 'introduced': 1, 'increase': 9, 'dependency': 1, 'suppressed': 1, 'adenine': 7, 'biosynthesis.': 1, '(in': 3, 'ySH9)': 1, 'produced.': 1, 'intermediate': 12, 'metabolite': 1, '(formed': 1, 'pathway)': 1, 'colorless': 1, 'products.': 1, 'absent,': 1, 'metabolite.': 1, '2020)': 5, '(a)': 8, 'figure': 3, 'BioRender': 1, '(b).La': 1, '(LARPs)': 2, 'arrangementsThe': 1, 'member': 2, '(LARPs),': 1, 'superfamily': 3, 'families:': 1, 'LARP1,': 1, 'LARP3': 1, '(genuine': 1, 'La),': 1, 'LARP4,': 1, 'LARP6': 1, '(Bousquet-Antonelli': 6, 'Deragon': 9, 'common': 6, 'LARP': 8, 'family': 3, 'domains:': 2, '(Alfano': 7, '2004;': 16, 'Kenan': 2, 'Keene': 2, 'Dong': 3, '2004).': 7, 'LARPs': 3, 'homology': 1, 'LaM-RRM': 5, 'arrangement,': 1, 'variations': 4, 'acquiring': 1, '(Dock-Bregeon': 1, 'Within': 1, 'group,': 1, 'closely': 1, 'related': 7, 'comparing': 19, 'tail,': 2, 'almost': 3, '(Krueger': 1, '2008;': 8, 'He': 1, 'initially': 1, '(Chambers': 1, '1983).': 4, 'RNP': 6, 'assembly,': 9, 'replaced': 1, 'recognizes': 1, 'UUU-3’OH': 24, '12b,': 2, 'left)': 4, '(He': 1, 'Muniz': 1, 'Uchikawa': 1, '2015),': 1, '12a,': 1, '(Kotik-Kogan': 5, 'Teplova': 6, 'Immunoprecipitation': 2, 'co-purifies': 1, 'methylphosphate': 1, 'capping': 1, '(MePCE)': 1, 'responsible': 4, '(Muniz': 1, 'modified': 2, 'replacement': 4, '(Bhattacharya': 1, 'interactions,': 3, 'hairpin': 2, 'α3': 3, 'helix': 4, 'RRM2': 7, 'right)': 1, '(Eichhorn': 1, '2016).': 3, 'thermophila,': 18, 'ortholog': 5, 'p65': 4, 'until': 4, 'recently': 4, 'way': 1, 'RNA:': 1, 'α-helical': 1, 'IV': 1, '12c).': 1, 'Telomerase': 1, 'similarly': 5, '2012;': 3, 'Cash': 1, 'Feigon': 1, 'Prathapam': 1, 'O’Connor': 1, 'Collins': 1, 'p43': 1, 'ciliated': 1, 'protozoa': 1, 'Euplotes': 1, 'critical': 1, 'functionality': 5, '(Aigner': 1, 'Cech': 1, 'Aigner': 1, '2003;': 5, 'Möllenbeck': 1, 'yeast,': 3, 'Pof8,': 1, 'constitutive': 1, '(Páez-Moscoso': 1, 'arrangement': 5, 'modes': 3, 'p65,': 1, 'La.': 1, 'Structural': 3, 'Structures': 3, 'sapiens.': 2, 'bind': 12, '2VOD)': 3, '1OWX).': 1, 'Similar': 2, 'forming': 4, '4WKR)': 1, '6D12).': 1, 'helical': 1, '4EYT,': 1, '4ERD).': 1, 'alveolates': 9, 'DUF3223A': 1, 'recent': 3, '(Deragon': 11, 'referred': 3, '(Arthur': 2, 'unlike': 4, 'arrangement.': 3, 'atypical': 3, 'members': 7, 'adjacent': 13, '(type': 1, 'IV)': 1, '13)': 1, 'proteins,': 13, 'LARP1': 2, 'orthologs': 1, 'fungi': 3, '(Slr1p)': 1, '(Sro9p': 1, 'Slf1p)': 1, '(Sobel': 1, '1999;': 6, 'Bousquet-Antonelli': 1, 'recent,': 1, '(Kozlov': 1, '2022).': 13, 'appearance': 2, 'immediately': 12, 'uncommon': 1, 'unexpected': 2, '10).': 2, '(Dong': 2, 'Goodier': 1, 'new': 7, 'function-3223': 2, '(DUF3223)': 2, 'region.': 5, 'DUF3223': 12, 'plant-specific': 1, 'IV/V': 1, '(NRPD1': 1, 'NRPE1)': 1, '(Wang': 2, 'Ma': 2, '2015)': 3, 'plant': 1, '(DOMINO1,': 1, 'DeCL': 1, 'DeCL-like)': 1, '(Lahmy': 1, 'Bellaoui': 1, 'bacteria': 1, 'protists': 1, 'animals': 3, 'eukaryotes.': 10, 'LaM:': 2, 'motif,': 7, 'motif-1,': 1, 'RRM2:': 1, 'motif-2,': 1, 'DUF3223:': 1, 'function-3223,': 1, 'GK/R:': 1, 'glycine,': 1, 'lysine/arginine-rich': 1, 'Overview': 1, 'lineages.': 6, 'Numbering': 1, 'Phylogenetic': 1, 'tree': 1, 'constructed': 1, 'Alveolates,': 1, 'alone': 1, 'used).': 1, 'color-coded': 1, 'right.': 1, 'Statement': 1, 'purposeThe': 1, 'discovery': 5, 'Ph.D.': 1, 'identification': 12, 'LaM-containing': 3, 'RRM1-lacking': 4, 'challenges': 1, 'expectation': 1, 'cooperatively': 2, 'RRM1.': 3, 'posed': 1, 'question': 5, 'missing': 3, 'confirmed': 24, 'indeed': 2, 'demonstrating': 2, 'promoting': 2, 'assay.': 6, 'discovered': 7, 'architecture': 9, 'absent': 4, 'continues': 1, 'altered': 9, 'eukaryote': 2, 'driven': 1, 'interesting': 6, 'contribution': 1, 'field.': 1, 'DUF3223.': 1, 'considered': 1, 'middle': 6, 'DUF3223,': 1, 'functions.': 2, 'investigated': 11, 'variant': 6, 'identifying': 2, 'partners': 2, 'active': 15, 'snRNPs': 17, 'splicing.': 3, 'pre-mRNAs.': 4, 'describing': 1, 'involvement': 2, 'able': 4, 'assign': 1, 'effect': 27, 'observing': 1, 'difference.': 1, 'elucidate': 4, 'field': 3, 'further.': 1, 'Author': 4, 'contributionsAltered': 1, 'Kerkhofs,': 3, 'Kyra1;': 2, 'Garg,': 1, 'Jyoti1;': 1, 'Fafard-Couture,': 2, 'Étienne2;': 2, 'Abou': 1, 'Elela,': 1, 'Sherif3;': 1, 'Scott,': 2, 'Michelle2;': 1, 'Pearlman,': 3, 'Ronald': 5, 'E.1;': 2, 'Bayfield,': 4, 'Mark': 3, 'Allan11': 1, 'Department': 2, 'Biology,': 3, 'Faculty': 3, 'Science,': 3, 'University,': 3, 'Ontario': 3, 'N3J': 3, '1P3,': 3, 'Canada,': 5, '2': 23, 'Département': 3, 'de': 17, 'Biochimie': 2, 'Génomique': 2, 'Fonctionnelle': 2, '(LGDP),': 2, 'Faculté': 3, 'Médecine': 3, 'des': 3, 'Sciences': 3, 'la': 3, 'Santé,': 3, 'Université': 3, 'Sherbrooke,': 6, 'Québec,': 3, 'J1E': 3, 'K8,': 3, 'Microbiologie': 1, \"d'Infectiologie,\": 1, 'Canada.': 1, 'Manuscript': 3, 'accepted': 1, 'inNature': 1, 'Communications': 1, '2022': 1, 'October': 1, '10th,': 1, 'contributions:Jyoti': 1, 'Garg': 1, 'transformed,': 1, 'selected,': 1, 'knockout': 30, '(KO)': 13, 'strain': 27, 'Southern': 5, 'blotting': 10, '7b': 1, '(original': 4, 'manuscript:': 4, 'S7b)': 1, 'previous': 15, 'versions': 4, 'immunofluorescent': 6, 'staining': 12, '7c': 1, 'S7c).': 1, 'Étienne': 2, 'Fafard-Couture': 3, 'TGIRT-sequencing': 5, 'analysis': 56, 'generating': 5, 'million': 9, '(TPM)': 5, 'counts': 23, 'custom': 7, 'python': 5, 'script': 3, '(CPM)': 6, 'data': 56, 'generate': 4, '15b': 4, '2b)': 1, '18c': 1, '5c).AbstractNascent': 1, 'sequence,': 10, 'tandem': 3, '(RRM1),': 1, 'promotion': 2, 'folding.': 1, 'classified': 4, 'predicted': 43, 'affecting': 1, 'yeast.': 2, 'examined': 9, 'stabilization.': 2, '3’-trailers': 8, 'uniquely': 1, 'relative': 32, '5’-leaders': 5, 'evolved': 4, 'disfavour': 1, 'leader/trailer': 1, 'Our': 13, 'altered,': 1, 'IntroductionRNA': 1, 'terminates': 1, '(UUU-3’OH).': 1, '2005)': 2, 'Copela': 1, '2009)': 5, 'manner': 4, 'protect': 1, '3’-': 2, '5’exonuclease': 1, 'digestion': 2, 'rescue': 2, 'Once': 4, 'acquired': 1, 'tRNA-like': 1, 'structure,': 1, 'cleavage,': 1, 'recycled': 2, 'alternative': 5, 'reversed:': 1, 'digests': 2, '2008),': 2, 'prone': 3, 'hypothesized': 10, 'establish': 1, 'determine': 18, 'versus': 3, 'processed,': 1, 'adds': 1, 'serve': 4, '2014).Genuine': 1, 'members,': 1, '14a)': 1, '2020),': 1, 'Utilizing': 1, 'organism,': 1, 'alveolate': 4, 'protozoan': 1, '(Orias': 1, '2011)),': 1, 'macronucleus/somatic': 1, 'characteristics': 1, 'TtMlp1': 1, 'insight': 1, 'observation': 3, 'apparent': 6, 'RRM1,': 9, 'nucleus.Figure': 1, 'Alveolate': 1, '(RRM1).a': 1, 'protein-7': 1, '(LARP7)': 1, 'together': 7, 'domains.': 1, 'RRM:': 1, 'DUF:': 1, 'Unknown': 1, 'Function.': 1, 'Primary': 11, 'alignments': 14, 'lineages': 4, 'showing': 4, '(highlighted,': 1, 'bottom).': 1, 'grey': 14, 'background': 7, 'indicates': 20, 'identical': 9, 'conservation.': 4, 'Color': 2, 'coded': 3, 'a.': 1, 'Full': 3, '1a,b.': 1, 'predictions': 8, 'Predicted': 1, '(dark': 3, 'blue:': 6, 'winged-helix': 3, 'RRM,': 3, 'inserted': 3, 'specifically)': 2, 'crystal': 1, 'Location': 1, 'bottom.': 2, 'a.The': 1, 'unexpected,': 1, 'due': 2, '(hLa)': 1, 'sandwich': 1, 'complete': 13, 'Thus,': 2, 'questions': 1, 'whether': 10, 'so,': 1, 'dissimilar': 1, 'possible': 1, 'how': 6, 'directs': 1, 'processing.Here,': 1, 'evidence': 4, 'protein.': 6, 'Using': 10, '(RIP)-Seq': 1, 'Mlp1,': 15, 'vivo,': 2, 'over': 4, 'vitro.': 2, 'assess': 2, 'might': 16, 'consequences': 1, 'tested': 15, 'well-established': 1, 'heterologous': 6, 'expression': 32, 'promotes': 7, 'suppression,': 4, 'protection.': 1, 'Consistent': 2, 'this,': 7, 'reveals': 4, 'destabilizes': 1, '3’-ends,': 1, 'thus': 8, 'acting': 1, 'accelerates': 1, 'considerably': 2, 'shorter,': 1, 'accordingly': 1, 'disfavor': 1, 'shortened': 2, '3’-trailers.': 1, 'consistent': 20, 'fulfills': 2, 'expected': 13, 'differs': 4, 'those': 2, 'ResultsMlp1': 2, 'phylogenetic': 2, 'kingdoms': 1, 'alveolates,': 3, 'To': 43, 'conducted': 1, '14b,': 3, '1,c-d).': 1, 'little': 1, '(Supplementary': 18, '1b)': 1, '(Kelley': 2, 'LARPs,': 2, 'inferred': 1, '14c).': 1, 'consistently': 4, 'examined,': 1, 'except': 3, 'candidate': 5, 'preferentially': 7, 'should': 6, 'immunoprecipitated': 13, 'Mlp1-associated': 23, 'detection': 10, 'Northern': 49, 'blot': 58, '15a,': 1, '2a).': 1, 'probes': 6, '3’-extensions,': 2, 'enriched': 15, 'input': 27, 'fractions.': 1, 'lesser': 4, 'enrichment': 38, '(snRNA),': 1, 'intermediates': 20, '(Xue': 9, '2000).': 13, 'conclude': 2, 'cohort,': 1, 'behaves': 1, 'partially': 3, 'uridylates.a': 1, 'ribonucleoprotein-immunoprecipitated': 4, '(RIP)': 3, 'samples': 35, 'enriches': 14, 'efficiently': 2, '(n=3': 24, 'biologically': 11, 'samples).': 8, 'IleUAU': 2, 'LeuUAA': 4, 'recognized': 4, 'probe': 18, 'used.': 2, 'non-binding': 3, 'loading': 27, 'control.': 7, 'FT:': 3, 'flowthrough,': 2, 'IgG:': 7, 'immunoglobin': 13, 'G': 28, 'antibody.': 5, 'Western': 26, 'confirming': 12, 'Mlp1-specific': 9, '2a.': 1, 'Next': 5, '(rows': 1, 'represent': 6, 'replicates),': 1, 'heatmap': 2, 'log2+1': 2, 'calculated': 24, 'taking': 15, 'ratios': 6, 'normalized': 15, 'Mlp1-immunoprecipitated': 8, 'isotypes': 8, '(top).': 1, 'Gogakos': 4, 'analyzed': 3, '(bottom).': 1, 'La-immunoprecipitated': 1, 'averaged': 6, 'CPM.': 1, 'split': 6, '(-CCA': 1, 'U-ending': 1, 'pre-tRNAs).': 1, 'Reproducibility': 2, 'replicates': 8, 'correlation': 9, 'plots': 6, '3.': 2, '32P-labeled': 21, '3’-trailer,': 2, 'only,': 1, 'd-i,k,': 1, '2d,e': 1, 'Fig.': 2, '3a,b.': 1, 'd-i': 1, 'Binding': 9, 'curves': 8, 'EMSAs': 10, 'LeuAAG': 5, '(n≥2': 2, 'experiments).': 6, 'Native': 16, 'gels': 21, '4a': 2, 'Kd': 6, 'quantification': 3, 'j': 1, 'hLa,': 10, 'mutants': 14, 'CUGCUGUUUU-3’OH': 2, 'k': 1, 'competition': 16, 'EMSA': 6, '(+5’PPP)': 3, 'unlabelled': 13, 'competitors': 1, '+5’PPP': 6, 'positive': 6, '(-5’PPP)': 2, '(n=2': 9, '4d,e.': 1, 'Source': 20, 'Data': 30, 'file.': 7, 'sequenced': 4, 'removing': 2, '>': 7, 'nt': 1, 'gel': 26, 'electrophoresis.': 2, 'reads': 16, 'mapping': 2, 'carrying': 5, 'abundance': 8, 'size-matched': 1, 'Mlp1).': 1, 'Mlp1-bound': 8, 'Mlp1;': 1, 'values': 5, 'presented': 3, '2b).': 1, 'our': 22, 'dataset': 21, 'hLa-bound': 4, 'photoactivatable': 1, 'crosslinking': 1, '(PAR-CLIP)': 1, 'done': 8, 'hydro-tRNAseq': 1, '(Gogakos': 4, 'hLa;': 1, '2c).': 1, 'strongly': 13, 'indicative': 1, 'thermophila.Mlp1': 1, 'vitroThe': 1, 'engagement': 1, 'sites,': 1, 'body,': 1, 'Fan': 1, 'correlated': 3, 'ligands,': 1, 'electrophoretic': 2, 'mobility': 2, 'shift': 7, '(EMSAs),': 1, 'well': 11, 'included': 7, '15c).': 1, 'Lowest': 1, 'affinities': 1, '2d,e,': 1, '1).To': 1, 'radioactively': 9, 'labeled': 10, 'targets': 9, '15d-i,': 1, '4a,': 2, 'test': 12, 'importance': 7, 'mutant': 12, 'recognize': 1, 'substituted': 1, '(Q11A/Y14A)(Dong': 1, '1d,': 2, 'Q20/Y23': 1, 'numbering': 4, 'hLa),': 1, 'entire': 2, '(Mlp1': 15, '95-340).': 1, 'increased': 2, 'lost': 3, 'Q11A/Y14A': 4, '(compare': 2, '15e': 2, 'f)': 1, 'g).': 1, 'discriminate': 4, '1-250)': 2, 'maintains': 1, '15h),': 1, 'context': 3, '1-250': 3, 'Q11A/Y14A)': 3, 'again': 2, 'resulted': 5, 'decreased': 8, '15i).': 1, 'directly,': 2, 'protein-RNA': 7, 'experiments': 5, 'CUGUGUUUU-3’OH': 1, '(Q11A/Y14A)': 1, '15j,': 1, '15j),': 1, 'individually': 4, '1-95': 2, '95-250)': 3, '4b).': 1, 'suggest': 4, 'substrates,': 2, 'predicted,': 1, 'determining': 2, 'targets.': 2, 'Kd:': 1, 'equilibrium': 1, 'dissociation': 2, 'constant,': 1, 'S.D.:': 1, 'deviation,': 3, 'N.D.:': 1, 'determined,': 1, 'N.A.:': 1, 'calculation': 7, 'applicable.The': 1, 'EMSA,': 1, 'containing,': 5, 'competitor': 13, 'phosphatase': 1, 'treatment': 2, '(-5’PPP;': 1, '+/-': 3, '5’PPP': 2, '4c).': 1, \"–5'PPP\": 1, 'competed': 2, 'poorly': 1, 'radioactive': 2, '15k': 2, 'compare': 18, \"-5'PPP\": 2, '4d),': 1, 'smaller': 2, '4e).': 1, 'discrimination': 7, 'hLa.': 3, 'hLaTo': 1, 'modes,': 1, '(U10)': 2, 'substrates.': 2, 'that,': 5, 'work': 32, '2009),': 1, 'stronger': 2, '16a,': 1, '5a).': 1, 'off': 1, 'low': 10, 'amounts': 2, '16b,': 1, '5a),': 1, 'suggesting': 9, 'weaker': 2, 'stringently': 2, 'hLa.a,b': 1, 'U10': 5, '(control),': 1, '(b).': 1, 'Error': 2, 'bars': 2, 'mean': 2, 'deviation': 1, 'U10:': 2, 'n=4,': 1, 'tRNA:': 2, 'n=2,': 2, 'n=1': 2, '5a.': 1, 'Three-dimensional': 1, 'last': 5, '(UUU-3’OH': 1, ')': 3, 'positioned': 2, 'yellow;': 1, 'proteins).': 1, 'Image': 10, 'PyMOL.': 3, 'd,e': 4, 'P32-labeled': 2, 'CUGCUGUUUU': 3, '(4U)': 1, '4U': 6, 'RNAs:': 1, 'CUGCUGUUUC': 2, '(U-1C),': 2, 'CUGCUGUUCU': 2, '(U-2C),': 2, 'CUGCUGUCUU': 2, '(U-3C)': 3, 'CUGCUGCCCC': 2, '(4C)': 2, '(d)': 4, '(e)': 3, '5c,d.': 1, 'f': 2, '3’-end.': 3, 'backbones': 1, 'c,': 3, 'PyMOl.': 1, 'g,h': 1, 'U10-3’OH': 2, '(control)': 1, 'U10-3’-P': 1, '(g)': 1, '(h)': 1, '5e.': 1, 'file.Previous': 1, 'high-resolution': 8, 'characterization': 1, '(UUU-3’OH)': 1, 'greatest': 3, 'specific,': 1, 'Notably,': 2, '16c).': 1, 'Previous': 15, '10-mers': 1, 'C10,': 1, 'G10': 1, 'A10,': 1, '20-mer': 1, 'G20': 1, 'A20': 1, 'considerable': 1, '(Vinayak': 3, '(two': 3, '3’-end:': 1, 'U-2),': 1, '(hence': 1, '4U)': 1, 'U': 7, 'substitution': 1, 'had': 3, '2006),': 3, 'C10': 1, '5b).': 1, 'expected,': 5, 'mutating': 1, 'cytidylate': 1, '(U-2C': 1, 'CUGCUGUUCU-3’OH)': 1, 'poor': 2, '(U-1C)': 1, 'third': 2, 'capability': 1, '16d,': 2, '5c).': 2, 'Mutating': 1, 'positions': 3, '(4C': 1, 'CUGCUGCCCC-3’OH)': 1, 'inability': 1, 'compete': 1, '(U-1C,': 1, 'U-2C': 2, 'U-3C)': 1, 'showed': 1, 'levels,': 4, '16e,': 1, '5d).': 1, '4C': 1, 'capable': 3, 'Mlp1.': 10, 'occurring': 2, 'specificity': 2, 'Mlp1.The': 1, '2’-': 3, '3’-hydroxyls': 1, '(-OH)': 1, 'aspartate': 1, '16f)': 1, 'hLa).': 1, 'hydroxyl': 2, 'regular': 1, '3’-phosphorylated': 2, 'unlabeled': 3, 'phosphorylated': 3, 'competitor,': 1, '3’-hydroxyl': 1, '16g,h,': 1, '5e).': 1, 'Together,': 3, 'demonstrate': 6, 'shows': 2, 'preference': 1, 'dependence': 2, 'hLa.RNA': 1, 'computational': 5, 'biochemical': 2, 'suggested': 1, 'well-established,': 1, 'report': 1, 'Porat': 1, 'differences': 5, 'Sla1p,': 5, 'sla1-': 5, '(ySH9)': 2, 'UGA-decoding': 2, '(tRNA-SerUCA)': 2, 'allele,': 1, 'tRNA-SerUCA': 1, 'suppress': 1, 'pigment': 1, 'adenine.': 1, 'Successful': 1, 'suitable': 1, 'chaperone,': 1, 'unsuppressed': 1, 'cells.': 3, 'Sla1p-transformants': 1, 'Mlp1-transformants,': 1, 'stabilize': 4, '17a).': 2, 'Maturation': 2, 'general': 1, 'activity,': 2, '95-340)': 2, 'pink': 1, 'equal': 10, '17a,': 1, '6a),': 1, 'vivo.': 2, 'near': 5, '17a),': 2, 'inactivation': 1, '(Q11': 1, 'Y14)': 1, 'page': 10, '3’-trailers.a': 1, 'pRep4': 8, '(positive': 3, 'control;': 2, 'protein),': 1, '6a.': 1, 'Levels': 1, 'colour': 1, 'annotated': 15, 'follows:': 1, '(+++),': 1, '(++),': 1, '(+),': 1, '(-).': 1, 'Top:': 1, 'diagram': 1, 'pre-tRNA-SerUCA': 1, 'transformants': 4, 'Accumulation': 2, 'SerUCA': 2, 'asterisk': 1, '3’-processed': 2, 'Excess': 1, 'endogenous': 11, 'SerUGA': 1, 'cross-reaction': 1, 'U5:': 3, 'LysCUU': 5, 'a,': 2, '4a)': 1, 'detects': 1, 'unprocessed': 5, '(band': 3, 'a),': 1, 'b)': 1, 'c)': 1, 'sla1': 1, 'lane': 5, '2).': 5, '3’-processed,': 1, 'band': 4, 'd.': 2, 'probed': 4, 'probe,': 6, 'probe.': 1, 'Black': 3, 'boxes': 3, 'marker': 5, 'size': 8, 'nt.': 1, 'd': 11, 'Sanger': 3, 'clonal': 2, 'isolates': 2, 'cDNAs': 1, 'derived': 3, 'Sla1p-': 4, 'a-c': 3, 'Mlp1-immunoprecipitations': 1, '6b.': 1, 'extracted': 15, 'Mlp1-': 6, 'Sla1p-expressing': 1, '17b).': 1, 'Mlp1-transformed': 3, '(lanes': 1, '2,': 5, 'corresponded': 1, 'opposed': 5, 'subsequent': 2, '(lane': 1, '17a,b).': 1, 'slightly': 7, 'cells': 10, 'asterisk).': 1, 'detected': 6, 'intron,': 3, '17c).': 1, 'transformation': 8, '17c': 3, 'lanes': 8, 'c).': 1, 'Sla1': 1, 'expressing': 3, 'co-migrate': 1, \"5'-leader,\": 1, 'd,': 3, 'beneath': 1, 'band).': 2, 'hypothesize': 5, 'represents': 3, 'stably': 1, 'whose': 2, '3’-ends': 11, 'nibbled': 2, 'yet': 4, 'retained': 2, 'stabilized': 5, 'LysCUU,': 1, 'co-migrated': 1, '-transformants': 1, '3),': 3, 'tRNA-mediation': 1, 'intermediates.': 1, 'engaging': 1, 'pombe,': 4, 'protections': 1, 'Sla1p.To': 1, 'RNP-complexes': 1, '6b)': 1, 'TyrGUA': 2, '3’-rapid': 1, 'amplification': 9, '(3’RACE).': 1, 'Sla1p-associated': 1, 'primarily': 1, '3’-trailers,': 3, '2005),': 1, 'largely': 2, 'depleted': 3, '17d),': 1, '17b,c),': 1, 'minority': 1, '3’-CCA,': 1, 'removal.': 1, 'vitro,': 1, 'proteins.Mlp1': 1, 'processingTo': 1, 'MLP1': 7, 'integration': 2, 'selection': 5, 'PCR': 8, '7a,b': 1, 'reduced': 6, '18a)': 1, 'indirect': 4, '7c).': 1, 'Complete': 2, 'locus': 4, 'macronuclei': 1, 'increasing': 3, 'drug': 3, '(phenotypic': 2, 'assortment': 2, '(Merriam': 3, 'Bruns': 3, '1988))': 1, 'achieved,': 1, 'likely': 13, 'system,': 2, 'Mus': 1, 'musculus,': 2, 'melanogaster,': 2, 'brucei': 3, 'thaliana': 3, '(Arhin,': 1, '2000;': 3, 'IleUAU,': 2, 'ValAAC.': 1, '18b': 1, 'bottom': 3, 'being': 6, 'correlate': 1, 'inversely': 1, 'system.': 2, 'Depletion': 1, 'tRNAs.a': 1, '(WT)': 12, 'Loading': 10, 'control:': 9, 'histone': 5, 'H3': 5, 'β-actin.': 3, 'Purified': 1, '6X-His-Mlp1': 1, 'antibody': 23, 'specificity.': 2, 'detecting': 5, 'ValCAC': 1, 'size.': 1, 'and/or': 2, 'controls.': 1, 'log10': 3, 'cumulative': 2, 'replicates).': 17, 'plotted': 5, 'longest': 1, \"(UUUUUU-3'OH).\": 1, 'Quantification': 5, 'samples)': 1, 'c.': 2, 'deviation.': 1, 'Two-way': 1, 'ANOVA': 4, 'Bonferroni’s': 1, 'correction': 1, 'comparisons': 1, 'statistical': 6, 'significance:': 1, '****': 1, '<': 9, '0.0001,': 1, '*': 1, '=': 18, '0.0491.': 1, 'isodecoder,': 1, '(U1,': 4, 'etc.)': 1, 'scored': 1, 'CPM': 4, 'greater': 4, '0.3;': 1, 'log10(2).': 1, 'e,f': 1, 'Scatterplots': 1, 'log2': 4, 'WT': 45, 'KO': 41, '(f)': 1, 'Additional': 8, 'calculated,': 1, '(r': 2, '0.746': 1, 'r': 6, '0.731,': 1, 'respectively)': 2, 'g': 12, 'Plotting': 1, 'gives': 1, 'strong': 8, '0.970).': 1, 'ArgUCU': 1, 'membranes.': 1, 'file.To': 1, 'transcriptome-wide': 1, 'scale,': 1, 'RNA-sequencing': 16, 'RNA-enriched': 1, 'samples,': 1, 'RIP-Seq': 1, '15b),': 1, 'unique': 2, \"3'-uridylate\": 1, '18c,d).': 1, 'limiting,': 2, 'efficiently.': 2, 'accelerating': 1, '3’-trailers.Previous': 1, 'predictor': 1, '(Behrens': 1, '2021;': 2, 'Tuller': 1, '(R2': 5, '0.55,': 1, '0.75)': 1, '18e).': 1, '0.53,': 1, '0.73)': 1, '18f)': 1, '0.94,': 1, '0.97)': 1, '18g),': 1, 'influence': 2, 'representative': 2, '18h),': 1, 'reminiscent': 1, '(despite': 1, 'processing)': 1, 'conditional': 1, 'mouse': 4, '(Gaidamakov': 1, '2014).The': 1, 'eukaryotesTo': 1, 'explore': 1, 'extensively,': 1, 'thymines': 2, '(Bogenhagen': 1, 'Brown': 1, 'give': 1, 'rise': 1, '3’UUU-OH': 2, 'analyzed,': 1, 'zero': 1, '19a,': 1, 'Identical': 1, 'trends': 1, 'stretches/candidate': 1, 'terminators': 1, 'five,': 1, 'six': 2, 'seven': 5, '19b,': 1, 'noted': 4, '2b': 1, 'pre-tRNAs)': 1, 'RIP-seq': 1, 'determinant': 1, 'affinity.': 1, 'moderately': 1, 'negatively': 1, '0.14,': 1, '-0.47)': 1, '8a),': 1, 'may': 12, 'data,': 2, '0.05,': 1, '-0.23)': 1, '8b).': 1, 'N-1': 6, '5’-leader.a': 1, 'Genome-wide': 2, 'determination': 1, '(the': 2, 'posttranscriptional': 1, 'CCA)': 1, 'Ts': 2, 'Violin': 2, 'median': 2, 'purple': 9, 'line': 6, 'quartiles': 2, 'black': 10, 'lines.': 4, 'summary': 2, '2.': 5, 'No': 2, 'significance': 4, '(P': 2, '0.05)': 2, 'one-way': 3, 'Tukey’s': 2, 'comparison': 7, 'test.': 2, 'written': 2, 'uppercase': 1, '(UCGA)': 1, 'cartoon.': 2, 'lowercase': 1, '(ucga)': 1, '(N73)': 7, 'highlighter': 2, 'green.': 3, 'green': 6, '(N-1).': 3, 'Logo': 2, '(d).': 2, '(e).': 2, 'representing': 2, 'condition': 1, 'underneath': 3, 'logo.': 2, 'file.Table': 1, 'Summary': 2, 'species.S.D.:': 1, 'nt:': 2, 'nucleotides.Processing': 1, '–1/+1': 1, '(Leontis': 1, '1988;': 1, 'Wu': 2, 'catalytically': 6, '(True': 1, 'Celander': 1, 'Optimal': 1, 'bulge': 4, 'N–1': 1, 'position;': 1, '19c),': 1, 'inhibits': 1, '(Lee': 1, 'mismatched': 1, 'logo': 2, 'frequent': 3, '3’-residue': 1, '(N-1)': 3, '(±': 3, '75%)': 1, '19d,': 2, '8c).': 2, 'distribution': 10, 'At': 1, '50%),': 1, '25%)': 1, '8d)': 1, 'study.': 1, 'then': 10, '19e,': 1, '8e),': 1, 'pair': 3, '(A73': 1, 'A-1;': 1, 'C73': 1, 'A-1': 1, 'G73': 1, 'A-1).': 1, 'Conversely,': 1, 'thymine': 1, '(T73)': 1, 'avoided': 1, 'partitioned': 1, 'evident': 1, '19e).': 1, 'leader-trailer': 1, 'bulges': 1, 'base,': 2, 'having': 1, 'pressure': 1, 'seen': 5, '8f)': 1, 'species.DiscussionPreviously': 1, 'collectively': 1, 'module.': 1, 'continued': 3, 'module’s': 1, 'domain.': 1, 'uncharacterized': 3, '(DUF3223),': 1, 'superfamily.': 1, 'unprecedented,': 1, 'clear': 1, 'perform': 3, 'what': 2, 'have,': 1, 'any,': 1, 'targets.We': 1, 'assays,': 1, 'Q11': 1, 'Y14,': 1, 'analogous': 3, 'trailer': 3, '1-95)': 2, 'included,': 1, 'isolation.': 1, 'Given': 1, 'paucity': 1, 'UUU-3’OH,': 1, '95-250': 2, 'function,': 3, 'making': 1, 'fold.': 1, 'combine': 1, 'support': 2, 'key': 1, 'exist.': 2, 'Competition': 5, 'trailers': 1, 'easily': 1, 'displaced': 2, 'less': 7, 'hLa-UUU-3’OH': 1, 'co-crystals': 1, 'Unlike': 1, 'effective': 1, 'U-3C': 1, 'U-1C': 1, 'residue,': 1, 'generally,': 1, 'differentiate': 1, 'RNAs.Previous': 1, '(LaM+RRM1)': 2, 'regions,': 1, 'exonucleases': 1, '(Hussain': 1, 'null': 1, '(sla1-)': 1, 'raise': 1, 'possibility': 1, 'proteins.Genuine': 1, 'however': 1, 'providing': 2, '3’-exonucleases.': 3, 'RIP-3’-RACE,': 1, 'stabilizes': 2, 'intermediates,': 1, 'lacking,': 1, 'accelerate': 1, '3’-processing': 3, 'imply': 2, 'leaving': 2, 'exposed': 6, 'Alternatively,': 4, 'Z.To': 1, 'better': 1, 'understand': 2, 'RNA-Seq': 1, 'blots': 6, 'appears': 5, 'nibbling': 1, 'ends,': 1, '20).': 2, 'impedes': 2, 'Z,': 1, 'delaying': 2, '3’-endonucleolytic': 1, 'proceeding': 1, '(Nashimoto': 1, 'seeing': 1, 'blocked.': 1, 'destabilization': 1, 'suggests': 4, 'differing': 1, 'caused': 3, 'Rex1p': 3, '3’-endonuclease': 3, '9a,b).': 1, 'mutually': 1, 'exclusive': 1, 'hypothesis': 2, 'reduction': 1, 'LSm2-8': 22, 'wild-type': 1, 'cells,': 1, 'Kufel': 1, 'systems,': 1, 'conserved,': 4, '9c,d).': 1, 'lead': 3, 'Mlp1-independent': 2, 'yeast.Figure': 1, 'Model': 1, 'thermophila.During': 1, '(top,': 2, '3’-stretch': 1, 'provides': 2, 'cut': 1, '3-trailer': 1, 'right),': 1, 'Rex1p,': 3, 'point': 5, 'Mlp1-dependent': 1, '(bottom,': 1, 'created': 1, 'BioRender.com.': 1, 'Publication': 1, 'License': 1, 'Along': 1, 'genome-wide': 1, 'dominated': 1, 'stretch,': 1, 'prevalence': 1, 'appear': 1, 'perfect': 2, 'matches': 1, '(preceding': 1, 'tail),': 1, 'ensuring': 1, 'minimized': 1, '(N-2)': 1, 'adenosine,': 1, 'proximal': 2, 'paired': 3, 'closer': 1, 'extremities.': 1, 'A,': 1, 'T': 2, 'G.': 2, 'prevalent': 1, 'T,': 1, 'flexibility': 1, 'introduce': 1, 'optimal': 2, 'P.': 1, 'It': 5, 'roles': 1, 'here': 1, 'MethodsConservation': 1, 'analysis.': 4, 'Accession': 2, 'numbers': 4, 'National': 2, 'Center': 2, 'Biotechnology': 2, 'Information': 2, '(NCBI)': 2, 'Z/ELAC': 1, 'Clustal': 2, 'Omega': 2, '(EMBL-EBI)': 2, '(Sievers': 2, '2011),': 1, 'annotate': 1, 'reference.': 1, 'Amino': 4, 'chain': 4, 'diversity:': 2, 'Asp': 2, '(D),': 2, 'Glu': 2, '(E),': 2, 'Asn': 2, '(N),': 2, 'Gln': 3, '(Q);': 2, 'Lys': 2, '(K),': 2, 'Arg': 2, '(R),': 2, 'His': 2, '(H);': 2, 'Phe': 2, '(F),': 2, 'Trp': 2, '(W),': 2, 'Tyr': 2, '(Y);': 2, 'Val': 2, '(V),': 2, 'Ile': 2, '(I),': 2, 'Leu': 2, '(L),': 2, 'Met': 2, '(M);': 2, '(5)': 2, 'Ser': 2, '(S),': 2, 'Thr': 2, '(T).': 2, 'Phyre2': 1, 'α-helices.': 1, 'High-resolution': 10, 'discoideum': 3, 'PDB:': 6, '2VOD,': 1, '1S29,': 1, '2M5W,': 1, '2VOD.': 1, 'prediction': 9, 'lomets2': 1, 'tool': 3, '6PNN': 1, '4M7D,': 1, 'Modeling': 2, 'constructs.': 1, '(NCBI': 2, 'Reference': 1, 'Sequence:': 1, 'XP_001019287.2)': 1, 'Genome': 4, 'Database': 5, '(TTHERM_00384860)': 1, '(Stover': 1, '2012),': 4, 'optimized': 2, 'coli.': 1, 'gBlocks': 1, 'ordered': 1, 'integrated': 1, 'technologies': 1, '(IDT)': 1, '(sequence': 1, '3).': 3, 'electromobility': 1, '(EMSAs)': 2, 'cloned': 4, 'NheI': 1, 'BamHI': 3, 'restriction': 6, 'pET28a': 2, 'vector': 2, 'plasmid': 9, '6XHis-tag': 1, 'purification.': 2, 'NcoI': 1, 'primer-encoded': 1, '6X-His': 2, 'tag.': 1, 'transformations': 1, 'SalI': 1, '(ura+)': 1, 'incorporating': 1, 'tag': 1, 'N-terminally': 1, 'forward': 2, 'primer': 5, 'amplification.': 1, 'All': 3, 'primers': 7, 'listed': 2, '3.T7': 1, 'templates': 2, 'pre-': 1, '(listed': 1, 'T7': 3, 'promotor.': 1, 'purified': 15, '7': 6, 'M': 10, 'urea': 9, 'denaturing': 9, '10%': 14, 'polyacrylamide': 18, 'eluted': 6, 'overnight': 24, 'mM': 122, 'sodium': 14, 'acetate': 1, 'phenol:chloroform:isoamylalcohol': 5, '(25:24:1)': 4, '4°C.': 12, 'aqueous': 7, 'layer': 7, 'centrifugation': 26, '10': 39, 'minutes': 60, '4°C': 22, 'ethanol': 11, 'precipitated': 6, '-80°C.T.': 1, 'cultivation': 2, 'generation.Liquid': 1, 'cultures': 8, 'grown': 7, 'mid-log': 6, 'phase': 11, '(0.1': 6, 'x': 6, '106': 7, 'cells/mL)': 6, '30°C': 2, 'shaking': 2, 'RPM': 4, 'SPP': 2, 'media': 3, '(1%': 1, 'proteose': 2, 'peptone,': 2, '0.1%': 18, 'extract,': 2, '0.2%': 3, 'glucose,': 2, '0.003%': 2, 'Fe-EDTA)': 1, '(Fillingham': 3, 'Cell': 7, 'pellets': 21, 'harvested': 3, '1000g.': 2, 'washes': 8, 'Tris-HCl,': 20, 'pH': 56, '7.4,': 16, 'stored': 3, '-80°C.': 4, 'PCR-amplification': 1, 'flanking': 4, '3’-regions': 1, 'KpnI/XhoI': 1, 'NotI/SacI,': 1, '(http://www.ciliate.org/': 1, 'TTHERM_00384860)': 1, 'pNeo4': 2, '(Mochizuki': 1, 'paromomycin': 2, '(Neo4)': 1, 'resistance': 1, 'cassette': 2, 'KpnI': 1, 'XhoI': 1, 'SacI': 1, 'NotI,': 1, 'Neo4': 1, 'downstream': 20, 'CdCl2': 1, 'inducible': 1, 'metallothionein': 1, '(MTT1)': 1, 'promoter.': 1, 'construct': 3, 'linearized': 1, 'ScaI': 1, 'transformation.': 2, 'Biolistic': 1, 'T.thermophila': 1, '(Chalker': 1, 'Integration': 1, 'homologous': 3, 'recombination': 3, 'concentration': 6, 'μg/mL': 2, '1000': 2, 'assortment)': 1, '1988).': 1, 'PCR.': 1, 'Partial': 1, 'available': 7, 'request.Protein': 1, 'isolation': 2, 'thermophila.T.': 3, 'resuspended': 17, 'Trichloroacetic': 4, '(TCA)': 2, '1X': 35, 'PBS,': 3, 'incubation': 16, '–20°C': 3, 'precipitation.': 3, 'fluffy': 1, 'pellet': 18, 'collected': 15, '15': 15, '10,000g': 4, '4°C,': 5, 'washed': 28, 'twice': 11, '100%': 5, 'ice-cold': 7, 'acetone.': 4, 'air-dried': 4, '50μL/mL': 4, 'culture': 7, '2.5X': 4, 'SDS': 15, 'dye': 11, '[5X': 6, 'dye:': 6, '5%': 7, 'β-mercaptoethanol': 7, '(v/v),': 12, '0.02%': 6, 'bromophenol': 12, '(w/v),': 12, '30%': 6, 'glycerol': 19, 'dodecyl': 9, 'sulfate': 9, '(SDS)': 9, '250': 8, '6.8].': 4, 'RNA-immunoprecipitation': 5, 'mL': 15, 'log': 4, 'PBS.': 4, 'cross-linked': 3, '1%': 17, 'formaldehyde': 4, 'quenching': 3, '0.25': 3, 'glycine': 4, 'room': 17, 'temperature.': 9, 'Cells': 1, 'PBS': 7, 'lysis': 9, 'buffer': 42, '[30': 3, 'Tris-HCl': 9, 'NaCl,': 8, 'KCl,': 5, 'MgCl2,': 3, 'Triton-X100,': 4, 'phenylmethylsulfonyl': 5, 'fluoride': 5, '(PMSF),': 5, 'Halt': 6, 'Protease': 6, 'Inhibitor': 12, 'Cocktail': 6, '(PIC)': 6, '(ThemoFisher': 6, 'Scientific),': 6, '100U': 3, 'SUPERase': 6, '(ThermoFisher': 26, 'Scientific)]': 4, 'sonication': 3, '(25%': 3, '15s': 3, 'intervals': 3, 'minutes).': 3, 'lysate': 6, 'clarified': 3, '45': 9, 'treated': 7, 'TurboDNase.': 1, 'lysates': 6, 'pre-cleared': 4, 'rabbit': 26, '(IgG)': 8, 'control-bound': 3, 'Dynabeads': 17, 'rotating': 14, 'hour': 18, '(12': 8, 'μg': 15, '/': 17, '50': 17, 'μL': 16, 'beads': 9, 'slurry).': 8, 'Immunoprecipitations': 6, 'anti-Mlp1': 13, 'Scientific': 10, 'Custom': 7, 'Antibodies)': 7, 'IgG': 14, 'coupled': 6, 'times': 14, '(without': 5, 'PIC,': 5, 'PMSF': 6, 'inhibitor).': 4, 'Input': 3, 'isolated': 8, 'cross-linking': 9, '70°C': 4, '[50': 10, 'EDTA': 4, '8.0,': 5, 'DTT,': 4, 'SDS],': 5, 'Trizol': 15, 'extraction.tRNA': 1, 'library': 1, 'preparation': 1, 'TGIRT': 1, 'analysis.Total': 1, 'size-selected': 1, '(20-300': 1, 'nucleotides)': 3, 'gel.': 3, 'acetate,': 1, '5.1': 5, '20,000g': 4, 'once': 4, '70%': 3, 'ethanol,': 1, 'air': 1, 'dried': 3, 'RNase-free': 3, 'H2O.': 1, 'structured': 1, 'inhibitory': 1, 'synthesis,': 1, 'TGIRT-III': 1, '(InGex)': 1, 'template-switching': 1, 'prepare': 1, 'libraries': 1, '(Fafard-Couture': 5, 'Nottingham': 1, 'Libraries': 1, 'NextSeq500': 1, 'platform': 2, '(Illumina)': 1, '(2x': 1, '75': 2, 'bp).': 1, 'tRNA-Seq': 1, 'pipeline.Details': 1, 'about': 2, 'tools': 6, 'parameters': 3, 'bioinformatic': 1, 'analyses': 1, 'reproducible': 1, 'Snakemake': 2, 'workflow': 2, 'below.': 4, 'raw': 7, '(.fastq': 2, 'files)': 1, 'download': 1, 'Gene': 2, 'Omnibus': 2, '(GEO)': 2, 'accession': 5, 'GSE199642.': 2, 'Briefly,': 4, 'paired-end': 1, 'Trimmomatic': 1, 'v0.36': 1, '(Bolger': 1, '2014),': 2, 'FastQC': 1, 'v0.11.5': 1, 'evaluate': 1, 'read': 6, 'trimming,': 1, '2021).': 9, 'Resulting': 1, 'aligned': 2, 'mismatch)': 1, 'version': 2, 'T_Thermophila_MAC_2021': 1, '(Sheng': 1, '(accessible': 1, 'STAR': 2, 'v2.6.1a': 2, '(Dobin': 1, 'Gingeras': 1, '(with': 2, '2021)': 5, 'parameter': 1, 'value:': 1, '--outFilterMismatchNmax': 1, '0.': 1, 'index': 1, 'needed': 3, 'align': 2, 'annotation': 6, '(.gtf)': 2, 'file': 4, 'built': 1, 'converting': 2, '.gff': 2, 'files': 6, '(one': 1, 'protein-coding': 1, 'genes;': 1, '.gtf': 1, 'concatenating': 1, 'bash': 1, 'scripts.': 2, 'corrected': 1, 'embedded': 1, 'CoCo': 4, 'v0.2.5.p1': 3, '(Deschamps-Francoeur': 1, 'correct_annotation': 1, 'default': 3, 'parameters.': 1, 'Counts': 2, 'attributed': 1, 'correct_count': 2, 'mode.': 1, 'Bedgraph': 1, 'correct_bedgraph': 1, 'parameters).': 1, 'Differential': 1, 'DESeq2': 1, '(Love': 1, '2014)': 2, 'count': 2, 'output': 1, 'WT,': 1, 'IP': 1, 'samples.': 4, 'Normalized': 1, 'measured': 1, 'TPM': 6, 'differential': 1, '4.': 2, 'biological': 31, 'filtering': 3, 'R.tRNA': 1, 'fishing': 2, 'binning': 2, 'tRNAs.Raw': 1, '(3’-UUU)': 1, '(3’-CCA)': 1, 'list': 2, '(e.g.': 1, 'TTG:': 1, 'AATCCTCTGACCTGGGTTCGAATCCCAGTACGACCT)': 1, '(Additional': 8, '5)': 2, '(“fish”)': 1, 'unmapped': 1, 'format).': 1, 'Each': 2, 'bins': 1, 'reads:': 1, '(mature': 1, 'tRNA),': 1, '1U,': 1, '2U,': 1, '3U,': 1, '4U,': 1, '5U,': 1, '6U,': 1, '7U,': 1, '8U,': 1, '9U': 1, '10U': 1, '(premature': 1, 'tRNA).': 1, 'Raw': 2, 'dividing': 1, 'fished': 1, 'bin': 1, 'replicate': 2, 'Fold-enrichment': 2, 'summation': 1, 'counts,': 1, 'ratio': 8, 'CMP': 1, 'displayed': 2, 'heatmaps': 2, '6).': 4, 'GEO': 1, 'database': 2, 'GSE95683': 1, 'identically': 1, 'Electromobility': 1, '(EMSA).U10': 1, '(20': 2, 'pmol)': 1, 'chemically': 1, 'synthesized': 2, '(Integrated': 1, 'Technologies': 1, '(IDT))': 1, 'pmol': 1, '[γ-32P]-ATP': 1, '(PerkinElmer,': 3, '10mCi/ml)': 2, 'units': 6, 'T4': 6, 'Polynucleotide': 5, 'Kinase': 5, '(PNK)': 5, '(New': 5, 'England': 5, 'Biolabs,': 5, 'cat#M0201S)': 1, 'hours': 4, '37°C': 7, 'PNK': 1, 'cat#B0201S).': 1, 'Radioactively': 1, '[α-32P]-UTP': 1, 'constructs).': 1, 'Dephosphorylation': 1, 'QuickCIP': 1, 'cat#M0525S)': 1, '30': 18, 'rCutSmart': 1, 'Buffer': 6, 'cat#': 5, 'B6004S),': 1, 'extraction.': 6, 'PAGE': 1, '0.5M': 1, 'NaCl': 5, 'His-tagged': 1, 'Escherichia': 1, 'BL21': 1, 'Co2+': 1, 'beads,': 1, 'heparin': 1, 'column': 1, 'exchanged': 1, 'quantified': 3, 'Bovine': 1, 'Serum': 1, 'Albumin': 1, '(BSA)': 1, 'quantifications': 1, 'SDS-PAGE.': 1, '3000': 1, '(~': 1, '0.1': 2, 'nM)': 1, 'incubated': 18, '[10%': 2, 'glycerol,': 1, 'EDTA,': 11, 'blue]': 2, '95°C': 12, 'minutes,': 6, 'slow': 2, 'cooling': 5, 'EMSAs,': 1, '95°C.': 1, '>85%': 1, '30°C.': 1, 'snap': 5, 'cooled': 2, 'ice': 2, '8%': 2, 'native': 12, '100V.': 2, '80°C': 5, 'Gel': 5, 'Dryer': 5, '(Bio-Rad)': 1, 'storage': 5, 'phosphor': 5, 'screen': 4, 'overnight.': 2, 'screens': 2, 'Typhoon.': 4, 'ImageJ': 3, 'nonlinear': 1, 'curve': 3, 'fitting': 1, 'program': 1, '(GraphPad': 1, 'Prism).': 1, 'pombe.tRNA': 1, 'brief,': 3, 'allele': 1, 'mutants.': 1, '(ura4+),': 1, 'selective': 1, '(Edinburgh': 1, 'Minimal': 1, 'Medium': 2, '(EMM)': 1, '–ura': 4, '–leu)': 1, 'liquid': 6, 'EMM': 2, 'leu': 1, '(OD': 2, '0.6-0.9).': 1, 'Spotting': 1, 'adenine-containing': 1, 'plates': 1, '(EMM': 1, '–leu': 1, 'ade10),': 1, '4-day': 1, '32°C.': 1, 'Yeast': 2, 'purification': 1, 'extraction': 14, '1800': 1, 'ddH2O.': 3, 'resuspension': 5, 'NET-2': 3, '0.05%': 3, 'NP40,': 1, 'PIC],': 1, 'bead-beating': 1, 'sec': 9, 'ON': 1, 'OFF': 1, 'intervals)': 1, 'g.': 1, 'resuspending': 2, 'NaOAc,': 7, '5.1,': 1, 'adding': 9, '5.1]-saturated': 1, 'phenol': 1, '65°C': 1, 'vortexing.': 1, 'phenol:chloroform:isoamyl': 4, 'alcohol': 4, '(25:24:1).': 2, 'precipitation': 6, '-80°C': 4, 'hour.': 1, '3’RACE': 1, 'pombe.Sla1p-': 1, '0.6-0.9)': 1, '–leu.': 1, '0.5%': 5, 'temperature': 9, 'minutes.': 4, '4000': 1, '(Beckman': 3, 'Coulter': 1, 'JLA9.1000': 1, 'rotor),': 1, 'ddH2O': 1, '1800g,': 1, 'wash': 7, '[1.2%': 1, 'polyvinylpyrrolidone': 1, '(PVP),': 1, 'HEPES,': 4, 'PMSF,': 2, 'DTT].': 1, 'flash': 1, 'frozen': 1, 'continuous': 1, 'bead,': 1, 'cryogrinding': 1, 'mortar': 1, 'pestle.': 1, 'powder': 3, 'lysed': 6, '[20': 6, '110': 1, 'KOAc,': 1, 'Tween-20,': 1, 'PIC': 2, '0.05': 4, 'U/µL': 2, 'inhibitor].': 1, '0.005': 1, 'TurboDNase': 2, 'AM2238)': 4, 'added,': 1, 'coated': 2, 'anti-Sla1p': 1, 'control,': 4, 'immunoprecipitations': 3, 'empty': 4, 'plasmid.': 1, 'extraction.RNA': 1, 'qScript®': 1, 'microRNA': 1, 'Kit': 5, '(QuantaBio).': 1, 'intron-specific': 1, 'interest': 1, 'PCR-amplified': 1, 'Taq': 2, 'PerfeCTa': 1, 'Universal': 1, 'Primer': 1, '(QuantaBio)': 1, 'anneals': 1, 'oligo-dT': 1, 'incorporated': 2, 'agarose': 6, 'ligated': 1, 'pGEM-T': 1, 'Easy': 1, 'Vector': 1, 'Systems': 1, '(Promega)': 1, 'Plasmid': 1, 'extracted,': 1, 'Hospital': 1, 'Sick': 1, 'Children': 1, 'Centre': 1, 'Applied': 1, 'Genomics': 7, '(TCAG).tRNA': 1, 'analysis.A': 1, 'scrape': 1, 'information': 2, 'M.': 4, '(GtRNAdb)': 1, 'UCSC': 2, 'Browser': 1, '()': 2, 'Trailer': 2, 'minimum': 2, 'DNA.': 1, 'excluded': 1, '+1': 1, 'N73,': 1, 'excluding': 1, '(official': 1, '(Sprinzl': 1, '1998)).': 1, 'Statistical': 2, 'WebLogo(Crooks': 1, '2004)': 1, 'identity.': 1, 'Immunofluorescent': 1, 'prepared': 8, '(Wenkert': 1, 'Allis': 1, 'Centrifugation': 1, 'fixative': 1, 'parts': 2, 'saturated': 1, 'HgCl2': 1, '95%': 2, 'ethanol)': 1, 'suspension': 3, 'temperature,': 1, 'collecting': 1, 'methanol': 1, 'twice.': 1, '(1:500': 1, 'dilution)': 1, 'goat': 3, 'anti-rabbit': 5, '(1:10,000,': 1, 'ThermoFisher': 1, 'A11008)': 1, 'dropped': 1, 'coverslip,': 1, 'airdried': 1, 'mounted': 1, 'Vectashield': 1, 'Antifade': 1, 'Mounting': 1, 'DAPI': 1, '(Vector': 1, 'Laboratories)': 1, 'onto': 8, 'microscope': 2, 'slide.': 1, 'Microscopy': 1, 'images': 1, '63x': 1, 'magnification': 1, 'LSM700': 1, 'confocal': 1, 'laser': 1, 'scanning': 1, '(Zeiss)': 1, 'ZEN3.3': 1, '(blue': 1, 'edition).': 1, 'blotting.RNA': 3, 'solution': 4, '[100%': 1, 'slurry': 1, 'µg': 1, 'GlycoBlue': 4, 'Coprecipitant': 1, 'AM9515)]': 1, 'airdried,': 2, 'Samples': 2, 'volume': 8, '2X': 4, 'formamide': 4, '[80%': 4, 'deionized': 4, 'formamide,': 4, '0.06%': 8, '(w/v)': 11, 'xylene': 4, 'cyanol,': 4, '8.0],': 2, 'ice.': 4, '100V': 2, 'charged': 3, 'nylon': 3, 'membrane': 10, '(PerkinElmer),': 1, 'UV': 5, 'Stratalinker': 4, 'drying': 3, '(Bio-Rad).': 4, 'membranes': 9, 'hybridized': 6, 'melting': 5, '(Tm)': 3, '-10°C': 3, '[6X': 3, 'SSC': 9, '(1X': 3, 'SSC:': 3, 'citrate),': 3, '4X': 3, 'Denhardt’s': 3, '(Bio': 3, 'Basic': 3, 'D0062)],': 3, '20-minute': 1, '[2X': 3, 'SDS]': 4, 'strip': 3, 'stripping': 3, '[0.1X': 3, '70°C.Western': 1, 'blotting.Protein': 1, 'Bradford': 1, 'Scientific,': 7, 'cat#23300).': 1, '6.8]': 2, 'electrophoresis': 4, '12%': 3, 'nitrocellulose': 3, 'membrane.': 1, 'blocked': 2, 'skim': 2, 'milk': 2, 'Tris-Buffered': 2, 'Saline': 2, 'NaCl]': 2, 'Tween20': 2, '(TBST)': 2, '(or': 8, '4°C),': 1, 'TBST': 2, '4°C).': 2, 'TBST,': 1, 'HRP-conjugated': 2, '1:10,000': 2, 'dilutions': 2, 'study:': 4, 'anti-beta': 2, 'actin': 2, '(abcam': 5, 'ab8224),': 2, 'anti-histone': 2, 'ab1791),': 2, 'anti-His': 1, 'ab18184)': 1, 'anti-Sla1p.': 1, 'HRP-coupled': 2, '(Cell': 4, 'Signaling': 4, 'Technology': 4, '7074)': 2, 'horse': 2, 'anti-mouse': 2, '7076).Genomic': 1, 'thermophila.Genomic': 1, '25': 4, '0.5': 6, '3.5': 1, '[42%': 1, 'urea,': 1, '350': 1, 'Tris,': 3, 'SDS,': 2, 'mg/mL': 1, 'Proteinase': 1, 'K]': 1, '50°C.': 1, '(25:24:1),': 1, 'chloroform:isoamylalcohol': 1, '(24:1)': 1, 'One-third': 1, 'isopropanol.': 1, 'pelleted': 1, 'Tris-EDTA': 1, '(TE),': 1, '8.0': 1, '[10': 6, '8.0].': 2, '(10': 2, 'mg/mL)': 1, '55°C': 1, '-20°C.Genomic': 1, 'digested': 4, 'EcoRI': 3, 'capillary': 1, 'forces,': 1, 'product': 1, '“Northern': 1, 'blotting”.': 1, 'reproducibility': 2, 'statistics.Mlp1': 1, 'RIP': 5, 'triplicates.': 3, 'size-excluded': 1, 'Mlp1-RIP,': 1, 'duplicates': 1, 'unless': 3, 'stated': 3, 'otherwise': 2, 'legends.': 1, 'triplicates': 1, 'legend.': 1, '3’-RACE': 1, 'duplicates.': 1, 'Comparison': 5, '(F': 2, '289.3,': 1, 'DF=6,': 2, 'value': 4, '0.0001)': 1, '0.9931,': 1, '0.428).Data': 1, 'availabilityTGIRT-sequencing': 1, 'deposited': 1, 'paper.Code': 1, 'availabilityCode': 1, 'Code': 1, 'scraping': 1, '.Supplementary': 1, 'Figures': 2, 'TablesFigure': 1, 'Motif': 1, 'alveolates.a,b': 1, 'variable.': 1, 'species:': 1, '1S29)': 1, '2004),': 2, 'teal': 1, '2M5W)': 1, '(Apostolidi': 1, 'La-U-1,': 1, 'interactions.': 6, 'possessing': 1, 'ends.': 1, '(human': 1, 'numbering)': 1, 'Carbon': 4, '(U-1': 1, 'U-2)': 1, 'red;': 4, 'dashed': 3, 'lines.Figure': 1, 'pageSupplementary': 1, 'demonstrates': 1, 'pre-tRNAs.a': 1, 'ribonucleoprotein-immunoprecipitation': 2, '(RNP-IP)': 2, 'western': 2, 'blotting.': 3, 'heavy': 2, '(50': 2, 'kDa)': 2, 'IgG.': 2, 'b,c': 5, '(n=3)': 2, '(b)': 9, '(n=2)': 2, '(c).': 6, 'Fold': 8, 'hLa-immunoprecipitated': 1, 'GlnUUA.': 1, 'See': 1, 'S1': 1, 'quantifications.': 1, 'plots.': 1, 'Biological': 1, 'represented': 5, 'reads.': 1, '(a),': 2, '(b),': 2, '(c)': 3, 'R2': 1, 'cor': 1, 'R.': 1, 'TPM:': 1, 'million.': 1, 'RNAs.a': 1, 'CUGCUGUUUU-3’OH.': 1, 'Unbound': 1, 'gel,': 2, 'upwards.': 1, 'events.': 1, '1:1': 1, 'denoted': 1, '2:1.': 1, 'only)': 1, '(middle': 1, 'only).': 1, 'Denaturing': 1, 'side-by-side': 2, 'unlabelled-competitor': 1, '(lacking': 1, 'labeling)': 1, '(d,e)': 1, 'successful': 2, 'group.': 1, 'unlabelled-competitors': 3, 'dephosphorylated': 1, 'decrease': 8, 'NP:': 4, 'lane,': 4, 'NC:': 4, 'lane.Supplementary': 1, 'uridylate.': 1, 'control),': 1, 'lane.': 3, 'C10.': 1, 'c,d': 1, '(4U),': 1, 'e': 4, '(U10-3’-P)': 1, '(U10-3’OH)': 1, 'U10-3’-OH.': 1, 'pombe.a': 1, 'act1.': 2, '(IP)': 1, 'strains.': 11, 'antibodies.': 2, 'flow': 1, 'through.': 1, 'Confirmation': 1, 'strain.': 2, 'overview': 1, '3’-flanking': 2, 'neomycin': 1, '(neoTet)': 1, 'marker.': 2, 'Restriction': 1, 'cloning': 1, 'below': 3, 'locus.': 2, 'digest': 4, 'southern': 1, '1262': 1, 'bp': 2, 'fragment': 2, '945': 1, 'PCR-generated': 1, 'schematic': 1, 'a).': 1, 'Indirect': 2, 'polyclonal': 2, 'Nuclei': 2, 'stained': 3, 'DAPI.': 2, 'arrows': 5, 'denoting': 2, 'transcriptionally': 6, 'macronucleus': 6, 'inactive': 2, 'micronucleus.': 2, 'affinity.a,b': 1, 'lengths.': 1, 'Distribution': 4, 'frequency': 1, 'adenosines': 1, 'position.': 1, 'ELAC2': 1, 'revealing': 1, 'Z/ELAC2': 1, 'Regions': 1, 'PDE.': 1, 'interacting': 6, 'Sm1': 1, 'Sm2': 1, 'motifs': 8, '4M7D)': 1, '6PPN)': 1, 'purple.': 5, 'loops.Supplementary': 1, 'signals.S.D.:': 1, 'NCBI': 2, 'contributionsGenuine': 1, 'Kyra;': 1, 'Laganas,': 2, 'Jarred,': 1, 'E.;': 1, 'A.Department': 1, 'preparationtAuthor': 1, 'contributions:Jarred': 1, 'Laganas': 2, 'Pearlman': 2, 'locations': 12, 'thermophila.AbstractSplicing': 1, 'numerous': 1, 'complexes.': 4, '(U1': 2, 'U2),': 2, 'heterodimer': 2, '(U4/U6)': 2, 'heterotrimer': 2, '(U4/U6∙U5).': 1, 'RNA-RNA': 18, 'snRNA.': 10, 'Nrd1-Nab3-Sen1': 2, '(NNS)': 2, 'exo-': 1, '2,2,7-trimethyl': 2, 'guanosine': 3, 'trimethylguanosine': 4, 'synthase-1': 4, '(tgs1)': 2, 'enzyme.': 4, 'pre-5S': 3, 'instead,': 2, 'IntroductionProtein': 1, '(pre-mRNAs),': 1, 'interrupt': 1, '(CDS).': 1, 'Removal': 1, 'Gehring': 1, 'Roignant': 8, '(U': 1, 'snRNAs),': 1, 'multi-step': 1, 'subcellular': 5, 'compartments': 1, 'complexes:': 1, '(Gehring': 7, 'catalyzing': 2, 'reaction': 6, 'excising': 1, 'boundaries': 1, '(Villa': 1, '(Xiong': 2, '12.4': 1, 'gene,': 3, '0.06': 1, 'Paramecium': 1, 'tetraurelia': 1, '2.3': 1, '3.6': 1, 'splicing.The': 1, '(U4/U6∙U5)': 1, '21,': 1, 'clarity).': 1, 'base-pairing': 2, 'snRNP': 60, '5’-splice': 3, '21': 5, 'complex).': 2, 'Early': 3, 'particle': 2, '22a).': 1, 'Then,': 2, 'U4/U6∙U5': 18, 'tri-snRNP': 18, '22b),': 1, 'assembling': 2, 'pre-catalytic': 1, 'complex)': 2, 'rearrangements': 2, 'snRNPs,': 5, 'B*': 2, 'spliceosome,': 2, 'transesterification': 2, 'excision': 2, 'lariat': 3, 'ligation': 2, 'exons.': 2, 'released,': 1, 'remaining': 4, 'pre-mRNAs.Precursor': 1, '(pre-mRNA)': 1, 'spliceosome.': 3, 'exons': 2, '99%': 1, 'GU': 1, 'AG': 1, 'dinucleotide': 1, '3’-splice': 1, 'point,': 1, 'pre-catalytically': 1, 'completes': 1, '2021).Figure': 1, 'Organization': 1, 'tri-snRNP.': 1, 'di-snRNAs': 1, 'shown):': 1, 'heteroheptameric': 10, 'rings': 2, 'Sm': 29, 'ring.': 1, 'present.': 1, 'site:': 2, 'LSm': 5, 'SL:': 1, 'loop,': 1, 'VSL:': 1, 'loop.': 1, 'snRNA,': 9, '(Nguyen': 1, '2015).The': 1, 'non-polyadenylated': 1, 'Richard': 1, 'Manley': 2, 'co-transcriptional': 2, 'Nrd1': 11, 'Nab3': 19, 'consensus': 15, 'GUA[A/G]': 7, 'UCUU,': 4, '(Carroll': 4, 'rate': 2, 'decreases': 1, 'released': 1, 'helicase': 4, 'Sen1': 4, 'Steinmetz': 2, '2001;': 2, 'Brow': 1, '1996).': 2, 'intended': 1, '(Allmang': 1, 'endonucleolytically': 4, '(Chanfreau': 2, 'Seipelt': 2, '7-methylguanosine': 3, 'hypermethylated': 2, '2’-O-methylations': 2, 'pseudouridylations': 2, '(Mouaikel': 7, 'Karijolich': 2, 'Addition': 1, 'transport': 1, 'back': 2, 'homologue': 1, 'Senataxin,': 1, 'DNA/RNA': 1, 'homologue,': 1, '(Suraweera': 1, 'Instead,': 2, 'Integrator': 7, 'Complex': 5, 'cleaves': 2, '3’-box': 4, 'cis-acting': 7, 'enabling': 2, '(Baillat': 2, 'termination,': 2, 'assemble': 4, '(Mattaj': 4, '1986).': 3, 'Upon': 1, 'synthesized,': 1, '23)': 1, '(Jády': 3, 'Mattaj': 1, '1986).Figure': 1, 'Spliceosomal': 1, 'compartments.': 1, 'nucleus.': 3, 'm7G-cap': 3, 'exported': 1, 'cytoplasm,': 2, 'SMN': 1, 'assembles': 4, 'replaces': 2, '(TGS1)': 2, 'hypermethylation': 2, 'signals': 5, 'Snurportin-1': 2, 'Importin': 1, 'β': 1, '(snRNP)': 1, 'remodelling': 1, 'bodies.': 2, '(Pellizzoni': 1, '2007).The': 1, 'aspects.': 1, 'adenylates': 1, 'Kunkel': 1, '1986;': 2, '1987).': 1, 'tgs1': 4, '2002),': 1, 'γ-methyltransferases': 1, '1989;': 1, 'Lastly,': 1, 'snRNP-specific': 1, '(B/B’,': 1, 'D1,': 1, 'D2,': 1, 'D3,': 1, 'E,': 1, 'F': 1, 'G),': 1, 'ring-shaped': 2, '(Raker': 1, 'Raker': 1, 'On': 3, 'hand,': 3, 'lacks': 4, 'Sm-Like': 1, '(LSm)': 1, '(LSm2-8)': 1, '(Achsel': 1, 'snRNP,': 3, '(Rappsilber': 3, '2002).Prior': 1, 'biogenesis,': 1, '(Rinke': 3, '1982;': 1, 'Rinke': 1, '1985).': 4, 'matures': 1, '(Guddat': 1, 'Terns': 2, '1992).': 4, '(Pannone': 4, 'pre-U6': 1, 'elongated': 1, 'subsequently': 3, '2’3’-cyclophosphate,': 1, 'reduces': 1, '10-fold': 1, '(Lund': 1, 'Dahlberg': 1, '1992;': 1, 'Achsel': 1, '1997),': 1, 'synthetically': 1, 'stability.': 2, 'Similarly,': 2, 'Pannone': 1, '1998),': 1, 'shared': 1, '(Kerkhofs': 17, 'study,': 4, 'yeast-like': 1, 'describe': 1, 'site,': 5, 'Ndr1': 2, 'site.': 3, 'UAAUCAAACAAAA': 7, 'close': 2, 'proximity': 2, 'observations,': 1, 'RNP-immunoprecipitations': 1, 'ResultsRNA': 1, 'newly': 7, 'terminating': 2, 'potential': 13, 'location': 7, 'estimate': 1, 'detailed': 1, 'Integrated': 1, 'Viewer': 6, '(IGV)': 1, 'browser': 1, 'published': 10, 'chromosomal': 9, '24,': 1, 'UCUU': 7, 'immediate': 2, 'region,': 4, '24': 1, 'orange).': 2, 'additional,': 1, 'unidentified': 8, 'Gapped': 1, 'Local': 2, 'Alignment': 2, 'Motifs': 1, '(GLAM2)': 1, 'search': 6, 'Nab3/UCUU': 3, '24).': 1, '24),': 1, 'extended': 2, 'observe': 5, '(data': 5, 'shown).': 5, 'Overall,': 2, '(UCUU)': 2, 'end.': 1, 'stretches': 1, 'intermediate,': 1, 'Nab3/UGUU': 1, 'Ndr1/GUA[A/G],': 1, 'motif.Zoomed': 1, 'display': 1, 'y-axis': 3, 'boxed': 4, '(Orum': 3, '1991).': 3, '(GTA[A/G])': 1, 'Note': 1, 'Integrative': 5, '(IGV).Table': 1, 'Chromosomal': 3, '1991).Mlp1': 1, 'reanalysed': 3, '25a).': 1, 'La/Mlp1': 1, '2022),': 1, 'Investigation': 1, 'mapped': 1, '25a),': 1, 'tailed': 1, '25c).': 1, 'Immunoglobin': 2, '25c,': 1, 'concluded': 2, '25c': 1, 'Mlp1-enrichment': 2, 'qRT-PCR': 11, '25d).': 1, 'IgG-immunoprecipitated': 3, '25d': 1, 'blue)': 1, '25d,': 1, 'grey)': 2, '17S': 8, 'I.': 2, 'y-axis.': 3, '(top': 2, 'rows)': 3, '(bottom': 2, 'snRNA-snRNA': 1, 'red/beige,': 1, 'snRNA-mRNA': 1, '5’SS:': 1, 'splice': 1, 'BPS:': 1, '(IGV).': 2, '(17S': 3, 'rRNA)': 4, 'snRNAs)': 2, 'grey.': 4, 'ΔCt': 6, 'method': 9, '(Ct': 5, 'Ct': 5, 'input),': 2, 'Mlp1-RIP': 3, 'IgG-RIP': 3, '(n=1': 3, 'replicate).': 3, 'control.To': 1, 'LC-MS/MS.': 1, 'Mlp1-immunoprecipitation': 9, 'IgG-immunoprecipitation': 1, '10a).': 1, 'protein-protein': 13, 'benzonase,': 1, 'degrades': 1, 'interactors': 8, 'silver': 1, '10b).': 1, 'LC/MS-MS': 1, 'peptides': 3, 'significant': 4, 'Mlp1-interacting': 3, 'sample': 2, '26a).': 1, 'Sm-B,': 1, '26b)': 1, '(Leung': 3, 'Sm-B': 9, '5’-AUUUUUG-3’': 1, 'methionine-38': 1, '(M38)': 1, 'asparagine-39': 1, '(N39)': 1, '26c,d).': 1, '26e),': 1, 'similarly.': 1, 'Sm-B.': 1, 'Mass': 3, 'spectrometry': 5, 'volcano': 2, 'plot': 2, 'significantly': 9, 'immunoprecipitation.': 1, 'Dotted': 2, 'lines': 2, '(-log10': 2, 'or)': 2, '(log': 2, '2-fold': 2, 'change': 4, 'x-axis.': 2, '(bait)': 2, 'diamond': 2, 'shape': 2, 'Front,': 1, 'SmE–SmG–SmD3–SmB–SmD1–SmD2–SmF.': 1, 'Mlp1-protein': 1, 'interactor': 1, 'SmB': 1, 'orange': 5, 'visualization': 1, 'AUUUUUG': 2, '4WZJ).': 1, 'Zoomed': 3, 'panel': 1, '(uridylate': 1, 'sulfur': 3, 'Residues': 3, 'conservation.Depletion': 1, 'localizationSince': 1, 'abundance.': 4, 'abundances': 1, '27a)': 1, '27b).': 2, 'substantial': 2, 'steady-state': 1, 'replicates)': 2, 'Band': 2, 'intensity': 2, '(5.8S': 2, 'reimport': 1, 'transiently': 1, 'travels': 1, 'Didychuk': 2, 'mislocalize': 1, 'dysfunctional': 1, 'cytoplasm.': 1, 'aberrant': 2, 'fractionation': 6, 'separating': 1, 'compartment': 3, 'β-actin': 2, '28a)': 1, 'snRNA-specific': 1, 'probes.': 2, '28b),': 1, 'snRNAs.a': 1, '(cyt)': 1, '(nuc)': 1, 'fraction': 2, 'Tgs1': 12, 'conservedRNA': 1, 'Mouaikel': 1, '5’-cap': 4, 'mammalian': 1, 'm3G-caps.': 1, 'resistant': 1, 'periodate': 1, 'oxidation': 1, 'β-elimination': 1, '3’-hydoxyl': 2, 'ribose,': 1, '(Pedersen': 3, '(Tgs1)': 1, 'conversion': 1, 'dimethyl': 1, 'transferase': 2, 'successive': 1, 'transfers': 2, 'S-adenosyl-L-methionine': 1, '(AdoMet)': 2, '(Monecke': 4, 'spatially': 1, '29a).': 1, 'AdoMet': 10, '29b,c).': 1, '29d).': 1, 'Strikingly,': 2, '29d': 2, 'blue),': 2, 'AlphaFold': 3, '(Jumper': 1, 'TGS1': 5, 'fold,': 1, 'alpha': 1, 'even': 1, 'm3G-cap.Figure': 1, 'Conservational': 1, 'm3G-capping': 2, '3GDH)': 2, 'm7GTP': 8, 'trimethyl': 2, 'synthetase-1': 1, 'panels': 2, 'highlight': 2, '(blue)': 1, '(orange)': 1, 'dual': 2, 'asterisk.': 1, '(AdoMet': 1, 'respectively.Next,': 1, 'm3G-like': 3, 'capped': 3, 'anti-m3G': 7, 'bands': 2, 'elution': 1, 'lanes,': 1, 'smear': 1, 'visualized': 1, 'ethidium': 5, 'bromide': 3, '(EtBr)': 5, 'staining,': 1, '30a).': 1, 'antibody,': 2, '30b).': 1, 'Unexpectedly,': 1, 'saw': 1, '30b)': 1, 'fact': 1, 'di-snRNP,': 2, 'possibly': 1, 'non-specific': 2, 'elution.': 1, 'Having': 2, 'Detection': 1, 'Mlp1-associated/m3G-antibody': 1, 'anti-m3G-cap': 1, '30d),': 1, 'end.Figure': 1, 'a,c': 3, 'EtBr': 1, 'EtBr:': 1, 'bromide.': 1, 'b,d': 2, 'ureal': 1, 'U5),': 1, 'affected': 3, 'depletionThe': 2, '10-30%': 7, 'gradients': 6, 'separate': 5, 'density,': 2, 'defect': 1, 'lighter': 5, '(top)': 1, 'fractions': 5, 'blot,': 2, '(Makarova': 1, 'Tanackovic': 1, '31a).': 1, 'depletion,': 6, 'shifted': 2, 'fractions,': 2, 'fewer': 1, '31b).': 1, 'et.': 2, '31c).': 1, 'Ethidium': 4, 'Bromide': 4, 'stain': 3, 'detect': 3, 'distributions': 2, 'b.': 2, 'gradient.': 3, 'Fractions': 4, 'Localization': 2, 'co-localization': 2, 'patterns': 1, 'weight': 3, 'di-snRNPs': 1, 'splicingNext,': 1, 'steps.': 3, 'base-pairs': 1, '22).': 1, 'RNP-immunoprecipitation': 5, '32a).': 1, 'conditions': 3, 'preserve': 1, 'U4-specific': 1, '32b).': 1, '32c).': 1, 'Total': 3, 'U6-specific': 1, '32d,e': 1, 'di-snRNP).': 1, 'RNA-protein': 1, 'denatures': 1, '(Rio': 1, '32d).': 1, 'unwound': 1, '21),': 1, 'U6,': 2, 'H3.': 2, '(P-value)': 1, 't-test.': 2, '0.05:': 1, '*,': 1, '0.01:': 1, '**.': 1, 'defectsTo': 1, 'investigating': 4, '(Tian': 1, 'Specifically,': 1, 'NLRC3': 1, '(TTHERM_00327090)': 1, 'hypothetical': 1, '(TTHERM_00526420)': 1, 'RT-PCR': 3, 'amplification,': 1, 'amplify': 2, '31a,c).': 1, 'unspliced': 3, '31b,d)': 1, 'mRNAs.': 1, 'Pre-mRNA': 1, 'depletion.a,': 1, 'Schematics': 1, '(containing': 2, 'introns)': 1, 'introns).': 1, 'rectangles': 1, 'thin': 1, 'introns.': 2, 'Blue': 1, 'amplicon': 1, 'left': 1, 'b,': 1, 'upper': 2, 'mRNA.': 2, 'Ratios': 1, '(unspliced)': 1, '(spliced)': 1, 'image': 2, 'd).': 1, 'Discussion': 1, 'NNS': 1, 'Sen1.': 1, '34a)': 2, 'Allmang': 1, 'repeats': 2, 'detected.': 1, 'distantly': 1, 'Ciliates': 1, 'RRM44': 2, 'polypyrimidine': 2, 'tract-binding': 2, '(TTHERM_00685970),': 1, 'vertebrates.': 1, 'utilized': 2, 'heterodimers': 1, '34a,b)': 1, '(Hobor': 1, 'element': 3, 'recruiting': 1, 'blocking': 2, 'progression': 2, '34c).': 1, 'Reb1': 1, '11': 1, '53': 1, '3’-downstream': 2, '(Németh': 1, 'Without': 1, 'DNA-based': 2, 'likely.': 1, 'biotin-tagged': 1, 'mass': 7, 'spectrometry.': 3, 'allow': 5, 'design': 1, 'trans-acting': 4, 'Nucleolar': 1, 'Nab3/Ndr1/Sen1': 2, '(Steinmetz': 1, 'searched': 1, 'absent.': 1, 'expression.Figure': 1, 'Conserved': 2, 'Nab3,': 1, 'Novel': 2, 'RNA-': 2, '(pol': 1, 'II)': 1, 'initiates': 1, 'element.': 1, 'RNA-based': 1, 'homolog,': 1, 'UAAUCAAACAAAA,': 2, 'TAATCAAACAAAA': 1, 'TTTGTTTGATTA': 1, 'strand,': 1, 'BioRender.In': 1, '13-16': 1, 'GTTTN0-3AAA[A/G]NNAGA': 1, '(Yuo': 1, '1985;': 1, 'Hernandez': 1, '1985)': 1, '(Uguen': 1, '(Mendoza-Figueroa': 1, 'proteome': 1, 'integrator': 3, '(70%,': 1, 'percentage': 1, 'IntS1-14).': 1, '18%': 1, 'plants,': 1, '2%': 3, 'contained': 2, '(Kirstein': 1, 'this.': 1, 'Xue': 2, 'Based': 2, 'analysis,': 1, 'potentially': 2, 'Taken': 1, 'together,': 1, 'Nrd1/GUA[A/G]': 2, 'signal(s)': 1, 'cerevisiae.': 1, 'blotting,': 1, 'reanalyzed': 2, 'RNA-sequence': 1, 'co-immunoprecipitation': 1, 'remains': 7, 'Will': 1, 'Lührmann': 1, 'failed': 1, 'ring,': 2, 'only.': 1, 'preparation.': 1, 'protein-proteins': 1, 'investigation': 1, 'conservational': 2, 'Resource': 1, 'Coordinators': 1, 'stage': 1, 'already': 1, 'structure).': 1, 'catalytical': 1, 'converts': 1, 'Monecke': 1, 'Tgs1.': 1, 'divergent': 4, 'link': 1, 'semi-conserved': 1, '5’-m3G-like': 1, 'investigation.': 1, 'Nonetheless,': 1, 'RNA-immunoprecipitations': 1, 'm3G-specific': 1, 'enriching': 1, 'targets,': 3, 'Specificity': 1, 'monophosphate': 1, '5’-terminus': 1, 'm3G-antibodies': 1, 'distinguish': 1, '(Reuter': 1, '1984),': 1, 'resembles': 1, 'gradient': 4, 'sediment': 1, 'incomplete.': 1, 'Assembly': 3, 'Sm-proteins': 1, 'LSm-proteins': 1, 'Snu13,': 1, 'Prp31,': 1, 'Prp3,': 1, 'Prp4': 1, 'others': 1, '(Hardin': 1, 'di-snRNA': 1, 'unpaired': 1, 'decreased.': 1, 'appropriately': 1, 'association.': 1, 'Inappropriate': 1, 'efficiency,': 1, 'intron-retention': 1, 'Currently,': 1, 'unable': 1, 'influences': 1, 'whole': 1, 'extracts': 2, 'assembly.Mlp1': 1, 'moderately.': 1, 'removal,': 1, '(Tanackovic': 1, 'complexes,': 3, 'time': 1, '21).In': 1, 'outstanding': 4, 'answered': 1, 'U4.': 1, 'answer': 1, 'question,': 2, '(CLIP),': 1, 'comprise': 2, 'sites.': 1, 'Future': 3, 'Mutational': 1, 'disruption': 1, 'uridylate-rich': 1, 'RNAs.': 3, 'presents': 1, 'explain': 1, '1998)': 2, 'Sm-D,': 1, 'temperatures': 1, 'spliceosome-associated': 1, 'spliceosomes': 1, 'identified,': 2, '96': 1, 'date,': 1, 'yet.': 1, 'MethodsMotif': 1, 'prediction.We': 1, 'annotations': 1, '(U1:': 1, 'chr_177:903140-902977,': 1, 'U2:': 1, 'chr_121:99366-99174,': 1, 'U4:': 1, 'chr_131:426400-426267,': 1, 'chr_058:3123508-3123391)': 1, 'browser.': 1, 'GLAM2': 1, '(Gapped': 1, 'Motifs)': 1, '(MEME': 1, 'Suite': 1, '5.5.0)': 1, '[1%': 1, 'Fe-EDTA]': 1, '1000g,': 1, '7.4.': 1, 'Pellets': 1, 'otherwise.': 1, 'homogenization,': 1, '-20°C': 1, 'RNP-': 1, 'immunoprecipitations.Cell': 1, 'store': 1, 'use.': 1, 'Lysates': 3, 'min': 3, 'RNP-immunoprecipitation,': 1, 'slurry)': 1, 'immunoprecipitations.': 1, 'protocol,': 1, 'denaturation': 2, 'Instead': 1, 'cross-linking,': 1, 'proteinase': 2, 'K': 2, '(40': 1, 'temperature)': 1, 'O0491)': 1, 'isolate': 1, 'm3G-capped': 1, '(Hayes': 1, '2018)': 3, 'adaptations.': 1, '7.4;': 8, 'NaCl;': 2, 'NP40]': 1, '(Sigma,': 1, 'MABE302)': 1, 'buffer,': 1, 'G-50': 1, '8.0;': 7, '0.25%': 1, 'SDS].': 3, 'Scientific)': 3, '-80°C.Northern': 1, 'denature': 1, 'placed': 1, 'non-denaturing': 2, 'heating.': 1, '15585011)': 1, 'irradiation,': 1, 'NEF1017001PK),': 1, '(Table': 4, '4).': 3, 'Prior': 2, 'exposure': 2, 'overnight,': 1, 'Storage': 1, '70°C.Table': 1, 'detection.': 3, 'qRT-PCR.Total': 1, 'Reaction': 4, 'Turbo': 2, 'DNase': 5, '37°C.': 2, 'DNase-treated': 5, 'inactivate': 2, 'activity.': 5, 'ng/μL': 3, 'iScript': 3, '(Bio-Rad,': 3, 'cat#1708891)': 3, 'according': 4, 'manufacturer’s': 5, 'instructions.': 4, 'diluted': 2, 'ng': 4, '12.5': 4, 'technical': 2, 'triplicate)': 2, 'SensiFAST': 2, 'SYBR': 2, 'No-Rox': 2, 'kit': 2, '(Bioline,': 2, 'BIO-98005)': 2, 'μM': 3, '(1x': 2, 'forward,': 2, '1x': 2, 'reverse)': 2, 'qPCR': 2, 'settings:': 4, '35': 3, 'cycles': 3, '60°C,': 2, '99°C': 2, 'amplicon.': 2, 'RNA-immunoprecipitation,': 1, '(CtRIP': 1, 'Ctinput),': 1, 'ratios.': 2, 'Protein-protein': 4, '(Nabeel-Shah': 1, '(previously': 2, '500': 5, 'thawed': 1, '[40': 1, '600': 1, 'NP-40;': 1, 'MgCl2;': 5, 'Scientific)].': 3, 'benzonase': 3, 'nuclease': 1, '(Millipore,': 1, '70746)': 1, '16,100': 1, 'clarify': 1, 'lysate.': 1, 'high-salt': 1, 'IPP300': 1, 'NP-40],': 1, 'IPP100-NP-40': 1, 'NP-40]': 1, 'IPP100': 1, 'NaCl].': 1, 'freshly': 1, 'NH4OH': 1, 'rotated': 1, 'lyophilized': 1, 'vacuum': 1, 'concentrator': 1, '(Eppendorf': 1, 'Vacufuge).': 1, 'fragmented': 1, 'in-solution': 1, 'tryptic': 2, 'LC-MS/MS': 2, 'IRCM': 1, 'Proteomics': 1, 'Discovery': 1, 'Platform': 1, 'Q': 1, 'Exactive': 1, 'HF.': 1, 'Scaffold': 1, 'v5.2.0': 1, 'threshold,': 2, 'FDR': 1, '1086': 1, 'Spectrum': 1, 'one-tailed': 1, 'Volcano': 1, 'GraphPad': 1, 'Prism': 1, 'v9.0.': 1, 'Silver': 3, 'separation': 2, 'ProteoSilver': 1, 'Stain': 1, '(Millipore': 1, 'Sigma,': 1, 'PROTSIL1)': 1, 'Cellular': 1, 'blotting.T.': 1, 'freezing)': 1, 'Triton': 3, 'X-100': 2, 'KCl].': 1, 'X-100,': 1, '40': 1, '2500': 1, 'supernatant,': 1, 'fraction,': 1, 'carefully': 2, 'pellet,': 1, 'nuclei,': 1, 'components.': 1, 'blot.': 2, '4°C)': 1, 'TBST.': 1, '7076).Conservation': 1, '(NP_003082.1),': 1, 'musculus': 2, '(NP_033251.1),': 1, 'rerio': 2, '(NP_991230.1),': 1, '(NP_001188780.1),': 1, '(NP_001032011.1),': 1, '(NP_594151.1),': 1, '(NP_010946.3),': 1, '(XP_001031807.1).': 1, 'included:': 1, '(NP_079107.6),': 1, '(NP_473430.3),': 1, '(XP_003197913.2),': 1, '(NP_732316.2),': 1, '(XP_636670.1),': 1, '(NP_973978.2),': 1, '(NP_593095.3),': 1, '(NP_015168.1)': 1, '(XP_001021693.2).': 1, '2011)': 1, 'script.': 1, '4WZJ': 1, '3GDH.': 1, '(amino': 1, '283–498)': 1, 'Colab': 1, 'notebook.Glycerol': 1, 'gradients.': 2, 'KCl;': 4, 'DTT;': 2, '0.2': 2, 'glycerol;': 2, '25U': 2, '5-step': 2, 'MgOAc;': 2, 'NP40': 2, 'PMSF]': 1, 'advance': 1, 'kept': 2, 'diffuse': 2, 'linear': 2, 'layered': 2, 'gently': 1, 'centrifuged': 1, '35,000': 2, 'RMP': 2, 'SW41Ti': 2, 'rotor)': 2, '(400': 2, 'uL/fraction)': 1, '(1:1': 2, 'volume)': 1, 'TCA': 2, 'brought': 1, '10%,': 1, 'assay.Total': 2, 'extraction,': 1, '(2U)': 1, 'Polymerase': 1, 'M0273L)': 1, 'fragments': 1, 'instructions': 1, 'initial': 1, '95°C,': 2, '45°C': 1, '68°C,': 1, '68°C.': 1, 'amplicons': 1, 'TrackIt': 1, 'Cyan/Orange': 1, '15585011).': 1, 'validation.': 2, 'interactors.': 2, 'G,': 1, 'control.Supplementary': 1, 'Highlighted': 1, '29b.': 1, 'AdoMet:': 1, 'S-adenosylmethionine,': 1, 'aa:': 1, 'contributionsThe': 1, 'Jarred1;': 1, 'Michelle': 1, 'S.2;': 1, 'A.11': 1, 'preparationAuthor': 1, 'contributions:Étienne': 1, 'rFam,': 1, 'snoscan': 4, 'SnoReport2.0.': 1, 'Jarred': 1, 'thermophila.AbstractSmall': 1, '(snRNPs)': 1, 'described.': 1, 'abundancy.': 1, '18': 3, 'high-confidence': 4, 'co-immunoprecipitates': 1, 'gradients,': 1, 'H-dependent': 2, 'Again,': 1, 'IntroductionIn': 1, 'spacers.': 1, 'spaces': 1, 'nucleolytic': 4, 'Klinge': 1, 'ribosome,': 1, 'peptidyl': 1, 'center,': 1, 'center': 1, 'E': 1, 'sites)': 1, 'intersubunit': 1, '(snRNPs),': 1, 'localize': 1, 'Three': 1, 'elements.': 3, 'class': 3, 'member,': 1, 'mitochondrial': 2, '(Tollervey': 1, '“boxes”)': 1, '35)': 2, '2’-O-methylation,': 2, 'double-stranded': 1, '2002).Figure': 1, 'snoRNAs.The': 1, 'closed': 2, '(RUGAUGA)': 2, 'D': 4, '(CUGA)': 2, 'C’': 2, 'D’': 4, 'ACA': 5, 'connected': 2, 'H': 24, '(ANANN': 2, 'motif)': 1, 'Target': 1, '2’-O-methylated': 2, 'pseudouridylated': 1, '2002).The': 1, 'motif).': 1, 'sets': 1, 'methyltransferase': 1, 'Nop1': 1, '(fibrillarin': 1, 'vertebrates),': 4, 'Nop56,': 4, 'Nop58': 6, 'Nop5)': 1, 'Snu13': 2, '(Cahill': 4, 'Nop1/fibrillarin': 1, '(Jansen': 1, 'likewise': 1, 'proteins:': 1, 'synthetase': 1, 'Cbf5p': 1, '(dyskerin': 1, 'Gar1,': 1, 'Nhp2': 3, 'Nop10': 5, 'Cbf5p/dyskerin.': 1, '2′': 1, '(2-O’-methylation)': 1, 'complementarity': 3, 'snoRNA-rRNA': 1, 'hybrid': 1, 'places': 1, '(Cavaillé': 1, 'Kiss-László': 1, 'isomerization': 1, 'snoRNA/rRNA': 1, 'duplexes': 1, 'NΨ)': 1, '(Ganot': 2, 'Ni': 1, 'U3,': 1, 'U8,': 1, 'U14': 2, 'U22': 1, '(Li': 2, 'Kass': 1, 'Peculis': 1, '1994),': 1, 'U17': 1, 'snR10': 1, '(Venema': 1, 'Enright': 1, 'cleavage.': 2, 'induce': 1, 'serving': 1, '(Dunbar': 1, 'Baserga': 1, 'Li': 1, '1990).': 1, '9000': 1, 'rDNA': 1, 'macronucleus,': 1, 'parental': 1, 'chromosome': 3, 'micronucleus': 1, 'copy': 2, '(Kapler': 1, '17S,': 1, '26Sα': 3, '26Sβ': 2, '(Kister': 3, '26S': 6, 'producing': 1, 'Fragmented': 1, 'ribosomes': 3, 'unicellular': 1, 'equivalent': 1, '(Barth': 1, 'Campbell': 1, '1987)': 1, '38': 3, '28': 3, '(Andersen': 19, 'Nielsen': 19, 'work,': 3, 'aim': 1, 'expand': 1, 'tools,': 1, 'obtaining': 1, 'dataset.': 4, '2’-O-methyltation': 1, 'defects': 1, '(Inada': 2, 'Guthrie': 7, 'microarray': 1, '(60-100': 1, '36a': 1, '(<': 1, 'abundancy': 3, '0.1%,': 1, '11%': 1, '1%,': 1, '36b': 1, 'abundancies': 3, 'a,b': 3, 'Pie': 1, 'charts': 1, 'percentages': 1, 'counts)': 1, 'hLa-': 1, 'n=2': 1, 'hLa-immunoprecipitation': 1, 'n=4': 1, 'input).': 1, '2017),': 2, '2022).Since': 1, 'reads,': 2, '37a).': 1, '37b': 1, 'Specific': 1, '37b,': 1, '37c)': 1, 'targets:': 1, '37c).': 1, 'snoRNA.': 3, 'rows).': 1, 'Typical': 1, 'C,': 3, 'D,': 2, 'Complementary': 1, 'underlined.': 1, 'R:': 1, 'purine': 1, '(guanine': 1, '(G)': 1, '(A)),': 1, 'N:': 1, 'nucleotide.': 1, 'C/D4': 1, 'H/ACA258': 1, 'grey,': 1, 'control.Mature': 1, 'nucleolus,': 1, '(rDNA),': 1, 'various': 2, 'stages': 1, '500-1000': 1, 'nucleoli': 1, 'inner': 1, 'envelope': 1, '(Nielsen': 3, 'From': 4, 'periphery': 1, '38).': 1, 'localizes': 1, 'nucleoli.Reannotation': 1, 'S7': 1, 'two-sided': 1, 'arrow': 1, 'illustrate': 1, 'nucleoli.': 1, 'Annotation': 1, 'Mlp1-enrichmentSince': 1, '(average': 2, '1),': 1, '39).': 2, 'know': 1, 'enrichments': 1, 'fold-enrichment': 2, '2022,': 1, 'dotted': 1, 'log10(2)': 1, '0.3.': 1, 'snoRNA,': 4, 'snoreport2.0': 3, '(de': 3, 'Araujo': 4, 'Oliveira': 4, 'Eddy': 2, '1999)': 1, 'reliably': 1, 'confidence': 3, '10)': 1, 'snoRNAs.Box': 1, 'filtered': 1, '(score': 1, '14)': 1, '(TPM': 1, 'orange.Mlp1': 1, 'abundanceSince': 1, 'Relative': 1, '40a,b).': 1, '1,': 1, 'validated': 3, '40c)': 1, '40d).': 1, '40c).': 1, 'TPM.': 1, 'C/D24': 2, 'strains.Mlp1': 1, 'proteinsRNA': 1, '(Falaleeva': 1, 'Nop10,': 2, 'Cbf5': 2, 'Dyskerin': 2, 'Gar1': 3, '41a)': 1, 'fibrillarin,': 3, '15.5': 3, 'kDa': 3, '41c)': 1, 'LC-MS': 2, '41b,d).': 1, 'enable': 1, 'sample,': 1, '41b': 1, 'reach': 1, 'threshold.': 1, '(Snu13': 1, 'annotated)': 1, 'Nop1/Fibrillarin': 1, 'enriched,': 2, 'Nop56': 2, '41d).': 1, 'archaeal': 1, 'Cbf5-Nop10-Gar1': 1, '(Nhp2': 1, 'structure)': 2, '17-mer': 1, '3MQK)': 1, 'Nop1/fibrillarin-Nop56': 1, '(Nop58': 1, '6ZDT)': 1, 'scheme': 1, 'fractionated': 2, '42a).': 1, '42b).': 1, 'slightly.': 1, 'Testing': 1, 'rRNAThe': 1, '(Lowe': 1, '(Yu': 4, 'chimeric': 13, 'oligonucleotides': 3, 'consist': 1, 'central': 4, 'deoxyribonucleotides': 1, 'flanked': 2, 'ribonucleotides': 2, '43a).': 2, 'oligonucleotide': 4, 'RNA/DNA': 2, 'hybrids,': 1, 'visualize': 2, 'undigested': 1, '43b': 4, 'strain,': 3, 'parts,': 1, '26Sβ,': 1, 'controls': 1, 'deoxyoligonucleotide': 1, '4,': 1, 'snoscan_3686': 3, 'target),': 2, 'snoscan_2791': 2, '(26S': 2, 'target)': 1, 'snoscan_3331': 5, 'designed': 1, '43a': 1, 'completely': 1, 'snoscan_2791,': 1, '3,': 1, '6,': 1, 'add': 2, 'target,': 4, 'alternatively': 3, 'chimeras': 2, 'optimized,': 1, 'modifies': 1, 'strain),': 1, 'deoxynucleotides': 1, 'duplex': 1, '(500': 1, 'ng)': 1, '100-fold': 1, 'excess': 1, 'oligonucleotides,': 1, 'double': 1, 'stranded': 1, 'DiscussionLa': 1, '3’-UUU': 1, 'tract': 1, '3’-extended': 1, 'Lhp1-immunoprecipitation': 1, 'snoRNP,': 1, 'Nop1,Nop56': 1, '(Kass': 1, 'Hughes': 1, 'Ares': 1, '1991)The': 1, 'precursors.': 1, 'sequencing.': 1, 'Cbf5,': 1, 'Nhp2,': 1, 'Nop58,': 1, 'statistically': 1, 'significant.': 1, 'limitations': 1, 'keep': 1, 'mind.': 1, 'detectable': 1, 'peptides,': 1, 'mixture': 1, 'proteome.Overall,': 1, 'RNA-DNA': 4, 'oligonucleotide-directed': 4, 'preliminary': 3, 'optimized.': 1, 'test,': 1, 'functionally,': 1, 'U3-55k': 1, '(Lukowiak': 1, 'current': 3, 'H/ACA11': 1, 'cannot': 1, 'exclude': 1, '2’-O-methylation.': 1, 'starvation,': 1, 'mating': 1, 'stress': 1, 'regulate': 1, 'condition-specific': 1, 'tailoring': 1, 'response': 1, '5.': 1, 'expansion': 1, 'novo': 2, 'transcriptome': 2, 'frequently': 1, 'quantify': 1, 'Dewey': 1, 'advantage': 2, 'avoids': 1, 'Mlp1-targets.': 1, 'algorithm': 1, 'machine': 1, 'learning': 1, 'Hertel': 2, 'snoreport,': 1, 'original': 1, 'snoreport2.0,': 1, 'inclusion': 2, 'disadvantage': 1, 'orphan': 4, 'unknown,': 1, 'HBII-52/SNORD115': 1, 'exon': 2, 'serotonin': 1, 'receptor.': 1, '(Kishore': 1, 'ensures': 1, '(62%)': 1, 'intergenic': 3, '38%': 1, '(9%),': 1, '(13%),': 1, 'intron/exon': 1, 'border': 1, '(3%)': 1, 'exon/5’-UTR/3’UTR': 1, '(13%)': 1, 'Approximately': 1, '25%': 1, 'clusters.': 2, 'cluster': 3, '123': 2, 'CD7,': 2, 'CD8,': 2, 'CD9,': 2, 'CD10,': 2, 'CD11,': 2, 'CD12,': 2, 'CD13,': 2, 'CD14,': 2, 'CD15': 2, 'CD16': 1, 'spacing': 1, '187': 1, '421': 1, '44)': 1, 'chromosomes': 1, '22,': 1, '65': 1, '181.': 1, '(USE),': 2, '5’-AAACCCATAA-3’,': 2, '54': 1, '179': 1, 'previously.': 1, 'clustered': 4, 'co-transcribed': 2, '5,': 1, 'exact': 1, 'USE': 1, '12': 1, 'diverging': 1, '5’-NNACCCATAA': 1, '5’-AAACCCATNN,': 1, 'Polycistronic': 1, '(IGV).We': 1, 'enrich': 1, '36b).': 1, 'GC-content': 1, 'AT-rich': 1, '(Eisen': 1, 'AT%': 1, '75%': 1, 'unannotated': 1, 'ncRNAs': 1, 'T3-4': 2, '(TTHERM_00048812)': 1, 'counted': 1, '127': 1, '1992)': 1, 'H/ACA24': 1, 'Unannotated': 1, 'overlapping': 1, '(IGV).MethodsRNP-immunoprecipitation': 1, 'Cross-linking': 1, 'Lysate': 1, 'clarification': 1, 'lysated': 1, 'extraction.Northern': 1, '8.0]': 1, '(PerkinElmer).': 1, 'Oligonucleotides': 1, 'blotting:': 1, 'TthnuCD24': 1, 'ATTCAGTCGCCAGTAGAACATCAGA-3’,': 1, 'TtnuHACA11a': 1, 'AAATGTCCTTATCTTGGTATCTTTT-3’,': 1, 'CTGCAATTCGCATTGCGT-3’': 1, 'TyrGTA': 1, '5’-GACCACCGGATTTACAGTCCG-3’.': 1, 'incubation,': 1, '20-minutes': 1, 'developing': 2, '70°C.qRT-PCR.Total': 1, '7)': 1, 'spectrometry.The': 1, '160:': 1, 'Method': 1, 'section': 1, 'spectrometry).': 1, 'Glycerol': 1, 'PMSF].': 1, 'sample.': 1, 'uL/fraction).': 1, 'volume).': 1, 'aqeous': 1, 'concentration,': 1, 'oligonucleotide,': 1, 'annealing': 2, '1.25': 1, '(NEB,': 1, 'M0297S)': 1, 'μL.': 1, 'mix': 1, 'stopped': 1, '[20g/L': 1, 'agarose,': 1, 'TBE': 2, '(100': 1, 'boric': 1, 'acid,': 1, 'EDTA),': 1, 'μL/mL': 1, 'bromide]': 1, 'run': 1, 'buffer.Oligonucleotides': 1, '5’-TTTCGCATTAGTTAGTCC-3’;': 1, 'CATTTAAGTAAACTCCGC-3’;': 1, 'mUmCGTTCmUmUmGmAmUmUmAmAmUmGmAmA;': 1, 'mCmAAGATmCmUmCmAmCmUmAmAmUmCmGmC;': 1, 'mUmUTCGGmGmUmCmCmUmAmAmCmAmAmUmU.Supplementary': 1, 'TablesSupplementary': 1, 'hLa-RNP': 1, 'immunoprecipitations.hLa': 1, 'table': 1, 'snoRNAs.Table': 1, 'pageTable': 1, 'perspectivesIntroduction': 1, 'originally': 2, 'autoimmune': 1, 'diseases': 1, '1974': 1, '1974b;': 1, 'years': 2, 'later,': 2, '1981,': 1, 'patient': 1, '1984,': 1, 'ten': 1, 'necessary': 1, '2004': 1, 'elaborated': 1, 'commonly': 1, 'Gajiwala': 1, 'Burley': 1, 'classical': 1, 'RRM-fold': 1, 'four-stranded': 2, 'α-helices,': 1, 'α3-helix': 2, 'prokaryotes': 1, 'viruses': 1, 'RRMs': 1, 'RNP1': 1, 'RNP2': 1, 'strands': 2, '(β1': 1, 'β3)': 2, 'β-sheet.': 1, 'Accordingly,': 1, 'putative': 1, 'single-stranded': 1, '(Maris': 1, 'came': 1, 'co-crystal': 1, '2006,': 1, 'lateral': 1, 'uridylate-tailed': 1, 'unanticipated': 1, 'keeps': 1, 'Huang': 2, 'Conte,': 3, 'now': 1, 'challenged': 1, 'presumption': 1, 'La-like': 1, 'recently,': 1, 'gathered': 1, 'NMR': 1, 'collaboration': 1, 'Conte': 1, 'King’s': 1, 'College': 1, 'London': 1, 'corroborates': 1, 'three-dimensional': 2, '(Kerkhofs,': 1, 'K.,': 1, 'Jarvis,': 1, 'J.,': 1, 'M.,': 1, 'unpublished': 1, 'data).': 1, 'solving': 1, 'NMR-based': 1, 'planned': 1, 'proteinIn': 1, 'aspects:': 1, 'deficient': 2, 'Q20/Y24/D33R': 1, '2018;': 1, 'LaM-residues': 1, 'β-sheet,': 1, 'loop-3': 1, '2021)).': 1, 'map': 2, 'range': 1, 'mutants,': 1, '1-340).': 1, 'impaired,': 1, 'mutant.': 1, 'systems': 1, 'Fairley': 1, 'induces': 1, 'modifying': 1, 'enzymes.': 1, 'released.': 1, '2006).We': 1, 'accelerated': 2, 'opposite:': 1, '46).': 2, 'depleted,': 1, 'inaccessible': 1, 'speed': 1, 'model,': 1, 'pulldown': 1, 'co-expression': 1, 'increases': 1, 'thermophila.Schematic': 1, '3’-cleavage': 1, 'Z.': 1, 'Rex1.': 1, 'delayed': 1, 'depicted': 1, 'black),': 1, 'teal).': 1, 'Endonuclease': 1, 'BioRender.RNA': 1, 'thermophilaNon-coding': 1, 'assume': 1, 'proportion': 1, 'transcriptome,': 1, 'vertebrate': 1, 'genomes.': 1, '(O’Reilly': 2, 'in-depth': 1, 'manual': 2, 'IGV': 1, 'identified.': 1, 'slight': 1, 'variation': 1, 'underlined:': 1, 'UAAUUAAAAAAAA.': 1, 'If': 1, 'normally,': 1, 'run-through': 1, 'elucidated.': 1, 'Computational': 1, 'valuable': 1, '(from': 1, 'ChIP': 3, 'data)': 1, 'presence/absence': 1, 'pseudogenes': 3, 'arose': 1, 'imperfect': 1, 'mutations,': 1, 'deletions': 1, 'insertions': 1, 'disrupt': 2, '(Balakirev': 1, 'Ayala': 1, 'location,': 1, 'Chromatin': 1, '(ChIP)': 1, 'actively': 1, '2013).snoRNAs': 1, '(approximately': 1, '60%)': 1, '40%': 1, 'CD16.': 1, 'elucidating': 1, 'besides': 1, 'independently': 1, 'half': 1, 'experimental': 1, 'bioinformatics-based': 1, 'predictions.': 1, 'biogenesisThe': 1, 'best-characterized': 1, '1982),': 1, '1985),': 1, '(SRP)': 1, 'Y': 1, '(Simons': 1, 'vault': 1, '(vRNAs)': 1, '(Kickhoefer': 1, 'Besides': 1, 'establishing': 1, 'Intine': 1, 'LC-MS.': 1, 'Sm-B)': 1, 'Nop10).': 1, 'RNPs': 1, 'basepairing': 1, 'particle.': 1, 'recruited,': 1, 'joining': 1, '(Spiller': 1, '1989)': 1, 'modifications,': 2, 'Lange': 1, 'Gerbi': 1, 'unmodified': 1, '(Bertrand': 1, 'Massenet': 1, 'modifications.': 1, '5’-stem': 4, 'C/D21': 1, 'C/D30': 1, 'subnuclear': 1, 'bodies': 3, 'scaRNAs': 4, 'Cajal-body': 1, 'CAB-box': 1, 'UGAG': 1, '(Richard': 1, '(Novotný': 2, '2011;': 3, 'Staněk': 1, 'Neugebauer': 1, 'guiding': 1, 'Prp24': 30, '(SART3': 1, 'p110': 1, 'humans)': 1, '(Rader': 3, 'Raghunathan': 2, 'Free': 1, '3’-internal': 5, 'stem-loop': 3, 'unfolded': 1, '47)': 3, '(Fortner': 1, 'established,': 1, '(Montemayor': 8, 'SNFFL': 7, 'RRMs,': 1, 'RRM1-3': 1, 'β1': 1, 'RRM4': 2, 'non-canonical': 1, 'occluded': 1, '(oRRM)': 1, 'obscure': 1, 'face': 1, '(Martin-Tumasz': 2, 'oRRM4': 4, 'unwind': 1, 'stepwise': 1, 'Prp24/SART3': 1, 'rearrangement': 1, '(ISL)': 1, '2-8': 1, 'image).': 1, 'clarity.': 1, '2018).Previous': 1, 'mutant,': 1, 'di-snRNP.': 1, 'ATP-independent': 1, 'chaperones,': 1, 'reorganizing': 1, '(Ohndorf': 1, 'notable': 1, 'immunoprecipitates': 1, 'rearranged': 1, '47).': 1, 'associates,': 1, 'U4-U6•U5': 1, 'tri-snRNPs': 1, 'Prp31': 1, 'hPrp6,': 1, 'combinations': 1, 'Mlp1.We': 1, '48a)': 1, 'Prp24-U6': 1, 'Prp24-LSm2-8': 1, 'express': 1, 'Conservation': 1, 'LSm5': 4, 'LSm7': 3, '48b,d).': 1, 'Lsm2': 1, '2018),': 1, '48c,d).': 1, 'Mutation': 1, 'two-fold': 1, 'U6•Prp24': 1, 'RRM3': 2, '48d).': 1, 'Prp24,': 2, 'caption': 1, 'protein-binding': 1, '5VSU)': 1, 'ring-structure': 1, 'shades': 3, 'brown,': 1, 'yellow,': 1, 'green),': 1, 'shaded': 1, 'purple)': 1, 'LSm2.': 1, 'LSm2': 2, '(different': 1, 'brown': 1, 'LSm7)': 1, 'conservation.The': 1, '49).': 1, 'species.Figure': 1, 'Experimentally': 1, '2’-O-methylates': 1, '2012).Box': 1, 'indirectly': 1, 'Misregulation': 1, 'effects.': 1, '(King': 1, '2003),': 1, '(Dönmez': 1, 'Valadkhan': 1, 'Yang': 1, '99': 1, '196': 1, 'optimization.': 1, '2012)': 1, 'spatial': 1, 'ribosome.': 1, 'address': 1, '60S': 1, '4V5O,': 1, '4BTS': 1, '4V8P,': 1, 'Rabl': 1, 'Weisser': 1, '2013)': 1, 'identifications': 1, 'future.': 1, 'phenotypic': 1, '1988)': 1, 'alleles': 1, 'concentrations': 1, 'amitotic': 1, 'division': 1, 'currently': 1, 'experiment': 1, 'inquire': 1, 'metabolism': 1, 'viability': 1, 'knock-in': 1, 'viability.': 1, 'hinging': 1, 'assumption': 1, 'partner': 1, 'shed': 1, 'why': 1, 'system.Overall,': 1, 'ultimately': 1, 'eventually': 1, 'matching': 1, 'Before': 1, 'translated,': 1, 'pre-mRNA,': 1, 'undergone': 1}\n"
     ]
    }
   ],
   "source": [
    "counter_lists = dict(Counter(results_string.replace('\\n',\" \").split()))\n",
    "print(counter_lists)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "a8921016",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-03T16:16:44.829930Z",
     "start_time": "2022-11-03T16:16:44.072584Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                0\n",
      "Processing      6\n",
      "of           1338\n",
      "non-coding     40\n",
      "RNAs           61\n",
      "by            404\n",
      "...           ...\n",
      "matching        1\n",
      "Before          1\n",
      "translated,     1\n",
      "pre-mRNA,       1\n",
      "undergone       1\n",
      "\n",
      "[6664 rows x 1 columns]\n"
     ]
    }
   ],
   "source": [
    "df = pd.DataFrame(data=counter_lists, index=[0])\n",
    "\n",
    "df = (df.T)\n",
    "\n",
    "print (df)\n",
    "\n",
    "df.to_excel('Word Frequency.xlsx')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e41903e9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.10"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": false
  },
  "varInspector": {
   "cols": {
    "lenName": 16,
    "lenType": 16,
    "lenVar": 40
   },
   "kernels_config": {
    "python": {
     "delete_cmd_postfix": "",
     "delete_cmd_prefix": "del ",
     "library": "var_list.py",
     "varRefreshCmd": "print(var_dic_list())"
    },
    "r": {
     "delete_cmd_postfix": ") ",
     "delete_cmd_prefix": "rm(",
     "library": "var_list.r",
     "varRefreshCmd": "cat(var_dic_list()) "
    }
   },
   "types_to_exclude": [
    "module",
    "function",
    "builtin_function_or_method",
    "instance",
    "_Feature"
   ],
   "window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
